{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "lovAkRkjuwVa"
      },
      "source": [
        "Add your drive"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "nU1lcLVMD9Wb",
        "outputId": "2dc91f1c-db19-4939-c557-7aa7c49b5b5f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ],
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "_yMnYV1VjZvU"
      },
      "source": [
        "https://drive.google.com/drive/folders/1TGB0qrdiBkFc9TTxIYgoqLmpCp1X0PZx?usp=sharing"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "5WYrl9hkhQGb"
      },
      "source": [
        "https://drive.google.com/drive/folders/1GKE0toQ8tJaDT0AXQjqgJP6rFKMy0r39?usp=sharing"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "EY-iD5EjkT8Z"
      },
      "source": [
        "qiime2 objects"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "id": "me3STQJGhHSQ"
      },
      "outputs": [],
      "source": [
        "!cp /content/drive/MyDrive/oralmicrobiomev2/*.* /content/sample_data/"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "KYs6kIJdkVw8"
      },
      "source": [
        "if you would like the fastq files"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "id": "agfz9NfqkAUI"
      },
      "outputs": [],
      "source": [
        "!cp /content/drive/MyDrive/oralmicrobiomev2/*.* /content/sample_data/"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MDk7MrRHSkft",
        "outputId": "0f0e63e0-476b-4f16-d1e1-b9384d0afc14"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "✨🍰✨ Everything looks OK!\n"
          ]
        }
      ],
      "source": [
        "!pip install -q condacolab\n",
        "import condacolab\n",
        "condacolab.install()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "uA4SkAMlSrT4",
        "outputId": "f147d598-721f-4da3-851a-3a8b9a8772b1"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "✨🍰✨ Everything looks OK!\n"
          ]
        }
      ],
      "source": [
        "import condacolab\n",
        "condacolab.check()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xLM2xpUTypb2",
        "outputId": "59549128-4dd0-416e-8598-c1f54aba74a1"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/sample_data\n"
          ]
        }
      ],
      "source": [
        "cd /content/sample_data/"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "iuEwG2Qs1STx",
        "outputId": "c80e5a6a-8c2a-4da5-c23c-9039cfca7079"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2022-04-11 20:55:55--  https://data.qiime2.org/distro/core/qiime2-2022.2-py38-linux-conda.yml\n",
            "Resolving data.qiime2.org (data.qiime2.org)... 54.200.1.12\n",
            "Connecting to data.qiime2.org (data.qiime2.org)|54.200.1.12|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 FOUND\n",
            "Location: https://raw.githubusercontent.com/qiime2/environment-files/master/2022.2/release/qiime2-2022.2-py38-linux-conda.yml [following]\n",
            "--2022-04-11 20:55:55--  https://raw.githubusercontent.com/qiime2/environment-files/master/2022.2/release/qiime2-2022.2-py38-linux-conda.yml\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 9667 (9.4K) [text/plain]\n",
            "Saving to: ‘qiime2-2022.2-py38-linux-conda.yml’\n",
            "\n",
            "qiime2-2022.2-py38- 100%[===================>]   9.44K  --.-KB/s    in 0s      \n",
            "\n",
            "2022-04-11 20:55:55 (42.8 MB/s) - ‘qiime2-2022.2-py38-linux-conda.yml’ saved [9667/9667]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!wget https://data.qiime2.org/distro/core/qiime2-2022.2-py38-linux-conda.yml"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "o22YcB-S2AGx",
        "outputId": "96a4a0d2-025d-41d8-d8e3-b8b005c84869"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) \n",
            "\u001b[1A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[1A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) \n",
            "\u001b[2A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) \n",
            "\u001b[3A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[3A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) \n",
            "\u001b[4A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[4A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) \n",
            "\u001b[5A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) 3 KB / ?? (18.90 KB/s)\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[5A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) Finalizing...\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[5A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) Done\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[5A\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) Done\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[5A\u001b[0K\u001b[2K\rqiime2/label/r2022.2/lin [] (00m:00s) Done\n",
            "\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) 130  B / ?? (866.00  B/s)\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[4A\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) Finalizing...\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[4A\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) Done\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[4A\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) Done\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[4A\u001b[0K\u001b[2K\rqiime2/label/r2022.2/noa [] (00m:00s) Done\n",
            "\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 275 KB / ?? (1.79 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 303 KB / ?? (1.97 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 242 KB / ?? (1.58 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[3A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/noarch          [] (00m:00s) \n",
            "\u001b[4A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "\u001b[4A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "\u001b[2K\rpkgs/main/noarch         [] (00m:00s) \n",
            "\u001b[5A\u001b[2K\rconda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "\u001b[2K\rconda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "\u001b[2K\rbioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "pkgs/main/noarch         [] (00m:00s) 248 KB / ?? (818.49 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "pkgs/main/noarch         [] (00m:00s) Finalizing...\n",
            "conda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "pkgs/main/noarch         [] (00m:00s) Done\n",
            "conda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "pkgs/main/noarch         [] (00m:00s) Done\n",
            "pkgs/main/noarch         [] (00m:00s) Done\n",
            "conda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 822 KB / ?? (2.67 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 897 KB / ?? (2.91 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 322 KB / ?? (1.04 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) \n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.27 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 1 MB / ?? (2.62 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 1 MB / ?? (2.61 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 605 KB / ?? (1.31 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 516 KB / ?? (862.89 KB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 1 MB / ?? (2.25 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.59 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.70 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 1 MB / ?? (2.42 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 1 MB / ?? (1.71 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.31 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.63 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.73 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 2 MB / ?? (2.47 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 2 MB / ?? (2.09 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 2 MB / ?? (2.36 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 2 MB / ?? (2.52 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 2 MB / ?? (2.64 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 3 MB / ?? (2.72 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 3 MB / ?? (2.72 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.87 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:00s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:00s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:00s) 3 MB / ?? (2.72 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) 3 MB / ?? (2.92 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "bioconda/noarch          [] (00m:00s) Finalizing...\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "bioconda/noarch          [] (00m:01s) Done\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "bioconda/noarch          [] (00m:01s) Done\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "bioconda/noarch          [] (00m:01s) Done\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.42 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.57 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.72 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.51 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 3 MB / ?? (2.27 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 4 MB / ?? (2.35 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 4 MB / ?? (2.35 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.23 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) 4 MB / ?? (2.35 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:00s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) Finalizing...\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) Done\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) Done\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "bioconda/linux-64        [] (00m:01s) Done\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (2.02 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 3 MB / ?? (2.21 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 3 MB / ?? (2.15 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "pkgs/r/noarch            [] (00m:00s) \n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "pkgs/r/noarch            [] (00m:00s) 164 KB / ?? (91.29 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "pkgs/r/noarch            [] (00m:00s) Finalizing...\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "pkgs/r/noarch            [] (00m:00s) Done\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "pkgs/r/noarch            [] (00m:00s) Done\n",
            "pkgs/r/noarch            [] (00m:00s) Done\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) 4 MB / ?? (2.03 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) Finalizing...\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) Done\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "pkgs/main/linux-64       [] (00m:01s) Done\n",
            "pkgs/main/linux-64       [] (00m:01s) Done\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 3 MB / ?? (1.90 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 4 MB / ?? (1.82 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 4 MB / ?? (1.82 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.08 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 4 MB / ?? (1.82 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.00 MB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 4 MB / ?? (1.82 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.00 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) \n",
            "conda-forge/linux-64     [] (00m:01s) 4 MB / ?? (1.82 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.00 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 17 KB / ?? (8.72 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 4 MB / ?? (1.82 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.00 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 17 KB / ?? (8.72 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.00 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 17 KB / ?? (8.72 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 4 MB / ?? (2.00 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 17 KB / ?? (8.72 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 5 MB / ?? (2.35 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 17 KB / ?? (8.72 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 5 MB / ?? (2.35 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 17 KB / ?? (8.72 KB/s)\n",
            "conda-forge/linux-64     [] (00m:01s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:01s) 5 MB / ?? (2.35 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) 920 KB / ?? (429.71 KB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) Finalizing...\n",
            "conda-forge/linux-64     [] (00m:02s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) Done\n",
            "conda-forge/linux-64     [] (00m:02s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "pkgs/r/linux-64          [] (00m:00s) Done\n",
            "pkgs/r/linux-64          [] (00m:00s) Done\n",
            "conda-forge/linux-64     [] (00m:02s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 5 MB / ?? (2.45 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.52 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.52 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.35 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.52 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.26 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.52 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.26 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.63 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.26 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.63 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 5 MB / ?? (2.26 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.63 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 6 MB / ?? (2.54 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 6 MB / ?? (2.63 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 6 MB / ?? (2.54 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 6 MB / ?? (2.54 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 6 MB / ?? (2.54 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) 7 MB / ?? (2.86 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) Finalizing...\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) Done\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/noarch       [] (00m:02s) Done\n",
            "conda-forge/noarch       [] (00m:02s) Done\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.78 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.55 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 7 MB / ?? (2.55 MB/s)\n",
            "conda-forge/linux-64     [] (00m:02s) 9 MB / ?? (3.12 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 9 MB / ?? (3.12 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 10 MB / ?? (3.19 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 10 MB / ?? (3.19 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 11 MB / ?? (3.26 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 11 MB / ?? (3.26 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 12 MB / ?? (3.32 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 12 MB / ?? (3.32 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 12 MB / ?? (3.36 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 12 MB / ?? (3.36 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 13 MB / ?? (3.39 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 13 MB / ?? (3.39 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 13 MB / ?? (3.43 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 13 MB / ?? (3.43 MB/s)\n",
            "conda-forge/linux-64     [] (00m:03s) 14 MB / ?? (3.48 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 14 MB / ?? (3.48 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 15 MB / ?? (3.49 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 15 MB / ?? (3.49 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 15 MB / ?? (3.52 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 15 MB / ?? (3.52 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 16 MB / ?? (3.55 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 16 MB / ?? (3.55 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 17 MB / ?? (3.57 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 17 MB / ?? (3.57 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 17 MB / ?? (3.59 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 17 MB / ?? (3.59 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 18 MB / ?? (3.61 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 18 MB / ?? (3.61 MB/s)\n",
            "conda-forge/linux-64     [] (00m:04s) 19 MB / ?? (3.63 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 19 MB / ?? (3.63 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 19 MB / ?? (3.64 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 19 MB / ?? (3.64 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 20 MB / ?? (3.65 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 20 MB / ?? (3.65 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 20 MB / ?? (3.67 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 20 MB / ?? (3.67 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) 21 MB / ?? (3.69 MB/s)\n",
            "conda-forge/linux-64     [] (00m:05s) Finalizing...\n",
            "conda-forge/linux-64     [] (00m:06s) Done\n",
            "conda-forge/linux-64     [] (00m:06s) Done\n",
            "conda-forge/linux-64     [] (00m:06s) Done\n",
            "\n",
            "\n",
            "Looking for: ['_libgcc_mutex=0.1', '_openmp_mutex=4.5', '_r-mutex=1.0.1', 'alsa-lib=1.2.3', 'argcomplete=2.0.0', 'argon2-cffi=21.3.0', 'argon2-cffi-bindings=21.2.0', 'asttokens=2.0.5', 'attrs=21.4.0', 'backcall=0.2.0', 'backports=1.0', 'backports.functools_lru_cache=1.6.4', 'beautifulsoup4=4.11.0', 'bibtexparser=1.1.0', 'binutils_impl_linux-64=2.36.1', 'binutils_linux-64=2.36', 'bioconductor-biobase=2.54.0', 'bioconductor-biocgenerics=0.40.0', 'bioconductor-biocparallel=1.28.3', 'bioconductor-biostrings=2.62.0', 'bioconductor-dada2=1.22.0', 'bioconductor-delayedarray=0.20.0', 'bioconductor-genomeinfodb=1.30.0', 'bioconductor-genomeinfodbdata=1.2.7', 'bioconductor-genomicalignments=1.30.0', 'bioconductor-genomicranges=1.46.1', 'bioconductor-iranges=2.28.0', 'bioconductor-matrixgenerics=1.6.0', 'bioconductor-rhtslib=1.26.0', 'bioconductor-rsamtools=2.10.0', 'bioconductor-s4vectors=0.32.3', 'bioconductor-shortread=1.52.0', 'bioconductor-summarizedexperiment=1.24.0', 'bioconductor-xvector=0.34.0', 'bioconductor-zlibbioc=1.40.0', 'biom-format=2.1.10', 'blast=2.12.0', 'bleach=5.0.0', 'bokeh=2.4.2', 'bowtie2=2.4.5', 'brotli=1.0.9', 'brotli-bin=1.0.9', 'brotlipy=0.7.0', 'bwidget=1.9.14', 'bzip2=1.0.8', 'c-ares=1.18.1', 'ca-certificates=2021.10.8', 'cachecontrol=0.12.10', 'cached-property=1.5.2', 'cached_property=1.5.2', 'cairo=1.16.0', 'certifi=2021.10.8', 'cffi=1.15.0', 'charset-normalizer=2.0.12', 'click=7.1.2', 'colorama=0.4.4', 'cryptography=36.0.2', 'curl=7.82.0', 'cutadapt=3.7', 'cycler=0.11.0', 'cython=0.29.28', 'dbus=1.13.6', 'deblur=1.1.0', 'debugpy=1.5.1', 'decorator=4.4.2', 'defusedxml=0.7.1', 'dendropy=4.5.2', 'dill=0.3.4', 'dnaio=0.8.1', 'emperor=1.0.3', 'entrez-direct=16.2', 'entrypoints=0.4', 'executing=0.8.3', 'expat=2.4.8', 'fastcluster=1.2.6', 'fasttree=2.1.10', 'flit-core=3.7.1', 'font-ttf-dejavu-sans-mono=2.37', 'font-ttf-inconsolata=3.000', 'font-ttf-source-code-pro=2.038', 'font-ttf-ubuntu=0.83', 'fontconfig=2.14.0', 'fonts-conda-ecosystem=1', 'fonts-conda-forge=1', 'fonttools=4.31.2', 'freetype=2.10.4', 'fribidi=1.0.10', 'future=0.18.2', 'gcc_impl_linux-64=10.3.0', 'gcc_linux-64=10.3.0', 'gettext=0.19.8.1', 'gfortran_impl_linux-64=10.3.0', 'gfortran_linux-64=10.3.0', 'giflib=5.2.1', 'gneiss=0.4.6', 'graphite2=1.3.13', 'gsl=2.7', 'gst-plugins-base=1.20.1', 'gstreamer=1.20.1', 'gxx_impl_linux-64=10.3.0', 'gxx_linux-64=10.3.0', 'h5py=3.6.0', 'harfbuzz=4.2.0', 'hdf5=1.12.1', 'hdmedians=0.14.2', 'hmmer=3.1b2', 'htslib=1.15.1', 'icu=69.1', 'idna=3.3', 'ijson=3.1.3', 'importlib-metadata=4.11.3', 'importlib_metadata=4.11.3', 'importlib_resources=5.6.0', 'iniconfig=1.1.1', 'ipykernel=6.12.1', 'ipython=8.2.0', 'ipython_genutils=0.2.0', 'ipywidgets=7.7.0', 'iqtree=2.2.0_beta', 'isa-l=2.30.0', 'jbig=2.1', 'jedi=0.18.1', 'jinja2=3.1.1', 'joblib=1.1.0', 'jpeg=9e', 'jq=1.6', 'jsonschema=4.4.0', 'jupyter_client=7.2.2', 'jupyter_core=4.9.2', 'jupyterlab_pygments=0.2.0', 'jupyterlab_widgets=1.1.0', 'kernel-headers_linux-64=2.6.32', 'keyutils=1.6.1', 'kiwisolver=1.4.2', 'krb5=1.19.3', 'lcms2=2.12', 'ld_impl_linux-64=2.36.1', 'lerc=3.0', 'libblas=3.9.0', 'libbrotlicommon=1.0.9', 'libbrotlidec=1.0.9', 'libbrotlienc=1.0.9', 'libcblas=3.9.0', 'libclang=13.0.1', 'libcurl=7.82.0', 'libdeflate=1.10', 'libedit=3.1.20191231', 'libev=4.33', 'libevent=2.1.10', 'libffi=3.4.2', 'libgcc=7.2.0', 'libgcc-devel_linux-64=10.3.0', 'libgcc-ng=11.2.0', 'libgfortran-ng=11.2.0', 'libgfortran5=11.2.0', 'libglib=2.70.2', 'libgomp=11.2.0', 'libiconv=1.16', 'liblapack=3.9.0', 'libllvm10=10.0.1', 'libllvm13=13.0.1', 'libnghttp2=1.47.0', 'libnsl=2.0.0', 'libogg=1.3.4', 'libopenblas=0.3.20', 'libopus=1.3.1', 'libpng=1.6.37', 'libpq=14.2', 'libsanitizer=10.3.0', 'libsodium=1.0.18', 'libssh2=1.10.0', 'libstdcxx-devel_linux-64=10.3.0', 'libstdcxx-ng=11.2.0', 'libtiff=4.3.0', 'libuuid=2.32.1', 'libvorbis=1.3.7', 'libwebp=1.2.2', 'libwebp-base=1.2.2', 'libxcb=1.13', 'libxkbcommon=1.0.3', 'libxml2=2.9.12', 'libxslt=1.1.33', 'libzlib=1.2.11', 'llvmlite=0.36.0', 'lockfile=0.12.2', 'lxml=4.8.0', 'lz4=4.0.0', 'lz4-c=1.9.3', 'mafft=7.490', 'make=4.3', 'markupsafe=2.1.1', 'matplotlib=3.5.1', 'matplotlib-base=3.5.1', 'matplotlib-inline=0.1.3', 'mistune=0.8.4', 'msgpack-python=1.0.3', 'munkres=1.1.4', 'mysql-common=8.0.28', 'mysql-libs=8.0.28', 'natsort=8.1.0', 'nbclient=0.5.13', 'nbconvert=6.4.5', 'nbconvert-core=6.4.5', 'nbconvert-pandoc=6.4.5', 'nbformat=5.3.0', 'ncurses=6.3', 'nest-asyncio=1.5.5', 'networkx=2.7.1', 'nose=1.3.7', 'notebook=6.4.10', 'nspr=4.32', 'nss=3.77', 'numba=0.53.1', 'numpy=1.22.3', 'oniguruma=6.9.7.1', 'openjdk=11.0.1', 'openjpeg=2.4.0', 'openssl=1.1.1n', 'packaging=21.3', 'pandas=1.2.5', 'pandoc=2.17.1.1', 'pandocfilters=1.5.0', 'pango=1.50.6', 'parso=0.8.3', 'patsy=0.5.2', 'pbzip2=1.1.13', 'pcre=8.45', 'pcre2=10.37', 'perl=5.32.1', 'perl-archive-tar=2.40', 'perl-carp=1.50', 'perl-common-sense=3.75', 'perl-compress-raw-bzip2=2.103', 'perl-compress-raw-zlib=2.103', 'perl-encode=3.17', 'perl-exporter=5.74', 'perl-exporter-tiny=1.002002', 'perl-extutils-makemaker=7.64', 'perl-io-compress=2.102', 'perl-io-zlib=1.11', 'perl-json=4.05', 'perl-json-xs=2.34', 'perl-list-moreutils=0.430', 'perl-list-moreutils-xs=0.430', 'perl-parent=0.238', 'perl-pathtools=3.75', 'perl-scalar-list-utils=1.62', 'perl-types-serialiser=1.01', 'pexpect=4.8.0', 'pickleshare=0.7.5', 'pigz=2.6', 'pillow=9.1.0', 'pip=22.0.4', 'pixman=0.40.0', 'pluggy=1.0.0', 'prometheus_client=0.14.0', 'prompt-toolkit=3.0.29', 'psutil=5.9.0', 'pthread-stubs=0.4', 'ptyprocess=0.7.0', 'pure_eval=0.2.2', 'py=1.11.0', 'pycparser=2.21', 'pygments=2.11.2', 'pynndescent=0.5.6', 'pyopenssl=22.0.0', 'pyparsing=3.0.7', 'pyqt=5.12.3', 'pyqt-impl=5.12.3', 'pyqt5-sip=4.19.18', 'pyqtchart=5.12', 'pyqtwebengine=5.12.1', 'pyrsistent=0.18.1', 'pysocks=1.7.1', 'pytest=7.1.1', 'python=3.8.13', 'python-dateutil=2.8.2', 'python-fastjsonschema=2.15.3', 'python-isal=0.11.1', 'python_abi=3.8', 'pytz=2022.1', 'pyyaml=6.0', 'pyzmq=22.3.0', 'q2-alignment=2022.2.0', 'q2-composition=2022.2.0', 'q2-cutadapt=2022.2.0', 'q2-dada2=2022.2.0', 'q2-deblur=2022.2.0', 'q2-demux=2022.2.0', 'q2-diversity=2022.2.0', 'q2-diversity-lib=2022.2.0', 'q2-emperor=2022.2.0', 'q2-feature-classifier=2022.2.0', 'q2-feature-table=2022.2.0', 'q2-fragment-insertion=2022.2.0', 'q2-gneiss=2022.2.0', 'q2-longitudinal=2022.2.0', 'q2-metadata=2022.2.0', 'q2-mystery-stew=2022.2.0', 'q2-phylogeny=2022.2.0', 'q2-quality-control=2022.2.0', 'q2-quality-filter=2022.2.0', 'q2-sample-classifier=2022.2.0', 'q2-taxa=2022.2.0', 'q2-types=2022.2.0', 'q2-vsearch=2022.2.0', 'q2cli=2022.2.0', 'q2galaxy=2022.2.0', 'q2templates=2022.2.0', 'qiime2=2022.2.0', 'qt=5.12.9', 'r-backports=1.4.1', 'r-base=4.1.3', 'r-bh=1.78.0_0', 'r-bitops=1.0_7', 'r-brio=1.1.3', 'r-callr=3.7.0', 'r-cli=3.2.0', 'r-cluster=2.1.3', 'r-colorspace=2.0_3', 'r-crayon=1.5.1', 'r-desc=1.4.1', 'r-diffobj=0.3.5', 'r-digest=0.6.29', 'r-ellipsis=0.3.2', 'r-evaluate=0.15', 'r-fansi=1.0.3', 'r-farver=2.1.0', 'r-formatr=1.12', 'r-futile.logger=1.4.3', 'r-futile.options=1.0.1', 'r-ggplot2=3.3.5', 'r-glue=1.6.2', 'r-gtable=0.3.0', 'r-hwriter=1.3.2', 'r-isoband=0.2.5', 'r-jpeg=0.1_9', 'r-jsonlite=1.8.0', 'r-labeling=0.4.2', 'r-lambda.r=1.2.4', 'r-lattice=0.20_45', 'r-latticeextra=0.6_29', 'r-lifecycle=1.0.1', 'r-magrittr=2.0.3', 'r-mass=7.3_56', 'r-matrix=1.4_1', 'r-matrixstats=0.61.0', 'r-mgcv=1.8_40', 'r-munsell=0.5.0', 'r-nlme=3.1_157', 'r-permute=0.9_7', 'r-pillar=1.7.0', 'r-pkgconfig=2.0.3', 'r-pkgload=1.2.4', 'r-plyr=1.8.7', 'r-png=0.1_7', 'r-praise=1.0.0', 'r-processx=3.5.3', 'r-ps=1.6.0', 'r-r6=2.5.1', 'r-rcolorbrewer=1.1_3', 'r-rcpp=1.0.8.3', 'r-rcppparallel=5.1.5', 'r-rcurl=1.98_1.6', 'r-rematch2=2.1.2', 'r-reshape2=1.4.4', 'r-rlang=1.0.2', 'r-rprojroot=2.0.3', 'r-rstudioapi=0.13', 'r-scales=1.1.1', 'r-snow=0.4_4', 'r-stringi=1.7.6', 'r-stringr=1.4.0', 'r-testthat=3.1.3', 'r-tibble=3.1.6', 'r-utf8=1.2.2', 'r-vctrs=0.4.0', 'r-vegan=2.5_7', 'r-viridislite=0.4.0', 'r-waldo=0.4.0', 'r-withr=2.5.0', 'raxml=8.2.12', 'readline=8.1', 'requests=2.27.1', 'samtools=1.15.1', 'scikit-bio=0.5.6', 'scikit-learn=0.24.1', 'scipy=1.8.0', 'seaborn=0.11.2', 'seaborn-base=0.11.2', 'sed=4.8', 'send2trash=1.8.0', 'sepp=4.3.10', 'setuptools=59.8.0', 'six=1.16.0', 'sortmerna=2.0', 'soupsieve=2.3.1', 'sqlite=3.37.1', 'stack_data=0.2.0', 'statsmodels=0.13.2', 'sysroot_linux-64=2.12', 'tbb=2020.2', 'terminado=0.13.3', 'testpath=0.6.0', 'threadpoolctl=3.1.0', 'tk=8.6.12', 'tktable=2.10', 'toml=0.10.2', 'tomli=2.0.1', 'tornado=6.1', 'tqdm=4.64.0', 'traitlets=5.1.1', 'typing_extensions=4.1.1', 'tzlocal=2.1', 'umap-learn=0.5.2', 'unicodedata2=14.0.0', 'unifrac=0.20.3', 'urllib3=1.26.9', 'vsearch=2.7.0', 'wcwidth=0.2.5', 'webencodings=0.5.1', 'wheel=0.37.1', 'widgetsnbextension=3.6.0', 'xmltodict=0.12.0', 'xopen=1.5.0', 'xorg-kbproto=1.0.7', 'xorg-libice=1.0.10', 'xorg-libsm=1.2.3', 'xorg-libx11=1.7.2', 'xorg-libxau=1.0.9', 'xorg-libxdmcp=1.1.3', 'xorg-libxext=1.3.4', 'xorg-libxrender=0.9.10', 'xorg-libxt=1.2.1', 'xorg-renderproto=0.11.1', 'xorg-xextproto=7.3.0', 'xorg-xproto=7.0.31', 'xz=5.2.5', 'yaml=0.2.5', 'yq=2.13.0', 'zeromq=4.3.4', 'zipp=3.8.0', 'zlib=1.2.11', 'zstd=1.5.2']\n",
            "\n",
            "\n",
            "Transaction\n",
            "\n",
            "  Prefix: /usr/local\n",
            "\n",
            "  Updating specs:\n",
            "\n",
            "   - _libgcc_mutex==0.1\n",
            "   - _openmp_mutex==4.5\n",
            "   - _r-mutex==1.0.1\n",
            "   - alsa-lib==1.2.3\n",
            "   - argcomplete==2.0.0\n",
            "   - argon2-cffi==21.3.0\n",
            "   - argon2-cffi-bindings==21.2.0\n",
            "   - asttokens==2.0.5\n",
            "   - attrs==21.4.0\n",
            "   - backcall==0.2.0\n",
            "   - backports==1.0\n",
            "   - backports.functools_lru_cache==1.6.4\n",
            "   - beautifulsoup4==4.11.0\n",
            "   - bibtexparser==1.1.0\n",
            "   - binutils_impl_linux-64==2.36.1\n",
            "   - binutils_linux-64==2.36\n",
            "   - bioconductor-biobase==2.54.0\n",
            "   - bioconductor-biocgenerics==0.40.0\n",
            "   - bioconductor-biocparallel==1.28.3\n",
            "   - bioconductor-biostrings==2.62.0\n",
            "   - bioconductor-dada2==1.22.0\n",
            "   - bioconductor-delayedarray==0.20.0\n",
            "   - bioconductor-genomeinfodb==1.30.0\n",
            "   - bioconductor-genomeinfodbdata==1.2.7\n",
            "   - bioconductor-genomicalignments==1.30.0\n",
            "   - bioconductor-genomicranges==1.46.1\n",
            "   - bioconductor-iranges==2.28.0\n",
            "   - bioconductor-matrixgenerics==1.6.0\n",
            "   - bioconductor-rhtslib==1.26.0\n",
            "   - bioconductor-rsamtools==2.10.0\n",
            "   - bioconductor-s4vectors==0.32.3\n",
            "   - bioconductor-shortread==1.52.0\n",
            "   - bioconductor-summarizedexperiment==1.24.0\n",
            "   - bioconductor-xvector==0.34.0\n",
            "   - bioconductor-zlibbioc==1.40.0\n",
            "   - biom-format==2.1.10\n",
            "   - blast==2.12.0\n",
            "   - bleach==5.0.0\n",
            "   - bokeh==2.4.2\n",
            "   - bowtie2==2.4.5\n",
            "   - brotli==1.0.9\n",
            "   - brotli-bin==1.0.9\n",
            "   - brotlipy==0.7.0\n",
            "   - bwidget==1.9.14\n",
            "   - bzip2==1.0.8\n",
            "   - c-ares==1.18.1\n",
            "   - ca-certificates==2021.10.8\n",
            "   - cachecontrol==0.12.10\n",
            "   - cached-property==1.5.2\n",
            "   - cached_property==1.5.2\n",
            "   - cairo==1.16.0\n",
            "   - certifi==2021.10.8\n",
            "   - cffi==1.15.0\n",
            "   - charset-normalizer==2.0.12\n",
            "   - click==7.1.2\n",
            "   - colorama==0.4.4\n",
            "   - cryptography==36.0.2\n",
            "   - curl==7.82.0\n",
            "   - cutadapt==3.7\n",
            "   - cycler==0.11.0\n",
            "   - cython==0.29.28\n",
            "   - dbus==1.13.6\n",
            "   - deblur==1.1.0\n",
            "   - debugpy==1.5.1\n",
            "   - decorator==4.4.2\n",
            "   - defusedxml==0.7.1\n",
            "   - dendropy==4.5.2\n",
            "   - dill==0.3.4\n",
            "   - dnaio==0.8.1\n",
            "   - emperor==1.0.3\n",
            "   - entrez-direct==16.2\n",
            "   - entrypoints==0.4\n",
            "   - executing==0.8.3\n",
            "   - expat==2.4.8\n",
            "   - fastcluster==1.2.6\n",
            "   - fasttree==2.1.10\n",
            "   - flit-core==3.7.1\n",
            "   - font-ttf-dejavu-sans-mono==2.37\n",
            "   - font-ttf-inconsolata==3.000\n",
            "   - font-ttf-source-code-pro==2.038\n",
            "   - font-ttf-ubuntu==0.83\n",
            "   - fontconfig==2.14.0\n",
            "   - fonts-conda-ecosystem==1\n",
            "   - fonts-conda-forge==1\n",
            "   - fonttools==4.31.2\n",
            "   - freetype==2.10.4\n",
            "   - fribidi==1.0.10\n",
            "   - future==0.18.2\n",
            "   - gcc_impl_linux-64==10.3.0\n",
            "   - gcc_linux-64==10.3.0\n",
            "   - gettext==0.19.8.1\n",
            "   - gfortran_impl_linux-64==10.3.0\n",
            "   - gfortran_linux-64==10.3.0\n",
            "   - giflib==5.2.1\n",
            "   - gneiss==0.4.6\n",
            "   - graphite2==1.3.13\n",
            "   - gsl==2.7\n",
            "   - gst-plugins-base==1.20.1\n",
            "   - gstreamer==1.20.1\n",
            "   - gxx_impl_linux-64==10.3.0\n",
            "   - gxx_linux-64==10.3.0\n",
            "   - h5py==3.6.0\n",
            "   - harfbuzz==4.2.0\n",
            "   - hdf5==1.12.1\n",
            "   - hdmedians==0.14.2\n",
            "   - hmmer==3.1b2\n",
            "   - htslib==1.15.1\n",
            "   - icu==69.1\n",
            "   - idna==3.3\n",
            "   - ijson==3.1.3\n",
            "   - importlib-metadata==4.11.3\n",
            "   - importlib_metadata==4.11.3\n",
            "   - importlib_resources==5.6.0\n",
            "   - iniconfig==1.1.1\n",
            "   - ipykernel==6.12.1\n",
            "   - ipython==8.2.0\n",
            "   - ipython_genutils==0.2.0\n",
            "   - ipywidgets==7.7.0\n",
            "   - iqtree==2.2.0_beta\n",
            "   - isa-l==2.30.0\n",
            "   - jbig==2.1\n",
            "   - jedi==0.18.1\n",
            "   - jinja2==3.1.1\n",
            "   - joblib==1.1.0\n",
            "   - jpeg==9e\n",
            "   - jq==1.6\n",
            "   - jsonschema==4.4.0\n",
            "   - jupyter_client==7.2.2\n",
            "   - jupyter_core==4.9.2\n",
            "   - jupyterlab_pygments==0.2.0\n",
            "   - jupyterlab_widgets==1.1.0\n",
            "   - kernel-headers_linux-64==2.6.32\n",
            "   - keyutils==1.6.1\n",
            "   - kiwisolver==1.4.2\n",
            "   - krb5==1.19.3\n",
            "   - lcms2==2.12\n",
            "   - ld_impl_linux-64==2.36.1\n",
            "   - lerc==3.0\n",
            "   - libblas==3.9.0\n",
            "   - libbrotlicommon==1.0.9\n",
            "   - libbrotlidec==1.0.9\n",
            "   - libbrotlienc==1.0.9\n",
            "   - libcblas==3.9.0\n",
            "   - libclang==13.0.1\n",
            "   - libcurl==7.82.0\n",
            "   - libdeflate==1.10\n",
            "   - libedit==3.1.20191231\n",
            "   - libev==4.33\n",
            "   - libevent==2.1.10\n",
            "   - libffi==3.4.2\n",
            "   - libgcc==7.2.0\n",
            "   - libgcc-devel_linux-64==10.3.0\n",
            "   - libgcc-ng==11.2.0\n",
            "   - libgfortran-ng==11.2.0\n",
            "   - libgfortran5==11.2.0\n",
            "   - libglib==2.70.2\n",
            "   - libgomp==11.2.0\n",
            "   - libiconv==1.16\n",
            "   - liblapack==3.9.0\n",
            "   - libllvm10==10.0.1\n",
            "   - libllvm13==13.0.1\n",
            "   - libnghttp2==1.47.0\n",
            "   - libnsl==2.0.0\n",
            "   - libogg==1.3.4\n",
            "   - libopenblas==0.3.20\n",
            "   - libopus==1.3.1\n",
            "   - libpng==1.6.37\n",
            "   - libpq==14.2\n",
            "   - libsanitizer==10.3.0\n",
            "   - libsodium==1.0.18\n",
            "   - libssh2==1.10.0\n",
            "   - libstdcxx-devel_linux-64==10.3.0\n",
            "   - libstdcxx-ng==11.2.0\n",
            "   - libtiff==4.3.0\n",
            "   - libuuid==2.32.1\n",
            "   - libvorbis==1.3.7\n",
            "   - libwebp==1.2.2\n",
            "   - libwebp-base==1.2.2\n",
            "   - libxcb==1.13\n",
            "   - libxkbcommon==1.0.3\n",
            "   - libxml2==2.9.12\n",
            "   - libxslt==1.1.33\n",
            "   - libzlib==1.2.11\n",
            "   - llvmlite==0.36.0\n",
            "   - lockfile==0.12.2\n",
            "   - lxml==4.8.0\n",
            "   - lz4==4.0.0\n",
            "   - lz4-c==1.9.3\n",
            "   - mafft==7.490\n",
            "   - make==4.3\n",
            "   - markupsafe==2.1.1\n",
            "   - matplotlib==3.5.1\n",
            "   - matplotlib-base==3.5.1\n",
            "   - matplotlib-inline==0.1.3\n",
            "   - mistune==0.8.4\n",
            "   - msgpack-python==1.0.3\n",
            "   - munkres==1.1.4\n",
            "   - mysql-common==8.0.28\n",
            "   - mysql-libs==8.0.28\n",
            "   - natsort==8.1.0\n",
            "   - nbclient==0.5.13\n",
            "   - nbconvert==6.4.5\n",
            "   - nbconvert-core==6.4.5\n",
            "   - nbconvert-pandoc==6.4.5\n",
            "   - nbformat==5.3.0\n",
            "   - ncurses==6.3\n",
            "   - nest-asyncio==1.5.5\n",
            "   - networkx==2.7.1\n",
            "   - nose==1.3.7\n",
            "   - notebook==6.4.10\n",
            "   - nspr==4.32\n",
            "   - nss==3.77\n",
            "   - numba==0.53.1\n",
            "   - numpy==1.22.3\n",
            "   - oniguruma==6.9.7.1\n",
            "   - openjdk==11.0.1\n",
            "   - openjpeg==2.4.0\n",
            "   - openssl==1.1.1n\n",
            "   - packaging==21.3\n",
            "   - pandas==1.2.5\n",
            "   - pandoc==2.17.1.1\n",
            "   - pandocfilters==1.5.0\n",
            "   - pango==1.50.6\n",
            "   - parso==0.8.3\n",
            "   - patsy==0.5.2\n",
            "   - pbzip2==1.1.13\n",
            "   - pcre==8.45\n",
            "   - pcre2==10.37\n",
            "   - perl==5.32.1\n",
            "   - perl-archive-tar==2.40\n",
            "   - perl-carp==1.50\n",
            "   - perl-common-sense==3.75\n",
            "   - perl-compress-raw-bzip2==2.103\n",
            "   - perl-compress-raw-zlib==2.103\n",
            "   - perl-encode==3.17\n",
            "   - perl-exporter==5.74\n",
            "   - perl-exporter-tiny==1.002002\n",
            "   - perl-extutils-makemaker==7.64\n",
            "   - perl-io-compress==2.102\n",
            "   - perl-io-zlib==1.11\n",
            "   - perl-json==4.05\n",
            "   - perl-json-xs==2.34\n",
            "   - perl-list-moreutils==0.430\n",
            "   - perl-list-moreutils-xs==0.430\n",
            "   - perl-parent==0.238\n",
            "   - perl-pathtools==3.75\n",
            "   - perl-scalar-list-utils==1.62\n",
            "   - perl-types-serialiser==1.01\n",
            "   - pexpect==4.8.0\n",
            "   - pickleshare==0.7.5\n",
            "   - pigz==2.6\n",
            "   - pillow==9.1.0\n",
            "   - pip==22.0.4\n",
            "   - pixman==0.40.0\n",
            "   - pluggy==1.0.0\n",
            "   - prometheus_client==0.14.0\n",
            "   - prompt-toolkit==3.0.29\n",
            "   - psutil==5.9.0\n",
            "   - pthread-stubs==0.4\n",
            "   - ptyprocess==0.7.0\n",
            "   - pure_eval==0.2.2\n",
            "   - py==1.11.0\n",
            "   - pycparser==2.21\n",
            "   - pygments==2.11.2\n",
            "   - pynndescent==0.5.6\n",
            "   - pyopenssl==22.0.0\n",
            "   - pyparsing==3.0.7\n",
            "   - pyqt==5.12.3\n",
            "   - pyqt-impl==5.12.3\n",
            "   - pyqt5-sip==4.19.18\n",
            "   - pyqtchart==5.12\n",
            "   - pyqtwebengine==5.12.1\n",
            "   - pyrsistent==0.18.1\n",
            "   - pysocks==1.7.1\n",
            "   - pytest==7.1.1\n",
            "   - python==3.8.13\n",
            "   - python-dateutil==2.8.2\n",
            "   - python-fastjsonschema==2.15.3\n",
            "   - python-isal==0.11.1\n",
            "   - python_abi==3.8\n",
            "   - pytz==2022.1\n",
            "   - pyyaml==6.0\n",
            "   - pyzmq==22.3.0\n",
            "   - q2-alignment==2022.2.0\n",
            "   - q2-composition==2022.2.0\n",
            "   - q2-cutadapt==2022.2.0\n",
            "   - q2-dada2==2022.2.0\n",
            "   - q2-deblur==2022.2.0\n",
            "   - q2-demux==2022.2.0\n",
            "   - q2-diversity==2022.2.0\n",
            "   - q2-diversity-lib==2022.2.0\n",
            "   - q2-emperor==2022.2.0\n",
            "   - q2-feature-classifier==2022.2.0\n",
            "   - q2-feature-table==2022.2.0\n",
            "   - q2-fragment-insertion==2022.2.0\n",
            "   - q2-gneiss==2022.2.0\n",
            "   - q2-longitudinal==2022.2.0\n",
            "   - q2-metadata==2022.2.0\n",
            "   - q2-mystery-stew==2022.2.0\n",
            "   - q2-phylogeny==2022.2.0\n",
            "   - q2-quality-control==2022.2.0\n",
            "   - q2-quality-filter==2022.2.0\n",
            "   - q2-sample-classifier==2022.2.0\n",
            "   - q2-taxa==2022.2.0\n",
            "   - q2-types==2022.2.0\n",
            "   - q2-vsearch==2022.2.0\n",
            "   - q2cli==2022.2.0\n",
            "   - q2galaxy==2022.2.0\n",
            "   - q2templates==2022.2.0\n",
            "   - qiime2==2022.2.0\n",
            "   - qt==5.12.9\n",
            "   - r-backports==1.4.1\n",
            "   - r-base==4.1.3\n",
            "   - r-bh==1.78.0_0\n",
            "   - r-bitops==1.0_7\n",
            "   - r-brio==1.1.3\n",
            "   - r-callr==3.7.0\n",
            "   - r-cli==3.2.0\n",
            "   - r-cluster==2.1.3\n",
            "   - r-colorspace==2.0_3\n",
            "   - r-crayon==1.5.1\n",
            "   - r-desc==1.4.1\n",
            "   - r-diffobj==0.3.5\n",
            "   - r-digest==0.6.29\n",
            "   - r-ellipsis==0.3.2\n",
            "   - r-evaluate==0.15\n",
            "   - r-fansi==1.0.3\n",
            "   - r-farver==2.1.0\n",
            "   - r-formatr==1.12\n",
            "   - r-futile.logger==1.4.3\n",
            "   - r-futile.options==1.0.1\n",
            "   - r-ggplot2==3.3.5\n",
            "   - r-glue==1.6.2\n",
            "   - r-gtable==0.3.0\n",
            "   - r-hwriter==1.3.2\n",
            "   - r-isoband==0.2.5\n",
            "   - r-jpeg==0.1_9\n",
            "   - r-jsonlite==1.8.0\n",
            "   - r-labeling==0.4.2\n",
            "   - r-lambda.r==1.2.4\n",
            "   - r-lattice==0.20_45\n",
            "   - r-latticeextra==0.6_29\n",
            "   - r-lifecycle==1.0.1\n",
            "   - r-magrittr==2.0.3\n",
            "   - r-mass==7.3_56\n",
            "   - r-matrix==1.4_1\n",
            "   - r-matrixstats==0.61.0\n",
            "   - r-mgcv==1.8_40\n",
            "   - r-munsell==0.5.0\n",
            "   - r-nlme==3.1_157\n",
            "   - r-permute==0.9_7\n",
            "   - r-pillar==1.7.0\n",
            "   - r-pkgconfig==2.0.3\n",
            "   - r-pkgload==1.2.4\n",
            "   - r-plyr==1.8.7\n",
            "   - r-png==0.1_7\n",
            "   - r-praise==1.0.0\n",
            "   - r-processx==3.5.3\n",
            "   - r-ps==1.6.0\n",
            "   - r-r6==2.5.1\n",
            "   - r-rcolorbrewer==1.1_3\n",
            "   - r-rcpp==1.0.8.3\n",
            "   - r-rcppparallel==5.1.5\n",
            "   - r-rcurl==1.98_1.6\n",
            "   - r-rematch2==2.1.2\n",
            "   - r-reshape2==1.4.4\n",
            "   - r-rlang==1.0.2\n",
            "   - r-rprojroot==2.0.3\n",
            "   - r-rstudioapi==0.13\n",
            "   - r-scales==1.1.1\n",
            "   - r-snow==0.4_4\n",
            "   - r-stringi==1.7.6\n",
            "   - r-stringr==1.4.0\n",
            "   - r-testthat==3.1.3\n",
            "   - r-tibble==3.1.6\n",
            "   - r-utf8==1.2.2\n",
            "   - r-vctrs==0.4.0\n",
            "   - r-vegan==2.5_7\n",
            "   - r-viridislite==0.4.0\n",
            "   - r-waldo==0.4.0\n",
            "   - r-withr==2.5.0\n",
            "   - raxml==8.2.12\n",
            "   - readline==8.1\n",
            "   - requests==2.27.1\n",
            "   - samtools==1.15.1\n",
            "   - scikit-bio==0.5.6\n",
            "   - scikit-learn==0.24.1\n",
            "   - scipy==1.8.0\n",
            "   - seaborn==0.11.2\n",
            "   - seaborn-base==0.11.2\n",
            "   - sed==4.8\n",
            "   - send2trash==1.8.0\n",
            "   - sepp==4.3.10\n",
            "   - setuptools==59.8.0\n",
            "   - six==1.16.0\n",
            "   - sortmerna==2.0\n",
            "   - soupsieve==2.3.1\n",
            "   - sqlite==3.37.1\n",
            "   - stack_data==0.2.0\n",
            "   - statsmodels==0.13.2\n",
            "   - sysroot_linux-64==2.12\n",
            "   - tbb==2020.2\n",
            "   - terminado==0.13.3\n",
            "   - testpath==0.6.0\n",
            "   - threadpoolctl==3.1.0\n",
            "   - tk==8.6.12\n",
            "   - tktable==2.10\n",
            "   - toml==0.10.2\n",
            "   - tomli==2.0.1\n",
            "   - tornado==6.1\n",
            "   - tqdm==4.64.0\n",
            "   - traitlets==5.1.1\n",
            "   - typing_extensions==4.1.1\n",
            "   - tzlocal==2.1\n",
            "   - umap-learn==0.5.2\n",
            "   - unicodedata2==14.0.0\n",
            "   - unifrac==0.20.3\n",
            "   - urllib3==1.26.9\n",
            "   - vsearch==2.7.0\n",
            "   - wcwidth==0.2.5\n",
            "   - webencodings==0.5.1\n",
            "   - wheel==0.37.1\n",
            "   - widgetsnbextension==3.6.0\n",
            "   - xmltodict==0.12.0\n",
            "   - xopen==1.5.0\n",
            "   - xorg-kbproto==1.0.7\n",
            "   - xorg-libice==1.0.10\n",
            "   - xorg-libsm==1.2.3\n",
            "   - xorg-libx11==1.7.2\n",
            "   - xorg-libxau==1.0.9\n",
            "   - xorg-libxdmcp==1.1.3\n",
            "   - xorg-libxext==1.3.4\n",
            "   - xorg-libxrender==0.9.10\n",
            "   - xorg-libxt==1.2.1\n",
            "   - xorg-renderproto==0.11.1\n",
            "   - xorg-xextproto==7.3.0\n",
            "   - xorg-xproto==7.0.31\n",
            "   - xz==5.2.5\n",
            "   - yaml==0.2.5\n",
            "   - yq==2.13.0\n",
            "   - zeromq==4.3.4\n",
            "   - zipp==3.8.0\n",
            "   - zlib==1.2.11\n",
            "   - zstd==1.5.2\n",
            "\n",
            "\n",
            "  Package                                 Version  Build                   Channel                              Size\n",
            "──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────\n",
            "  Install:\n",
            "──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────\n",
            "\n",
            "\u001b[32m  _r-mutex                         \u001b[00m         1.0.1  anacondar_1             conda-forge/noarch                   3 KB\n",
            "\u001b[32m  alsa-lib                         \u001b[00m         1.2.3  h516909a_0              conda-forge/linux-64               560 KB\n",
            "\u001b[32m  argcomplete                      \u001b[00m         2.0.0  pyhd8ed1ab_0            conda-forge/noarch                  34 KB\n",
            "\u001b[32m  argon2-cffi                      \u001b[00m        21.3.0  pyhd8ed1ab_0            conda-forge/noarch                  15 KB\n",
            "\u001b[32m  argon2-cffi-bindings             \u001b[00m        21.2.0  py38h0a891b7_2          conda-forge/linux-64                34 KB\n",
            "\u001b[32m  asttokens                        \u001b[00m         2.0.5  pyhd8ed1ab_0            conda-forge/noarch                  21 KB\n",
            "\u001b[32m  attrs                            \u001b[00m        21.4.0  pyhd8ed1ab_0            conda-forge/noarch                  49 KB\n",
            "\u001b[32m  backcall                         \u001b[00m         0.2.0  pyh9f0ad1d_0            conda-forge/noarch                  13 KB\n",
            "\u001b[32m  backports                        \u001b[00m           1.0  py_2                    conda-forge/noarch                   4 KB\n",
            "\u001b[32m  backports.functools_lru_cache    \u001b[00m         1.6.4  pyhd8ed1ab_0            conda-forge/noarch                   9 KB\n",
            "\u001b[32m  beautifulsoup4                   \u001b[00m        4.11.0  pyha770c72_0            conda-forge/noarch                  58 KB\n",
            "\u001b[32m  bibtexparser                     \u001b[00m         1.1.0  py_0                    conda-forge/noarch                  32 KB\n",
            "\u001b[32m  binutils_impl_linux-64           \u001b[00m        2.36.1  h193b22a_2              conda-forge/linux-64                10 MB\n",
            "\u001b[32m  binutils_linux-64                \u001b[00m          2.36  hf3e587d_9              conda-forge/linux-64                24 KB\n",
            "\u001b[32m  bioconductor-biobase             \u001b[00m        2.54.0  r41h5c21468_1           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-biocgenerics        \u001b[00m        0.40.0  r41hdfd78af_0           bioconda/noarch                    631 KB\n",
            "\u001b[32m  bioconductor-biocparallel        \u001b[00m        1.28.3  r41h619a076_0           bioconda/linux-64                    1 MB\n",
            "\u001b[32m  bioconductor-biostrings          \u001b[00m        2.62.0  r41h5c21468_1           bioconda/linux-64                   14 MB\n",
            "\u001b[32m  bioconductor-dada2               \u001b[00m        1.22.0  r41h619a076_1           bioconda/linux-64                    3 MB\n",
            "\u001b[32m  bioconductor-delayedarray        \u001b[00m        0.20.0  r41h5c21468_1           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-genomeinfodb        \u001b[00m        1.30.0  r41hdfd78af_0           bioconda/noarch                      4 MB\n",
            "\u001b[32m  bioconductor-genomeinfodbdata    \u001b[00m         1.2.7  r41hdfd78af_1           bioconda/noarch                      7 KB\n",
            "\u001b[32m  bioconductor-genomicalignments   \u001b[00m        1.30.0  r41h5c21468_1           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-genomicranges       \u001b[00m        1.46.1  r41h5c21468_0           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-iranges             \u001b[00m        2.28.0  r41h5c21468_1           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-matrixgenerics      \u001b[00m         1.6.0  r41hdfd78af_0           bioconda/noarch                    342 KB\n",
            "\u001b[32m  bioconductor-rhtslib             \u001b[00m        1.26.0  r41h5c21468_1           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-rsamtools           \u001b[00m        2.10.0  r41h619a076_1           bioconda/linux-64                    4 MB\n",
            "\u001b[32m  bioconductor-s4vectors           \u001b[00m        0.32.3  r41h5c21468_0           bioconda/linux-64                    2 MB\n",
            "\u001b[32m  bioconductor-shortread           \u001b[00m        1.52.0  r41h619a076_1           bioconda/linux-64                    5 MB\n",
            "\u001b[32m  bioconductor-summarizedexperiment\u001b[00m        1.24.0  r41hdfd78af_0           bioconda/noarch                      3 MB\n",
            "\u001b[32m  bioconductor-xvector             \u001b[00m        0.34.0  r41h5c21468_1           bioconda/linux-64                  728 KB\n",
            "\u001b[32m  bioconductor-zlibbioc            \u001b[00m        1.40.0  r41h5c21468_1           bioconda/linux-64                  114 KB\n",
            "\u001b[32m  biom-format                      \u001b[00m        2.1.10  py38h6c62de6_1          conda-forge/linux-64                10 MB\n",
            "\u001b[32m  blast                            \u001b[00m        2.12.0  hf3cf87c_4              bioconda/linux-64                   22 MB\n",
            "\u001b[32m  bleach                           \u001b[00m         5.0.0  pyhd8ed1ab_0            conda-forge/noarch                 123 KB\n",
            "\u001b[32m  bokeh                            \u001b[00m         2.4.2  py38h578d9bd_0          conda-forge/linux-64                14 MB\n",
            "\u001b[32m  bowtie2                          \u001b[00m         2.4.5  py38hfbc8389_2          bioconda/linux-64                   14 MB\n",
            "\u001b[32m  brotli                           \u001b[00m         1.0.9  h166bdaf_7              conda-forge/linux-64                18 KB\n",
            "\u001b[32m  brotli-bin                       \u001b[00m         1.0.9  h166bdaf_7              conda-forge/linux-64                19 KB\n",
            "\u001b[32m  bwidget                          \u001b[00m        1.9.14  ha770c72_1              conda-forge/linux-64               120 KB\n",
            "\u001b[32m  cachecontrol                     \u001b[00m       0.12.10  pyhd8ed1ab_0            conda-forge/noarch                  23 KB\n",
            "\u001b[32m  cached-property                  \u001b[00m         1.5.2  hd8ed1ab_1              conda-forge/noarch                   4 KB\n",
            "\u001b[32m  cached_property                  \u001b[00m         1.5.2  pyha770c72_1            conda-forge/noarch                  11 KB\n",
            "\u001b[32m  cairo                            \u001b[00m        1.16.0  ha12eb4b_1010           conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  charset-normalizer               \u001b[00m        2.0.12  pyhd8ed1ab_0            conda-forge/noarch                  35 KB\n",
            "\u001b[32m  click                            \u001b[00m         7.1.2  pyh9f0ad1d_0            conda-forge/noarch                  64 KB\n",
            "\u001b[32m  colorama                         \u001b[00m         0.4.4  pyh9f0ad1d_0            conda-forge/noarch                  18 KB\n",
            "\u001b[32m  curl                             \u001b[00m        7.82.0  h7bff187_0              conda-forge/linux-64                90 KB\n",
            "\u001b[32m  cutadapt                         \u001b[00m           3.7  py38hbff2b2d_1          bioconda/linux-64                  201 KB\n",
            "\u001b[32m  cycler                           \u001b[00m        0.11.0  pyhd8ed1ab_0            conda-forge/noarch                  10 KB\n",
            "\u001b[32m  cython                           \u001b[00m       0.29.28  py38hfa26641_2          conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  dbus                             \u001b[00m        1.13.6  h5008d03_3              conda-forge/linux-64               604 KB\n",
            "\u001b[32m  deblur                           \u001b[00m         1.1.0  py_2                    bioconda/noarch                      2 MB\n",
            "\u001b[32m  debugpy                          \u001b[00m         1.5.1  py38hfa26641_1          conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  decorator                        \u001b[00m         4.4.2  py_0                    conda-forge/noarch                  11 KB\n",
            "\u001b[32m  defusedxml                       \u001b[00m         0.7.1  pyhd8ed1ab_0            conda-forge/noarch                  23 KB\n",
            "\u001b[32m  dendropy                         \u001b[00m         4.5.2  pyh3252c3a_0            bioconda/noarch                    308 KB\n",
            "\u001b[32m  dill                             \u001b[00m         0.3.4  pyhd8ed1ab_0            conda-forge/noarch                  62 KB\n",
            "\u001b[32m  dnaio                            \u001b[00m         0.8.1  py38hbff2b2d_0          bioconda/linux-64                   76 KB\n",
            "\u001b[32m  emperor                          \u001b[00m         1.0.3  py38h578d9bd_0          conda-forge/linux-64               625 KB\n",
            "\u001b[32m  entrez-direct                    \u001b[00m          16.2  he881be0_0              bioconda/linux-64                    7 MB\n",
            "\u001b[32m  entrypoints                      \u001b[00m           0.4  pyhd8ed1ab_0            conda-forge/noarch                   9 KB\n",
            "\u001b[32m  executing                        \u001b[00m         0.8.3  pyhd8ed1ab_0            conda-forge/noarch                  18 KB\n",
            "\u001b[32m  expat                            \u001b[00m         2.4.8  h27087fc_0              conda-forge/linux-64               187 KB\n",
            "\u001b[32m  fastcluster                      \u001b[00m         1.2.6  py38h56ea1f7_0          conda-forge/linux-64                48 KB\n",
            "\u001b[32m  fasttree                         \u001b[00m        2.1.10  0                       bioconda/linux-64                  477 KB\n",
            "\u001b[32m  flit-core                        \u001b[00m         3.7.1  pyhd8ed1ab_0            conda-forge/noarch                  44 KB\n",
            "\u001b[32m  font-ttf-dejavu-sans-mono        \u001b[00m          2.37  hab24e00_0              conda-forge/noarch                 388 KB\n",
            "\u001b[32m  font-ttf-inconsolata             \u001b[00m         3.000  h77eed37_0              conda-forge/noarch                  94 KB\n",
            "\u001b[32m  font-ttf-source-code-pro         \u001b[00m         2.038  h77eed37_0              conda-forge/noarch                 684 KB\n",
            "\u001b[32m  font-ttf-ubuntu                  \u001b[00m          0.83  hab24e00_0              conda-forge/noarch                   2 MB\n",
            "\u001b[32m  fontconfig                       \u001b[00m        2.14.0  h8e229c2_0              conda-forge/linux-64               305 KB\n",
            "\u001b[32m  fonts-conda-ecosystem            \u001b[00m             1  0                       conda-forge/noarch                   4 KB\n",
            "\u001b[32m  fonts-conda-forge                \u001b[00m             1  0                       conda-forge/noarch                   4 KB\n",
            "\u001b[32m  fonttools                        \u001b[00m        4.31.2  py38h0a891b7_1          conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  freetype                         \u001b[00m        2.10.4  h0708190_1              conda-forge/linux-64               890 KB\n",
            "\u001b[32m  fribidi                          \u001b[00m        1.0.10  h516909a_0              conda-forge/linux-64               113 KB\n",
            "\u001b[32m  future                           \u001b[00m        0.18.2  py38h578d9bd_5          conda-forge/linux-64               716 KB\n",
            "\u001b[32m  gcc_impl_linux-64                \u001b[00m        10.3.0  hf2f2afa_15             conda-forge/linux-64                46 MB\n",
            "\u001b[32m  gcc_linux-64                     \u001b[00m        10.3.0  hc39de41_9              conda-forge/linux-64                25 KB\n",
            "\u001b[32m  gettext                          \u001b[00m      0.19.8.1  h73d1719_1008           conda-forge/linux-64                 4 MB\n",
            "\u001b[32m  gfortran_impl_linux-64           \u001b[00m        10.3.0  h73f4979_15             conda-forge/linux-64                15 MB\n",
            "\u001b[32m  gfortran_linux-64                \u001b[00m        10.3.0  hb09a455_9              conda-forge/linux-64                24 KB\n",
            "\u001b[32m  giflib                           \u001b[00m         5.2.1  h516909a_2              conda-forge/linux-64                80 KB\n",
            "\u001b[32m  gneiss                           \u001b[00m         0.4.6  py_0                    bioconda/noarch                     41 KB\n",
            "\u001b[32m  graphite2                        \u001b[00m        1.3.13  he1b5a44_1001           conda-forge/linux-64               102 KB\n",
            "\u001b[32m  gsl                              \u001b[00m         2.7.1  hd82f3ee_0              pkgs/main/linux-64                   2 MB\n",
            "\u001b[32m  gst-plugins-base                 \u001b[00m        1.20.1  hcf0ee16_1              conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  gstreamer                        \u001b[00m        1.20.1  hd4edc92_1              conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  gxx_impl_linux-64                \u001b[00m        10.3.0  hf2f2afa_15             conda-forge/linux-64                11 MB\n",
            "\u001b[32m  gxx_linux-64                     \u001b[00m        10.3.0  h2593f52_9              conda-forge/linux-64                24 KB\n",
            "\u001b[32m  h5py                             \u001b[00m         3.6.0  nompi_py38hfbb2109_100  conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  harfbuzz                         \u001b[00m         4.2.0  h40b6f09_0              conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  hdf5                             \u001b[00m        1.12.1  nompi_h2386368_104      conda-forge/linux-64                 4 MB\n",
            "\u001b[32m  hdmedians                        \u001b[00m        0.14.2  py38h6c62de6_1          conda-forge/linux-64               161 KB\n",
            "\u001b[32m  hmmer                            \u001b[00m         3.1b2  3                       bioconda/linux-64                    6 MB\n",
            "\u001b[32m  htslib                           \u001b[00m        1.15.1  h9753748_0              bioconda/linux-64                    2 MB\n",
            "\u001b[32m  ijson                            \u001b[00m         3.1.3  pyhd3deb0d_0            conda-forge/noarch                  24 KB\n",
            "\u001b[32m  importlib-metadata               \u001b[00m        4.11.3  py38h578d9bd_1          conda-forge/linux-64                33 KB\n",
            "\u001b[32m  importlib_metadata               \u001b[00m        4.11.3  hd8ed1ab_1              conda-forge/noarch                   4 KB\n",
            "\u001b[32m  importlib_resources              \u001b[00m         5.6.0  pyhd8ed1ab_1            conda-forge/noarch                  22 KB\n",
            "\u001b[32m  iniconfig                        \u001b[00m         1.1.1  pyh9f0ad1d_0            conda-forge/noarch                   8 KB\n",
            "\u001b[32m  ipykernel                        \u001b[00m        6.12.1  py38h7f3c49e_0          conda-forge/linux-64               184 KB\n",
            "\u001b[32m  ipython                          \u001b[00m         8.2.0  py38h578d9bd_0          conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  ipython_genutils                 \u001b[00m         0.2.0  py_1                    conda-forge/noarch                  21 KB\n",
            "\u001b[32m  ipywidgets                       \u001b[00m         7.7.0  pyhd8ed1ab_0            conda-forge/noarch                 103 KB\n",
            "\u001b[32m  iqtree                           \u001b[00m    2.2.0_beta  hb97b32f_1              bioconda/linux-64                    7 MB\n",
            "\u001b[32m  isa-l                            \u001b[00m        2.30.0  ha770c72_4              conda-forge/linux-64               192 KB\n",
            "\u001b[32m  jbig                             \u001b[00m           2.1  h7f98852_2003           conda-forge/linux-64                43 KB\n",
            "\u001b[32m  jedi                             \u001b[00m        0.18.1  py38h578d9bd_1          conda-forge/linux-64               999 KB\n",
            "\u001b[32m  jinja2                           \u001b[00m         3.1.1  pyhd8ed1ab_0            conda-forge/noarch                  99 KB\n",
            "\u001b[32m  joblib                           \u001b[00m         1.1.0  pyhd8ed1ab_0            conda-forge/noarch                 210 KB\n",
            "\u001b[32m  jpeg                             \u001b[00m            9e  h7f98852_0              conda-forge/linux-64               269 KB\n",
            "\u001b[32m  jq                               \u001b[00m           1.6  h36c2ea0_1000           conda-forge/linux-64               346 KB\n",
            "\u001b[32m  jsonschema                       \u001b[00m         4.4.0  pyhd8ed1ab_0            conda-forge/noarch                  57 KB\n",
            "\u001b[32m  jupyter_client                   \u001b[00m         7.2.2  pyhd8ed1ab_1            conda-forge/noarch                  90 KB\n",
            "\u001b[32m  jupyter_core                     \u001b[00m         4.9.2  py38h578d9bd_0          conda-forge/linux-64                81 KB\n",
            "\u001b[32m  jupyterlab_pygments              \u001b[00m         0.2.0  pyhd8ed1ab_0            conda-forge/noarch                  17 KB\n",
            "\u001b[32m  jupyterlab_widgets               \u001b[00m         1.1.0  pyhd8ed1ab_0            conda-forge/noarch                 133 KB\n",
            "\u001b[32m  kernel-headers_linux-64          \u001b[00m        2.6.32  he073ed8_15             conda-forge/noarch                 707 KB\n",
            "\u001b[32m  keyutils                         \u001b[00m         1.6.1  h166bdaf_0              conda-forge/linux-64               115 KB\n",
            "\u001b[32m  kiwisolver                       \u001b[00m         1.4.2  py38h43d8883_1          conda-forge/linux-64                75 KB\n",
            "\u001b[32m  lcms2                            \u001b[00m          2.12  hddcbb42_0              conda-forge/linux-64               443 KB\n",
            "\u001b[32m  lerc                             \u001b[00m           3.0  h9c3ff4c_0              conda-forge/linux-64               216 KB\n",
            "\u001b[32m  libblas                          \u001b[00m         3.9.0  14_linux64_openblas     conda-forge/linux-64                12 KB\n",
            "\u001b[32m  libbrotlicommon                  \u001b[00m         1.0.9  h166bdaf_7              conda-forge/linux-64                65 KB\n",
            "\u001b[32m  libbrotlidec                     \u001b[00m         1.0.9  h166bdaf_7              conda-forge/linux-64                33 KB\n",
            "\u001b[32m  libbrotlienc                     \u001b[00m         1.0.9  h166bdaf_7              conda-forge/linux-64               287 KB\n",
            "\u001b[32m  libcblas                         \u001b[00m         3.9.0  14_linux64_openblas     conda-forge/linux-64                12 KB\n",
            "\u001b[32m  libclang                         \u001b[00m        13.0.1  default_hc23dcda_0      conda-forge/linux-64                11 MB\n",
            "\u001b[32m  libdeflate                       \u001b[00m          1.10  h7f98852_0              conda-forge/linux-64                77 KB\n",
            "\u001b[32m  libevent                         \u001b[00m        2.1.10  h9b69904_4              conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  libgcc                           \u001b[00m         7.2.0  h69d50b8_2              conda-forge/linux-64               304 KB\n",
            "\u001b[32m  libgcc-devel_linux-64            \u001b[00m        10.3.0  he6cfe16_15             conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  libgfortran-ng                   \u001b[00m        11.2.0  h69a702a_15             conda-forge/linux-64                23 KB\n",
            "\u001b[32m  libgfortran5                     \u001b[00m        11.2.0  h5c6108e_15             conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  libglib                          \u001b[00m        2.70.2  h174f98d_4              conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  liblapack                        \u001b[00m         3.9.0  14_linux64_openblas     conda-forge/linux-64                12 KB\n",
            "\u001b[32m  libllvm10                        \u001b[00m        10.0.1  he513fc3_3              conda-forge/linux-64                26 MB\n",
            "\u001b[32m  libllvm13                        \u001b[00m        13.0.1  hf817b99_2              conda-forge/linux-64                34 MB\n",
            "\u001b[32m  libnsl                           \u001b[00m         2.0.0  h7f98852_0              conda-forge/linux-64                31 KB\n",
            "\u001b[32m  libogg                           \u001b[00m         1.3.4  h7f98852_1              conda-forge/linux-64               206 KB\n",
            "\u001b[32m  libopenblas                      \u001b[00m        0.3.20  pthreads_h78a6416_0     conda-forge/linux-64                10 MB\n",
            "\u001b[32m  libopus                          \u001b[00m         1.3.1  h7f98852_1              conda-forge/linux-64               255 KB\n",
            "\u001b[32m  libpng                           \u001b[00m        1.6.37  hed695b0_2              conda-forge/linux-64               359 KB\n",
            "\u001b[32m  libpq                            \u001b[00m          14.2  hd57d9b9_0              conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  libsanitizer                     \u001b[00m        10.3.0  h26c7422_15             conda-forge/linux-64                 6 MB\n",
            "\u001b[32m  libsodium                        \u001b[00m        1.0.18  h516909a_1              conda-forge/linux-64               366 KB\n",
            "\u001b[32m  libstdcxx-devel_linux-64         \u001b[00m        10.3.0  he6cfe16_15             conda-forge/linux-64                10 MB\n",
            "\u001b[32m  libtiff                          \u001b[00m         4.3.0  h542a066_3              conda-forge/linux-64               638 KB\n",
            "\u001b[32m  libuuid                          \u001b[00m        2.32.1  h14c3975_1000           conda-forge/linux-64                26 KB\n",
            "\u001b[32m  libvorbis                        \u001b[00m         1.3.7  he1b5a44_0              conda-forge/linux-64               287 KB\n",
            "\u001b[32m  libwebp                          \u001b[00m         1.2.2  h3452ae3_0              conda-forge/linux-64                85 KB\n",
            "\u001b[32m  libwebp-base                     \u001b[00m         1.2.2  h7f98852_1              conda-forge/linux-64               824 KB\n",
            "\u001b[32m  libxcb                           \u001b[00m          1.13  h7f98852_1004           conda-forge/linux-64               391 KB\n",
            "\u001b[32m  libxkbcommon                     \u001b[00m         1.0.3  he3ba5ed_0              conda-forge/linux-64               581 KB\n",
            "\u001b[32m  libxslt                          \u001b[00m        1.1.33  h0ef7038_3              conda-forge/linux-64               521 KB\n",
            "\u001b[32m  libzlib                          \u001b[00m        1.2.11  h166bdaf_1014           conda-forge/linux-64                60 KB\n",
            "\u001b[32m  llvmlite                         \u001b[00m        0.36.0  py38h4630a5e_0          conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  lockfile                         \u001b[00m        0.12.2  py_1                    conda-forge/noarch                  11 KB\n",
            "\u001b[32m  lxml                             \u001b[00m         4.8.0  py38h0a891b7_2          conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  lz4                              \u001b[00m         4.0.0  py38h1bf946c_1          conda-forge/linux-64                36 KB\n",
            "\u001b[32m  mafft                            \u001b[00m         7.490  hec16e2b_1              bioconda/linux-64                    3 MB\n",
            "\u001b[32m  make                             \u001b[00m           4.3  hd18ef5c_1              conda-forge/linux-64               507 KB\n",
            "\u001b[32m  markupsafe                       \u001b[00m         2.1.1  py38h0a891b7_1          conda-forge/linux-64                22 KB\n",
            "\u001b[32m  matplotlib                       \u001b[00m         3.5.1  py38h578d9bd_0          conda-forge/linux-64                 6 KB\n",
            "\u001b[32m  matplotlib-base                  \u001b[00m         3.5.1  py38hf4fb855_0          conda-forge/linux-64                 7 MB\n",
            "\u001b[32m  matplotlib-inline                \u001b[00m         0.1.3  pyhd8ed1ab_0            conda-forge/noarch                  11 KB\n",
            "\u001b[32m  mistune                          \u001b[00m         0.8.4  py38h497a2fe_1005       conda-forge/linux-64                54 KB\n",
            "\u001b[32m  msgpack-python                   \u001b[00m         1.0.3  py38h43d8883_1          conda-forge/linux-64                90 KB\n",
            "\u001b[32m  munkres                          \u001b[00m         1.1.4  pyh9f0ad1d_0            conda-forge/noarch                  12 KB\n",
            "\u001b[32m  mysql-common                     \u001b[00m        8.0.28  haf5c9bc_3              conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  mysql-libs                       \u001b[00m        8.0.28  h28c427c_3              conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  natsort                          \u001b[00m         8.1.0  pyhd8ed1ab_0            conda-forge/noarch                  35 KB\n",
            "\u001b[32m  nbclient                         \u001b[00m        0.5.13  pyhd8ed1ab_0            conda-forge/noarch                  64 KB\n",
            "\u001b[32m  nbconvert                        \u001b[00m         6.4.5  pyhd8ed1ab_2            conda-forge/noarch                   6 KB\n",
            "\u001b[32m  nbconvert-core                   \u001b[00m         6.4.5  pyhd8ed1ab_2            conda-forge/noarch                 447 KB\n",
            "\u001b[32m  nbconvert-pandoc                 \u001b[00m         6.4.5  pyhd8ed1ab_2            conda-forge/noarch                   4 KB\n",
            "\u001b[32m  nbformat                         \u001b[00m         5.3.0  pyhd8ed1ab_0            conda-forge/noarch                 104 KB\n",
            "\u001b[32m  nest-asyncio                     \u001b[00m         1.5.5  pyhd8ed1ab_0            conda-forge/noarch                   9 KB\n",
            "\u001b[32m  networkx                         \u001b[00m         2.7.1  pyhd8ed1ab_0            conda-forge/noarch                   2 MB\n",
            "\u001b[32m  nose                             \u001b[00m         1.3.7  py38h32f6830_1004       conda-forge/linux-64               213 KB\n",
            "\u001b[32m  notebook                         \u001b[00m        6.4.10  pyha770c72_0            conda-forge/noarch                   6 MB\n",
            "\u001b[32m  nspr                             \u001b[00m          4.32  h9c3ff4c_1              conda-forge/linux-64               233 KB\n",
            "\u001b[32m  nss                              \u001b[00m          3.77  h2350873_0              conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  numba                            \u001b[00m        0.53.1  py38h8b71fd7_1          conda-forge/linux-64                 4 MB\n",
            "\u001b[32m  numpy                            \u001b[00m        1.22.3  py38h1d589f8_2          conda-forge/linux-64                 7 MB\n",
            "\u001b[32m  oniguruma                        \u001b[00m       6.9.7.1  h7f98852_0              conda-forge/linux-64               409 KB\n",
            "\u001b[32m  openjdk                          \u001b[00m        11.0.1  h516909a_1016           conda-forge/linux-64               176 MB\n",
            "\u001b[32m  openjpeg                         \u001b[00m         2.4.0  hb52868f_1              conda-forge/linux-64               444 KB\n",
            "\u001b[32m  packaging                        \u001b[00m          21.3  pyhd8ed1ab_0            conda-forge/noarch                  36 KB\n",
            "\u001b[32m  pandas                           \u001b[00m         1.2.5  py38h1abd341_0          conda-forge/linux-64                12 MB\n",
            "\u001b[32m  pandoc                           \u001b[00m      2.17.1.1  ha770c72_0              conda-forge/linux-64                12 MB\n",
            "\u001b[32m  pandocfilters                    \u001b[00m         1.5.0  pyhd8ed1ab_0            conda-forge/noarch                  11 KB\n",
            "\u001b[32m  pango                            \u001b[00m        1.50.6  hbd2fdc8_0              conda-forge/linux-64               456 KB\n",
            "\u001b[32m  parso                            \u001b[00m         0.8.3  pyhd8ed1ab_0            conda-forge/noarch                  69 KB\n",
            "\u001b[32m  patsy                            \u001b[00m         0.5.2  pyhd8ed1ab_0            conda-forge/noarch                 188 KB\n",
            "\u001b[32m  pbzip2                           \u001b[00m        1.1.13  0                       conda-forge/linux-64               114 KB\n",
            "\u001b[32m  pcre                             \u001b[00m          8.45  h9c3ff4c_0              conda-forge/linux-64               253 KB\n",
            "\u001b[32m  pcre2                            \u001b[00m         10.37  h032f7d1_0              conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  perl                             \u001b[00m        5.32.1  2_h7f98852_perl5        conda-forge/linux-64                14 MB\n",
            "\u001b[32m  perl-archive-tar                 \u001b[00m          2.40  pl5321hdfd78af_0        bioconda/noarch                     33 KB\n",
            "\u001b[32m  perl-carp                        \u001b[00m          1.50  pl5321hd8ed1ab_0        conda-forge/noarch                  22 KB\n",
            "\u001b[32m  perl-common-sense                \u001b[00m          3.75  pl5321hdfd78af_0        bioconda/noarch                     13 KB\n",
            "\u001b[32m  perl-compress-raw-bzip2          \u001b[00m         2.103  pl5321h87f3376_0        bioconda/linux-64                   47 KB\n",
            "\u001b[32m  perl-compress-raw-zlib           \u001b[00m         2.103  pl5321h87f3376_0        bioconda/linux-64                   75 KB\n",
            "\u001b[32m  perl-encode                      \u001b[00m          3.17  pl5321hec16e2b_0        bioconda/linux-64                    2 MB\n",
            "\u001b[32m  perl-exporter                    \u001b[00m          5.74  pl5321hd8ed1ab_0        conda-forge/noarch                  19 KB\n",
            "\u001b[32m  perl-exporter-tiny               \u001b[00m      1.002002  pl5321hdfd78af_0        bioconda/noarch                     25 KB\n",
            "\u001b[32m  perl-extutils-makemaker          \u001b[00m          7.64  pl5321hd8ed1ab_0        conda-forge/noarch                 145 KB\n",
            "\u001b[32m  perl-io-compress                 \u001b[00m         2.102  pl5321h87f3376_1        bioconda/linux-64                   83 KB\n",
            "\u001b[32m  perl-io-zlib                     \u001b[00m          1.11  pl5321hdfd78af_0        bioconda/noarch                     11 KB\n",
            "\u001b[32m  perl-json                        \u001b[00m          4.05  pl5321hdfd78af_0        bioconda/noarch                     54 KB\n",
            "\u001b[32m  perl-json-xs                     \u001b[00m          2.34  pl5321h9f5acd7_5        bioconda/linux-64                   64 KB\n",
            "\u001b[32m  perl-list-moreutils              \u001b[00m         0.430  pl5321hdfd78af_0        bioconda/noarch                     32 KB\n",
            "\u001b[32m  perl-list-moreutils-xs           \u001b[00m         0.430  pl5321hec16e2b_1        bioconda/linux-64                   49 KB\n",
            "\u001b[32m  perl-parent                      \u001b[00m         0.238  pl5321hd8ed1ab_0        conda-forge/noarch                  11 KB\n",
            "\u001b[32m  perl-pathtools                   \u001b[00m          3.75  pl5321hec16e2b_3        bioconda/linux-64                   42 KB\n",
            "\u001b[32m  perl-scalar-list-utils           \u001b[00m          1.62  pl5321hec16e2b_0        bioconda/linux-64                   43 KB\n",
            "\u001b[32m  perl-types-serialiser            \u001b[00m          1.01  pl5321hdfd78af_0        bioconda/noarch                     13 KB\n",
            "\u001b[32m  pexpect                          \u001b[00m         4.8.0  py38h32f6830_1          conda-forge/linux-64                80 KB\n",
            "\u001b[32m  pickleshare                      \u001b[00m         0.7.5  py38h32f6830_1002       conda-forge/linux-64                13 KB\n",
            "\u001b[32m  pigz                             \u001b[00m           2.6  h27826a3_0              conda-forge/linux-64                87 KB\n",
            "\u001b[32m  pillow                           \u001b[00m         9.1.0  py38h0ee0e06_0          conda-forge/linux-64                45 MB\n",
            "\u001b[32m  pixman                           \u001b[00m        0.40.0  h36c2ea0_0              conda-forge/linux-64               627 KB\n",
            "\u001b[32m  pluggy                           \u001b[00m         1.0.0  py38h578d9bd_3          conda-forge/linux-64                25 KB\n",
            "\u001b[32m  prometheus_client                \u001b[00m        0.14.0  pyhd8ed1ab_0            conda-forge/noarch                  49 KB\n",
            "\u001b[32m  prompt-toolkit                   \u001b[00m        3.0.29  pyha770c72_0            conda-forge/noarch                 252 KB\n",
            "\u001b[32m  psutil                           \u001b[00m         5.9.0  py38h0a891b7_1          conda-forge/linux-64               346 KB\n",
            "\u001b[32m  pthread-stubs                    \u001b[00m           0.4  h36c2ea0_1001           conda-forge/linux-64                 5 KB\n",
            "\u001b[32m  ptyprocess                       \u001b[00m         0.7.0  pyhd3deb0d_0            conda-forge/noarch                  16 KB\n",
            "\u001b[32m  pure_eval                        \u001b[00m         0.2.2  pyhd8ed1ab_0            conda-forge/noarch                  14 KB\n",
            "\u001b[32m  py                               \u001b[00m        1.11.0  pyh6c4a22f_0            conda-forge/noarch                  74 KB\n",
            "\u001b[32m  pygments                         \u001b[00m        2.11.2  pyhd8ed1ab_0            conda-forge/noarch                 796 KB\n",
            "\u001b[32m  pynndescent                      \u001b[00m         0.5.6  pyh6c4a22f_0            conda-forge/noarch                  45 KB\n",
            "\u001b[32m  pyparsing                        \u001b[00m         3.0.7  pyhd8ed1ab_0            conda-forge/noarch                  79 KB\n",
            "\u001b[32m  pyqt                             \u001b[00m        5.12.3  py38h578d9bd_8          conda-forge/linux-64                22 KB\n",
            "\u001b[32m  pyqt-impl                        \u001b[00m        5.12.3  py38h0ffb2e6_8          conda-forge/linux-64                 6 MB\n",
            "\u001b[32m  pyqt5-sip                        \u001b[00m       4.19.18  py38h709712a_8          conda-forge/linux-64               311 KB\n",
            "\u001b[32m  pyqtchart                        \u001b[00m          5.12  py38h7400c14_8          conda-forge/linux-64               257 KB\n",
            "\u001b[32m  pyqtwebengine                    \u001b[00m        5.12.1  py38h7400c14_8          conda-forge/linux-64               175 KB\n",
            "\u001b[32m  pyrsistent                       \u001b[00m        0.18.1  py38h0a891b7_1          conda-forge/linux-64                92 KB\n",
            "\u001b[32m  pytest                           \u001b[00m         7.1.1  py38h578d9bd_1          conda-forge/linux-64               459 KB\n",
            "\u001b[32m  python-dateutil                  \u001b[00m         2.8.2  pyhd8ed1ab_0            conda-forge/noarch                 240 KB\n",
            "\u001b[32m  python-fastjsonschema            \u001b[00m        2.15.3  pyhd8ed1ab_0            conda-forge/noarch                 243 KB\n",
            "\u001b[32m  python-isal                      \u001b[00m        0.11.1  py38h497a2fe_1          conda-forge/linux-64               141 KB\n",
            "\u001b[32m  pytz                             \u001b[00m        2022.1  pyhd8ed1ab_0            conda-forge/noarch                 242 KB\n",
            "\u001b[32m  pyyaml                           \u001b[00m           6.0  py38h0a891b7_4          conda-forge/linux-64               182 KB\n",
            "\u001b[32m  pyzmq                            \u001b[00m        22.3.0  py38hfc09fa9_2          conda-forge/linux-64               513 KB\n",
            "\u001b[32m  q2-alignment                     \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       20 KB\n",
            "\u001b[32m  q2-composition                   \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      170 KB\n",
            "\u001b[32m  q2-cutadapt                      \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64        6 MB\n",
            "\u001b[32m  q2-dada2                         \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64        2 MB\n",
            "\u001b[32m  q2-deblur                        \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      271 KB\n",
            "\u001b[32m  q2-demux                         \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      192 KB\n",
            "\u001b[32m  q2-diversity                     \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      259 KB\n",
            "\u001b[32m  q2-diversity-lib                 \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       88 KB\n",
            "\u001b[32m  q2-emperor                       \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       15 KB\n",
            "\u001b[32m  q2-feature-classifier            \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      141 KB\n",
            "\u001b[32m  q2-feature-table                 \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      230 KB\n",
            "\u001b[32m  q2-fragment-insertion            \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       31 KB\n",
            "\u001b[32m  q2-gneiss                        \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       54 KB\n",
            "\u001b[32m  q2-longitudinal                  \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64        8 MB\n",
            "\u001b[32m  q2-metadata                      \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      970 KB\n",
            "\u001b[32m  q2-mystery-stew                  \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       27 KB\n",
            "\u001b[32m  q2-phylogeny                     \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       51 KB\n",
            "\u001b[32m  q2-quality-control               \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      286 KB\n",
            "\u001b[32m  q2-quality-filter                \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       32 KB\n",
            "\u001b[32m  q2-sample-classifier             \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      627 KB\n",
            "\u001b[32m  q2-taxa                          \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      110 KB\n",
            "\u001b[32m  q2-types                         \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      132 KB\n",
            "\u001b[32m  q2-vsearch                       \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      619 KB\n",
            "\u001b[32m  q2cli                            \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       88 KB\n",
            "\u001b[32m  q2galaxy                         \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64       53 KB\n",
            "\u001b[32m  q2templates                      \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      189 KB\n",
            "\u001b[32m  qiime2                           \u001b[00m      2022.2.0  py38_0                  qiime2/label/r2022.2/linux-64      353 KB\n",
            "\u001b[32m  qt                               \u001b[00m        5.12.9  h1304e3e_6              conda-forge/linux-64                99 MB\n",
            "\u001b[32m  r-backports                      \u001b[00m         1.4.1  r41hcfec24a_0           conda-forge/linux-64               110 KB\n",
            "\u001b[32m  r-base                           \u001b[00m         4.1.3  hd930d0e_0              conda-forge/linux-64                25 MB\n",
            "\u001b[32m  r-bh                             \u001b[00m      1.78.0_0  r41hc72bb7e_0           conda-forge/noarch                  12 MB\n",
            "\u001b[32m  r-bitops                         \u001b[00m         1.0_7  r41hcfec24a_0           conda-forge/linux-64                44 KB\n",
            "\u001b[32m  r-brio                           \u001b[00m         1.1.3  r41hcfec24a_0           conda-forge/linux-64                40 KB\n",
            "\u001b[32m  r-callr                          \u001b[00m         3.7.0  r41hc72bb7e_0           conda-forge/noarch                 439 KB\n",
            "\u001b[32m  r-cli                            \u001b[00m         3.2.0  r41h03ef668_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-cluster                        \u001b[00m         2.1.3  r41h8da6f51_0           conda-forge/linux-64               572 KB\n",
            "\u001b[32m  r-colorspace                     \u001b[00m         2.0_3  r41h06615bd_0           conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  r-crayon                         \u001b[00m         1.5.1  r41hc72bb7e_0           conda-forge/noarch                 165 KB\n",
            "\u001b[32m  r-desc                           \u001b[00m         1.4.1  r41hc72bb7e_0           conda-forge/noarch                 332 KB\n",
            "\u001b[32m  r-diffobj                        \u001b[00m         0.3.5  r41hcfec24a_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-digest                         \u001b[00m        0.6.29  r41h03ef668_0           conda-forge/linux-64               208 KB\n",
            "\u001b[32m  r-ellipsis                       \u001b[00m         0.3.2  r41hcfec24a_0           conda-forge/linux-64                42 KB\n",
            "\u001b[32m  r-evaluate                       \u001b[00m          0.15  r41hc72bb7e_0           conda-forge/noarch                  84 KB\n",
            "\u001b[32m  r-fansi                          \u001b[00m         1.0.3  r41h06615bd_0           conda-forge/linux-64               323 KB\n",
            "\u001b[32m  r-farver                         \u001b[00m         2.1.0  r41h03ef668_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-formatr                        \u001b[00m          1.12  r41hc72bb7e_0           conda-forge/noarch                 170 KB\n",
            "\u001b[32m  r-futile.logger                  \u001b[00m         1.4.3  r41hc72bb7e_1003        conda-forge/noarch                 108 KB\n",
            "\u001b[32m  r-futile.options                 \u001b[00m         1.0.1  r41hc72bb7e_1002        conda-forge/noarch                  27 KB\n",
            "\u001b[32m  r-ggplot2                        \u001b[00m         3.3.5  r41hc72bb7e_0           conda-forge/noarch                   4 MB\n",
            "\u001b[32m  r-glue                           \u001b[00m         1.6.2  r41h06615bd_0           conda-forge/linux-64               154 KB\n",
            "\u001b[32m  r-gtable                         \u001b[00m         0.3.0  r41hc72bb7e_3           conda-forge/noarch                 423 KB\n",
            "\u001b[32m  r-hwriter                        \u001b[00m       1.3.2.1  r41hc72bb7e_0           conda-forge/noarch                 122 KB\n",
            "\u001b[32m  r-isoband                        \u001b[00m         0.2.5  r41h03ef668_0           conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  r-jpeg                           \u001b[00m         0.1_9  r41hcfec24a_0           conda-forge/linux-64                52 KB\n",
            "\u001b[32m  r-jsonlite                       \u001b[00m         1.8.0  r41h06615bd_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-labeling                       \u001b[00m         0.4.2  r41hc72bb7e_1           conda-forge/noarch                  67 KB\n",
            "\u001b[32m  r-lambda.r                       \u001b[00m         1.2.4  r41hc72bb7e_1           conda-forge/noarch                 120 KB\n",
            "\u001b[32m  r-lattice                        \u001b[00m       0.20_45  r41hcfec24a_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-latticeextra                   \u001b[00m        0.6_29  r41hc72bb7e_1           conda-forge/noarch                   2 MB\n",
            "\u001b[32m  r-lifecycle                      \u001b[00m         1.0.1  r41hc72bb7e_0           conda-forge/noarch                 109 KB\n",
            "\u001b[32m  r-magrittr                       \u001b[00m         2.0.3  r41h06615bd_0           conda-forge/linux-64               215 KB\n",
            "\u001b[32m  r-mass                           \u001b[00m        7.3_56  r41h06615bd_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-matrix                         \u001b[00m         1.4_1  r41h0154571_0           conda-forge/linux-64                 5 MB\n",
            "\u001b[32m  r-matrixstats                    \u001b[00m        0.61.0  r41hcfec24a_0           conda-forge/linux-64               468 KB\n",
            "\u001b[32m  r-mgcv                           \u001b[00m        1.8_40  r41h0154571_0           conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  r-munsell                        \u001b[00m         0.5.0  r41hc72bb7e_1004        conda-forge/noarch                 247 KB\n",
            "\u001b[32m  r-nlme                           \u001b[00m       3.1_157  r41h8da6f51_0           conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  r-permute                        \u001b[00m         0.9_7  r41hc72bb7e_0           conda-forge/noarch                 227 KB\n",
            "\u001b[32m  r-pillar                         \u001b[00m         1.7.0  r41hc72bb7e_0           conda-forge/noarch                 706 KB\n",
            "\u001b[32m  r-pkgconfig                      \u001b[00m         2.0.3  r41hc72bb7e_1           conda-forge/noarch                  25 KB\n",
            "\u001b[32m  r-pkgload                        \u001b[00m         1.2.4  r41h03ef668_0           conda-forge/linux-64               172 KB\n",
            "\u001b[32m  r-plyr                           \u001b[00m         1.8.7  r41h7525677_0           conda-forge/linux-64               836 KB\n",
            "\u001b[32m  r-png                            \u001b[00m         0.1_7  r41hcfec24a_1004        conda-forge/linux-64                58 KB\n",
            "\u001b[32m  r-praise                         \u001b[00m         1.0.0  r41hc72bb7e_1005        conda-forge/noarch                  24 KB\n",
            "\u001b[32m  r-processx                       \u001b[00m         3.5.3  r41h06615bd_0           conda-forge/linux-64               318 KB\n",
            "\u001b[32m  r-ps                             \u001b[00m         1.6.0  r41hcfec24a_0           conda-forge/linux-64               299 KB\n",
            "\u001b[32m  r-r6                             \u001b[00m         2.5.1  r41hc72bb7e_0           conda-forge/noarch                  89 KB\n",
            "\u001b[32m  r-rcolorbrewer                   \u001b[00m         1.1_3  r41h785f33e_0           conda-forge/noarch                  65 KB\n",
            "\u001b[32m  r-rcpp                           \u001b[00m       1.0.8.3  r41h7525677_0           conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  r-rcppparallel                   \u001b[00m         5.1.5  r41h03ef668_0           conda-forge/linux-64               568 KB\n",
            "\u001b[32m  r-rcurl                          \u001b[00m      1.98_1.6  r41hcfec24a_0           conda-forge/linux-64               967 KB\n",
            "\u001b[32m  r-rematch2                       \u001b[00m         2.1.2  r41hc72bb7e_1           conda-forge/noarch                  52 KB\n",
            "\u001b[32m  r-reshape2                       \u001b[00m         1.4.4  r41h03ef668_1           conda-forge/linux-64               128 KB\n",
            "\u001b[32m  r-rlang                          \u001b[00m         1.0.2  r41h7525677_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-rprojroot                      \u001b[00m         2.0.3  r41hc72bb7e_0           conda-forge/noarch                 115 KB\n",
            "\u001b[32m  r-rstudioapi                     \u001b[00m          0.13  r41hc72bb7e_0           conda-forge/noarch                 281 KB\n",
            "\u001b[32m  r-scales                         \u001b[00m         1.1.1  r41hc72bb7e_0           conda-forge/noarch                 557 KB\n",
            "\u001b[32m  r-snow                           \u001b[00m         0.4_4  r41hc72bb7e_0           conda-forge/noarch                 113 KB\n",
            "\u001b[32m  r-stringi                        \u001b[00m         1.7.6  r41h337692f_1           conda-forge/linux-64               896 KB\n",
            "\u001b[32m  r-stringr                        \u001b[00m         1.4.0  r41hc72bb7e_2           conda-forge/noarch                 210 KB\n",
            "\u001b[32m  r-testthat                       \u001b[00m         3.1.3  r41h7525677_0           conda-forge/linux-64                 2 MB\n",
            "\u001b[32m  r-tibble                         \u001b[00m         3.1.6  r41hcfec24a_0           conda-forge/linux-64               774 KB\n",
            "\u001b[32m  r-utf8                           \u001b[00m         1.2.2  r41hcfec24a_0           conda-forge/linux-64               162 KB\n",
            "\u001b[32m  r-vctrs                          \u001b[00m         0.4.0  r41h7525677_0           conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  r-vegan                          \u001b[00m         2.5_7  r41h52d45c5_0           conda-forge/linux-64                 3 MB\n",
            "\u001b[32m  r-viridislite                    \u001b[00m         0.4.0  r41hc72bb7e_0           conda-forge/noarch                   1 MB\n",
            "\u001b[32m  r-waldo                          \u001b[00m         0.4.0  r41hc72bb7e_0           conda-forge/noarch                 109 KB\n",
            "\u001b[32m  r-withr                          \u001b[00m         2.5.0  r41hc72bb7e_0           conda-forge/noarch                 240 KB\n",
            "\u001b[32m  raxml                            \u001b[00m        8.2.12  hec16e2b_4              bioconda/linux-64                    3 MB\n",
            "\u001b[32m  samtools                         \u001b[00m        1.15.1  h1170115_0              bioconda/linux-64                  393 KB\n",
            "\u001b[32m  scikit-bio                       \u001b[00m         0.5.6  py38h0b5ebd8_4          conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  scikit-learn                     \u001b[00m        0.24.1  py38h658cfdd_0          conda-forge/linux-64                 8 MB\n",
            "\u001b[32m  scipy                            \u001b[00m         1.8.0  py38h56a6a73_1          conda-forge/linux-64                24 MB\n",
            "\u001b[32m  seaborn                          \u001b[00m        0.11.2  hd8ed1ab_0              conda-forge/noarch                   4 KB\n",
            "\u001b[32m  seaborn-base                     \u001b[00m        0.11.2  pyhd8ed1ab_0            conda-forge/noarch                 223 KB\n",
            "\u001b[32m  sed                              \u001b[00m           4.8  he412f7d_0              conda-forge/linux-64               264 KB\n",
            "\u001b[32m  send2trash                       \u001b[00m         1.8.0  pyhd8ed1ab_0            conda-forge/noarch                  17 KB\n",
            "\u001b[32m  sepp                             \u001b[00m        4.3.10  py38h3252c3a_2          bioconda/linux-64                    7 MB\n",
            "\u001b[32m  sortmerna                        \u001b[00m           2.0  he860b03_4              bioconda/linux-64                  337 KB\n",
            "\u001b[32m  soupsieve                        \u001b[00m         2.3.1  pyhd8ed1ab_0            conda-forge/noarch                  33 KB\n",
            "\u001b[32m  stack_data                       \u001b[00m         0.2.0  pyhd8ed1ab_0            conda-forge/noarch                  21 KB\n",
            "\u001b[32m  statsmodels                      \u001b[00m        0.13.2  py38h6c62de6_0          conda-forge/linux-64                12 MB\n",
            "\u001b[32m  sysroot_linux-64                 \u001b[00m          2.12  he073ed8_15             conda-forge/noarch                  31 MB\n",
            "\u001b[32m  tbb                              \u001b[00m        2020.2  h4bd325d_4              conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  terminado                        \u001b[00m        0.13.3  py38h578d9bd_1          conda-forge/linux-64                27 KB\n",
            "\u001b[32m  testpath                         \u001b[00m         0.6.0  pyhd8ed1ab_0            conda-forge/noarch                  85 KB\n",
            "\u001b[32m  threadpoolctl                    \u001b[00m         3.1.0  pyh8a188c0_0            conda-forge/noarch                  18 KB\n",
            "\u001b[32m  tktable                          \u001b[00m          2.10  hb7b940f_3              conda-forge/linux-64                89 KB\n",
            "\u001b[32m  toml                             \u001b[00m        0.10.2  pyhd8ed1ab_0            conda-forge/noarch                  18 KB\n",
            "\u001b[32m  tomli                            \u001b[00m         2.0.1  pyhd8ed1ab_0            conda-forge/noarch                  16 KB\n",
            "\u001b[32m  tornado                          \u001b[00m           6.1  py38h0a891b7_3          conda-forge/linux-64               645 KB\n",
            "\u001b[32m  traitlets                        \u001b[00m         5.1.1  pyhd8ed1ab_0            conda-forge/noarch                  82 KB\n",
            "\u001b[32m  typing_extensions                \u001b[00m         4.1.1  pyha770c72_0            conda-forge/noarch                  29 KB\n",
            "\u001b[32m  tzlocal                          \u001b[00m           2.1  pyh9f0ad1d_0            conda-forge/noarch                  18 KB\n",
            "\u001b[32m  umap-learn                       \u001b[00m         0.5.2  py38h578d9bd_1          conda-forge/linux-64               131 KB\n",
            "\u001b[32m  unicodedata2                     \u001b[00m        14.0.0  py38h0a891b7_1          conda-forge/linux-64               498 KB\n",
            "\u001b[32m  unifrac                          \u001b[00m        0.20.3  py38hc37a69a_0          bioconda/linux-64                  461 KB\n",
            "\u001b[32m  vsearch                          \u001b[00m         2.7.0  1                       bioconda/linux-64                  713 KB\n",
            "\u001b[32m  wcwidth                          \u001b[00m         0.2.5  pyh9f0ad1d_2            conda-forge/noarch                  33 KB\n",
            "\u001b[32m  webencodings                     \u001b[00m         0.5.1  py_1                    conda-forge/noarch                  12 KB\n",
            "\u001b[32m  widgetsnbextension               \u001b[00m         3.6.0  py38h578d9bd_0          conda-forge/linux-64                 1 MB\n",
            "\u001b[32m  xmltodict                        \u001b[00m        0.12.0  py_0                    conda-forge/noarch                  11 KB\n",
            "\u001b[32m  xopen                            \u001b[00m         1.5.0  py38h578d9bd_0          conda-forge/linux-64                25 KB\n",
            "\u001b[32m  xorg-kbproto                     \u001b[00m         1.0.7  h14c3975_1002           conda-forge/linux-64                26 KB\n",
            "\u001b[32m  xorg-libice                      \u001b[00m        1.0.10  h516909a_0              conda-forge/linux-64                57 KB\n",
            "\u001b[32m  xorg-libsm                       \u001b[00m         1.2.3  hd9c2040_1000           conda-forge/linux-64                26 KB\n",
            "\u001b[32m  xorg-libx11                      \u001b[00m         1.7.2  h7f98852_0              conda-forge/linux-64               941 KB\n",
            "\u001b[32m  xorg-libxau                      \u001b[00m         1.0.9  h14c3975_0              conda-forge/linux-64                13 KB\n",
            "\u001b[32m  xorg-libxdmcp                    \u001b[00m         1.1.3  h516909a_0              conda-forge/linux-64                18 KB\n",
            "\u001b[32m  xorg-libxext                     \u001b[00m         1.3.4  h7f98852_1              conda-forge/linux-64                54 KB\n",
            "\u001b[32m  xorg-libxrender                  \u001b[00m        0.9.10  h7f98852_1003           conda-forge/linux-64                32 KB\n",
            "\u001b[32m  xorg-libxt                       \u001b[00m         1.2.1  h7f98852_2              conda-forge/linux-64               375 KB\n",
            "\u001b[32m  xorg-renderproto                 \u001b[00m        0.11.1  h14c3975_1002           conda-forge/linux-64                 8 KB\n",
            "\u001b[32m  xorg-xextproto                   \u001b[00m         7.3.0  h14c3975_1002           conda-forge/linux-64                27 KB\n",
            "\u001b[32m  xorg-xproto                      \u001b[00m        7.0.31  h14c3975_1007           conda-forge/linux-64                72 KB\n",
            "\u001b[32m  yq                               \u001b[00m        2.13.0  pyhd8ed1ab_0            conda-forge/noarch                  20 KB\n",
            "\u001b[32m  zeromq                           \u001b[00m         4.3.4  h9c3ff4c_1              conda-forge/linux-64               351 KB\n",
            "\u001b[32m  zipp                             \u001b[00m         3.8.0  pyhd8ed1ab_0            conda-forge/noarch                  12 KB\n",
            "\n",
            "  Change:\n",
            "──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────\n",
            "\n",
            "\u001b[31m  brotlipy                         \u001b[00m         0.7.0  py37h5e8e339_1001       installed                                \n",
            "\u001b[32m  brotlipy                         \u001b[00m         0.7.0  py38h0a891b7_1004       conda-forge/linux-64               342 KB\n",
            "\u001b[31m  chardet                          \u001b[00m         4.0.0  py37h89c1867_1          installed                                \n",
            "\u001b[32m  chardet                          \u001b[00m         4.0.0  py38h578d9bd_3          conda-forge/linux-64               205 KB\n",
            "\u001b[31m  libedit                          \u001b[00m  3.1.20191231  he28a2e2_2              installed                                \n",
            "\u001b[32m  libedit                          \u001b[00m  3.1.20191231  he28a2e2_2              conda-forge/linux-64         \u001b[32m      Cached\u001b[00m\n",
            "\u001b[31m  pycosat                          \u001b[00m         0.6.3  py37h5e8e339_1006       installed                                \n",
            "\u001b[32m  pycosat                          \u001b[00m         0.6.3  py38h0a891b7_1010       conda-forge/linux-64               108 KB\n",
            "\u001b[31m  pysocks                          \u001b[00m         1.7.1  py37h89c1867_3          installed                                \n",
            "\u001b[32m  pysocks                          \u001b[00m         1.7.1  py38h578d9bd_5          conda-forge/linux-64                28 KB\n",
            "\u001b[31m  zlib                             \u001b[00m        1.2.11  h516909a_1010           installed                                \n",
            "\u001b[32m  zlib                             \u001b[00m        1.2.11  h166bdaf_1014           conda-forge/linux-64                88 KB\n",
            "\n",
            "  Upgrade:\n",
            "──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────\n",
            "\n",
            "\u001b[31m  c-ares                           \u001b[00m        1.17.1  h7f98852_1              installed                                \n",
            "\u001b[32m  c-ares                           \u001b[00m        1.18.1  h7f98852_0              conda-forge/linux-64               113 KB\n",
            "\u001b[31m  ca-certificates                  \u001b[00m     2020.12.5  ha878542_0              installed                                \n",
            "\u001b[32m  ca-certificates                  \u001b[00m     2021.10.8  ha878542_0              conda-forge/linux-64               139 KB\n",
            "\u001b[31m  certifi                          \u001b[00m     2020.12.5  py37h89c1867_1          installed                                \n",
            "\u001b[32m  certifi                          \u001b[00m     2021.10.8  py38h578d9bd_2          conda-forge/linux-64               145 KB\n",
            "\u001b[31m  cffi                             \u001b[00m        1.14.5  py37hc58025e_0          installed                                \n",
            "\u001b[32m  cffi                             \u001b[00m        1.15.0  py38h3931269_0          conda-forge/linux-64               226 KB\n",
            "\u001b[31m  conda                            \u001b[00m         4.9.2  py37h89c1867_0          installed                                \n",
            "\u001b[32m  conda                            \u001b[00m        4.12.0  py38h578d9bd_0          conda-forge/linux-64                 1 MB\n",
            "\u001b[31m  conda-package-handling           \u001b[00m         1.7.2  py37hb5d75c8_0          installed                                \n",
            "\u001b[32m  conda-package-handling           \u001b[00m         1.8.1  py38h0a891b7_1          conda-forge/linux-64              1018 KB\n",
            "\u001b[31m  cryptography                     \u001b[00m         3.4.5  py37h5d9358c_1          installed                                \n",
            "\u001b[32m  cryptography                     \u001b[00m        36.0.2  py38h2b5fc30_1          conda-forge/linux-64                 2 MB\n",
            "\u001b[31m  icu                              \u001b[00m          68.1  h58526e2_0              installed                                \n",
            "\u001b[32m  icu                              \u001b[00m          69.1  h9c3ff4c_0              conda-forge/linux-64                13 MB\n",
            "\u001b[31m  idna                             \u001b[00m          2.10  pyh9f0ad1d_0            installed                                \n",
            "\u001b[32m  idna                             \u001b[00m           3.3  pyhd8ed1ab_0            conda-forge/noarch                  55 KB\n",
            "\u001b[31m  krb5                             \u001b[00m        1.17.2  h926e7f8_0              installed                                \n",
            "\u001b[32m  krb5                             \u001b[00m        1.19.3  h3790be6_0              conda-forge/linux-64                 1 MB\n",
            "\u001b[31m  ld_impl_linux-64                 \u001b[00m        2.35.1  hea4e1c9_2              installed                                \n",
            "\u001b[32m  ld_impl_linux-64                 \u001b[00m        2.36.1  hea4e1c9_2              conda-forge/linux-64               667 KB\n",
            "\u001b[31m  libarchive                       \u001b[00m         3.5.1  h3f442fb_1              installed                                \n",
            "\u001b[32m  libarchive                       \u001b[00m         3.5.2  hccf745f_1              conda-forge/linux-64                 2 MB\n",
            "\u001b[31m  libcurl                          \u001b[00m        7.75.0  hc4aaa36_0              installed                                \n",
            "\u001b[32m  libcurl                          \u001b[00m        7.82.0  h7bff187_0              conda-forge/linux-64               341 KB\n",
            "\u001b[31m  libffi                           \u001b[00m           3.3  h58526e2_2              installed                                \n",
            "\u001b[32m  libffi                           \u001b[00m         3.4.2  h7f98852_5              conda-forge/linux-64                57 KB\n",
            "\u001b[31m  libgcc-ng                        \u001b[00m         9.3.0  h2828fa1_18             installed                                \n",
            "\u001b[32m  libgcc-ng                        \u001b[00m        11.2.0  h1d223b6_15             conda-forge/linux-64               911 KB\n",
            "\u001b[31m  libgomp                          \u001b[00m         9.3.0  h2828fa1_18             installed                                \n",
            "\u001b[32m  libgomp                          \u001b[00m        11.2.0  h1d223b6_15             conda-forge/linux-64               432 KB\n",
            "\u001b[31m  libnghttp2                       \u001b[00m        1.43.0  h812cca2_0              installed                                \n",
            "\u001b[32m  libnghttp2                       \u001b[00m        1.47.0  h727a467_0              conda-forge/linux-64               808 KB\n",
            "\u001b[31m  libssh2                          \u001b[00m         1.9.0  ha56f1ee_6              installed                                \n",
            "\u001b[32m  libssh2                          \u001b[00m        1.10.0  ha56f1ee_2              conda-forge/linux-64               233 KB\n",
            "\u001b[31m  libstdcxx-ng                     \u001b[00m         9.3.0  h6de172a_18             installed                                \n",
            "\u001b[32m  libstdcxx-ng                     \u001b[00m        11.2.0  he4da1e4_15             conda-forge/linux-64                 4 MB\n",
            "\u001b[31m  libxml2                          \u001b[00m        2.9.10  h72842e0_3              installed                                \n",
            "\u001b[32m  libxml2                          \u001b[00m        2.9.12  h885dcf4_1              conda-forge/linux-64               760 KB\n",
            "\u001b[31m  mamba                            \u001b[00m         0.8.0  py37h7f483ca_0          installed                                \n",
            "\u001b[32m  mamba                            \u001b[00m         0.9.0  py38h2aa5da1_0          conda-forge/linux-64               644 KB\n",
            "\u001b[31m  ncurses                          \u001b[00m           6.2  h58526e2_4              installed                                \n",
            "\u001b[32m  ncurses                          \u001b[00m           6.3  h27087fc_1              conda-forge/linux-64              1002 KB\n",
            "\u001b[31m  openssl                          \u001b[00m        1.1.1j  h7f98852_0              installed                                \n",
            "\u001b[32m  openssl                          \u001b[00m        1.1.1n  h166bdaf_0              conda-forge/linux-64                 2 MB\n",
            "\u001b[31m  pip                              \u001b[00m        21.0.1  pyhd8ed1ab_0            installed                                \n",
            "\u001b[32m  pip                              \u001b[00m        22.0.4  pyhd8ed1ab_0            conda-forge/noarch                   2 MB\n",
            "\u001b[31m  pycparser                        \u001b[00m          2.20  pyh9f0ad1d_2            installed                                \n",
            "\u001b[32m  pycparser                        \u001b[00m          2.21  pyhd8ed1ab_0            conda-forge/noarch                 100 KB\n",
            "\u001b[31m  pyopenssl                        \u001b[00m        20.0.1  pyhd8ed1ab_0            installed                                \n",
            "\u001b[32m  pyopenssl                        \u001b[00m        22.0.0  pyhd8ed1ab_0            conda-forge/noarch                  49 KB\n",
            "\u001b[31m  python                           \u001b[00m        3.7.10  hffdb5ce_100_cpython    installed                                \n",
            "\u001b[32m  python                           \u001b[00m        3.8.13  h582c2e5_0_cpython      conda-forge/linux-64                25 MB\n",
            "\u001b[31m  python_abi                       \u001b[00m           3.7  1_cp37m                 installed                                \n",
            "\u001b[32m  python_abi                       \u001b[00m           3.8  2_cp38                  conda-forge/linux-64                 4 KB\n",
            "\u001b[31m  readline                         \u001b[00m           8.0  he28a2e2_2              installed                                \n",
            "\u001b[32m  readline                         \u001b[00m         8.1.2  h7f8727e_1              pkgs/main/linux-64                 354 KB\n",
            "\u001b[31m  requests                         \u001b[00m        2.25.1  pyhd3deb0d_0            installed                                \n",
            "\u001b[32m  requests                         \u001b[00m        2.27.1  pyhd8ed1ab_0            conda-forge/noarch                  53 KB\n",
            "\u001b[31m  ruamel_yaml                      \u001b[00m       0.15.80  py37h5e8e339_1004       installed                                \n",
            "\u001b[32m  ruamel_yaml                      \u001b[00m      0.15.100  py38h27cfd23_0          pkgs/main/linux-64                 258 KB\n",
            "\u001b[31m  setuptools                       \u001b[00m        49.6.0  py37h89c1867_3          installed                                \n",
            "\u001b[32m  setuptools                       \u001b[00m        59.8.0  py38h578d9bd_1          conda-forge/linux-64              1017 KB\n",
            "\u001b[31m  six                              \u001b[00m        1.15.0  pyh9f0ad1d_0            installed                                \n",
            "\u001b[32m  six                              \u001b[00m        1.16.0  pyh6c4a22f_0            conda-forge/noarch                  14 KB\n",
            "\u001b[31m  sqlite                           \u001b[00m        3.34.0  h74cdb3f_0              installed                                \n",
            "\u001b[32m  sqlite                           \u001b[00m        3.37.1  h4ff8645_0              conda-forge/linux-64                 2 MB\n",
            "\u001b[31m  tk                               \u001b[00m        8.6.10  h21135ba_1              installed                                \n",
            "\u001b[32m  tk                               \u001b[00m        8.6.12  h27826a3_0              conda-forge/linux-64                 3 MB\n",
            "\u001b[31m  tqdm                             \u001b[00m        4.59.0  pyhd8ed1ab_0            installed                                \n",
            "\u001b[32m  tqdm                             \u001b[00m        4.64.0  pyhd8ed1ab_0            conda-forge/noarch                  81 KB\n",
            "\u001b[31m  urllib3                          \u001b[00m        1.26.3  pyhd8ed1ab_0            installed                                \n",
            "\u001b[32m  urllib3                          \u001b[00m        1.26.9  pyhd8ed1ab_0            conda-forge/noarch                 100 KB\n",
            "\u001b[31m  wheel                            \u001b[00m        0.36.2  pyhd3deb0d_0            installed                                \n",
            "\u001b[32m  wheel                            \u001b[00m        0.37.1  pyhd8ed1ab_0            conda-forge/noarch                  31 KB\n",
            "\u001b[31m  zstd                             \u001b[00m         1.4.9  ha95c52a_0              installed                                \n",
            "\u001b[32m  zstd                             \u001b[00m         1.5.2  ha95c52a_0              conda-forge/linux-64               458 KB\n",
            "\n",
            "  Summary:\n",
            "\n",
            "  Install: 398 packages\n",
            "  Change: 6 packages\n",
            "  Upgrade: 39 packages\n",
            "\n",
            "  Total download: 1 GB\n",
            "\n",
            "──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "font-ttf-inconsolata | 94 KB     | : 100% 1.0/1 [00:00<00:00, 10.20it/s]\n",
            "xorg-libxau-1.0.9    | 13 KB     | : 100% 1.0/1 [00:00<00:00, 28.38it/s]\n",
            "sysroot_linux-64-2.1 | 31.4 MB   | : 100% 1.0/1 [00:07<00:00,  7.85s/it]               \n",
            "r-lifecycle-1.0.1    | 109 KB    | : 100% 1.0/1 [00:00<00:00, 15.28it/s]\n",
            "asttokens-2.0.5      | 21 KB     | : 100% 1.0/1 [00:00<00:00, 22.39it/s]\n",
            "mafft-7.490          | 3.5 MB    | : 100% 1.0/1 [00:00<00:00,  1.25it/s]\n",
            "freetype-2.10.4      | 890 KB    | : 100% 1.0/1 [00:00<00:00,  4.22it/s]\n",
            "wheel-0.37.1         | 31 KB     | : 100% 1.0/1 [00:00<00:00, 13.07it/s]\n",
            "r-munsell-0.5.0      | 247 KB    | : 100% 1.0/1 [00:00<00:00, 13.50it/s]\n",
            "joblib-1.1.0         | 210 KB    | : 100% 1.0/1 [00:00<00:00,  7.75it/s]\n",
            "cachecontrol-0.12.10 | 23 KB     | : 100% 1.0/1 [00:00<00:00, 19.16it/s]\n",
            "entrypoints-0.4      | 9 KB      | : 100% 1.0/1 [00:00<00:00, 30.36it/s]\n",
            "r-vctrs-0.4.0        | 1.2 MB    | : 100% 1.0/1 [00:00<00:00,  4.75it/s]\n",
            "fonts-conda-forge-1  | 4 KB      | : 100% 1.0/1 [00:00<00:00, 24.88it/s]\n",
            "libogg-1.3.4         | 206 KB    | : 100% 1.0/1 [00:00<00:00, 14.09it/s]\n",
            "r-mgcv-1.8_40        | 2.9 MB    | : 100% 1.0/1 [00:00<00:00,  2.25it/s]\n",
            "libstdcxx-devel_linu | 9.6 MB    | : 100% 1.0/1 [00:02<00:00,  2.55s/it]\n",
            "jbig-2.1             | 43 KB     | : 100% 1.0/1 [00:00<00:00, 22.52it/s]\n",
            "libxcb-1.13          | 391 KB    | : 100% 1.0/1 [00:00<00:00,  3.92it/s]\n",
            "bioconductor-matrixg | 342 KB    | : 100% 1.0/1 [00:00<00:00, 11.08it/s]\n",
            "r-crayon-1.5.1       | 165 KB    | : 100% 1.0/1 [00:00<00:00, 12.01it/s]\n",
            "r-evaluate-0.15      | 84 KB     | : 100% 1.0/1 [00:00<00:00, 14.76it/s]\n",
            "cryptography-36.0.2  | 1.6 MB    | : 100% 1.0/1 [00:00<00:00,  1.79it/s]\n",
            "pillow-9.1.0         | 45.1 MB   | : 100% 1.0/1 [00:07<00:00,  7.45s/it]               \n",
            "libnsl-2.0.0         | 31 KB     | : 100% 1.0/1 [00:00<00:00, 14.52it/s]\n",
            "bwidget-1.9.14       | 120 KB    | : 100% 1.0/1 [00:00<00:00, 10.90it/s]\n",
            "ruamel_yaml-0.15.100 | 258 KB    | : 100% 1.0/1 [00:00<00:00,  6.46it/s]\n",
            "click-7.1.2          | 64 KB     | : 100% 1.0/1 [00:00<00:00, 18.12it/s]\n",
            "lockfile-0.12.2      | 11 KB     | : 100% 1.0/1 [00:00<00:00, 28.04it/s]\n",
            "perl-list-moreutils- | 32 KB     | : 100% 1.0/1 [00:00<00:00, 26.02it/s]\n",
            "emperor-1.0.3        | 625 KB    | : 100% 1.0/1 [00:00<00:00,  3.74it/s]\n",
            "ld_impl_linux-64-2.3 | 667 KB    | : 100% 1.0/1 [00:00<00:00,  3.56it/s]\n",
            "q2-composition-2022. | 170 KB    | : 100% 1.0/1 [00:00<00:00,  4.15it/s]\n",
            "perl-scalar-list-uti | 43 KB     | : 100% 1.0/1 [00:00<00:00, 22.48it/s]\n",
            "mysql-common-8.0.28  | 1.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.50it/s]\n",
            "r-ps-1.6.0           | 299 KB    | : 100% 1.0/1 [00:00<00:00, 12.80it/s]\n",
            "webencodings-0.5.1   | 12 KB     | : 100% 1.0/1 [00:00<00:00, 24.43it/s]\n",
            "gcc_impl_linux-64-10 | 46.5 MB   | : 100% 1.0/1 [00:11<00:00, 11.20s/it]               \n",
            "munkres-1.1.4        | 12 KB     | : 100% 1.0/1 [00:00<00:00, 23.21it/s]\n",
            "xorg-libsm-1.2.3     | 26 KB     | : 100% 1.0/1 [00:00<00:00, 27.36it/s]\n",
            "font-ttf-source-code | 684 KB    | : 100% 1.0/1 [00:00<00:00,  5.23it/s]\n",
            "bowtie2-2.4.5        | 14.4 MB   | : 100% 1.0/1 [00:03<00:00,  3.30s/it]\n",
            "importlib_resources- | 22 KB     | : 100% 1.0/1 [00:00<00:00, 22.55it/s]\n",
            "readline-8.1.2       | 354 KB    | : 100% 1.0/1 [00:00<00:00, 11.99it/s]\n",
            "pluggy-1.0.0         | 25 KB     | : 100% 1.0/1 [00:00<00:00, 19.64it/s]\n",
            "nspr-4.32            | 233 KB    | : 100% 1.0/1 [00:00<00:00, 10.52it/s]\n",
            "pysocks-1.7.1        | 28 KB     | : 100% 1.0/1 [00:00<00:00, 21.27it/s]\n",
            "pigz-2.6             | 87 KB     | : 100% 1.0/1 [00:00<00:00, 17.77it/s]\n",
            "patsy-0.5.2          | 188 KB    | : 100% 1.0/1 [00:00<00:00, 11.40it/s]\n",
            "yq-2.13.0            | 20 KB     | : 100% 1.0/1 [00:00<00:00, 27.24it/s]\n",
            "libstdcxx-ng-11.2.0  | 4.2 MB    | : 100% 1.0/1 [00:01<00:00,  1.01s/it]\n",
            "zstd-1.5.2           | 458 KB    | : 100% 1.0/1 [00:00<00:00,  7.98it/s]\n",
            "cairo-1.16.0         | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  2.13it/s]\n",
            "colorama-0.4.4       | 18 KB     | : 100% 1.0/1 [00:00<00:00, 29.68it/s]\n",
            "perl-exporter-tiny-1 | 25 KB     | : 100% 1.0/1 [00:00<00:00, 23.80it/s]\n",
            "r-futile.options-1.0 | 27 KB     | : 100% 1.0/1 [00:00<00:00, 21.35it/s]\n",
            "nss-3.77             | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.99it/s]\n",
            "bioconductor-genomic | 2.3 MB    | : 100% 1.0/1 [00:00<00:00,  2.32it/s]\n",
            "r-lambda.r-1.2.4     | 120 KB    | : 100% 1.0/1 [00:00<00:00, 19.37it/s]\n",
            "htslib-1.15.1        | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  2.01it/s]\n",
            "isa-l-2.30.0         | 192 KB    | : 100% 1.0/1 [00:00<00:00, 12.35it/s]\n",
            "q2-mystery-stew-2022 | 27 KB     | : 100% 1.0/1 [00:00<00:00,  6.25it/s]\n",
            "r-ellipsis-0.3.2     | 42 KB     | : 100% 1.0/1 [00:00<00:00, 22.76it/s]\n",
            "q2-demux-2022.2.0    | 192 KB    | : 100% 1.0/1 [00:00<00:00,  4.68it/s]\n",
            "fonts-conda-ecosyste | 4 KB      | : 100% 1.0/1 [00:00<00:00, 25.31it/s]\n",
            "libvorbis-1.3.7      | 287 KB    | : 100% 1.0/1 [00:00<00:00,  9.34it/s]\n",
            "msgpack-python-1.0.3 | 90 KB     | : 100% 1.0/1 [00:00<00:00, 19.84it/s]\n",
            "conda-4.12.0         | 1.0 MB    | : 100% 1.0/1 [00:00<00:00,  2.46it/s]\n",
            "q2-gneiss-2022.2.0   | 54 KB     | : 100% 1.0/1 [00:00<00:00,  7.34it/s]\n",
            "r-isoband-0.2.5      | 1.9 MB    | : 100% 1.0/1 [00:00<00:00,  3.64it/s]\n",
            "pynndescent-0.5.6    | 45 KB     | : 100% 1.0/1 [00:00<00:00, 22.83it/s]\n",
            "jq-1.6               | 346 KB    | : 100% 1.0/1 [00:00<00:00,  8.17it/s]\n",
            "oniguruma-6.9.7.1    | 409 KB    | : 100% 1.0/1 [00:00<00:00,  7.94it/s]\n",
            "bioconductor-genomei | 4.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.31it/s]\n",
            "bibtexparser-1.1.0   | 32 KB     | : 100% 1.0/1 [00:00<00:00, 18.34it/s]\n",
            "keyutils-1.6.1       | 115 KB    | : 100% 1.0/1 [00:00<00:00, 15.17it/s]\n",
            "xorg-renderproto-0.1 | 8 KB      | : 100% 1.0/1 [00:00<00:00, 18.44it/s]\n",
            "xorg-libxrender-0.9. | 32 KB     | : 100% 1.0/1 [00:00<00:00, 23.41it/s]\n",
            "libsanitizer-10.3.0  | 6.0 MB    | : 100% 1.0/1 [00:01<00:00,  1.38s/it]               \n",
            "pip-22.0.4           | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  1.42it/s]\n",
            "q2-diversity-lib-202 | 88 KB     | : 100% 1.0/1 [00:00<00:00,  5.17it/s]\n",
            "libnghttp2-1.47.0    | 808 KB    | : 100% 1.0/1 [00:00<00:00,  3.76it/s]\n",
            "bioconductor-shortre | 5.2 MB    | : 100% 1.0/1 [00:00<00:00,  1.01it/s]\n",
            "tqdm-4.64.0          | 81 KB     | : 100% 1.0/1 [00:00<00:00, 18.49it/s]\n",
            "gettext-0.19.8.1     | 3.6 MB    | : 100% 1.0/1 [00:01<00:00,  1.27s/it]\n",
            "r-farver-2.1.0       | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  2.70it/s]\n",
            "r-pillar-1.7.0       | 706 KB    | : 100% 1.0/1 [00:00<00:00,  5.82it/s]\n",
            "c-ares-1.18.1        | 113 KB    | : 100% 1.0/1 [00:00<00:00, 15.12it/s]\n",
            "r-callr-3.7.0        | 439 KB    | : 100% 1.0/1 [00:00<00:00,  9.95it/s]\n",
            "argon2-cffi-bindings | 34 KB     | : 100% 1.0/1 [00:00<00:00, 24.86it/s]\n",
            "h5py-3.6.0           | 1.4 MB    | : 100% 1.0/1 [00:00<00:00,  2.01it/s]\n",
            "ijson-3.1.3          | 24 KB     | : 100% 1.0/1 [00:00<00:00, 17.78it/s]\n",
            "xorg-libice-1.0.10   | 57 KB     | : 100% 1.0/1 [00:00<00:00, 18.08it/s]\n",
            "dill-0.3.4           | 62 KB     | : 100% 1.0/1 [00:00<00:00, 12.53it/s]\n",
            "jpeg-9e              | 269 KB    | : 100% 1.0/1 [00:00<00:00,  7.31it/s]\n",
            "binutils_impl_linux- | 10.4 MB   | : 100% 1.0/1 [00:03<00:00,  3.00s/it]               \n",
            "testpath-0.6.0       | 85 KB     | : 100% 1.0/1 [00:00<00:00, 24.72it/s]\n",
            "r-gtable-0.3.0       | 423 KB    | : 100% 1.0/1 [00:00<00:00,  8.26it/s]\n",
            "perl-5.32.1          | 14.4 MB   | : 100% 1.0/1 [00:04<00:00,  4.53s/it]               \n",
            "q2-fragment-insertio | 31 KB     | : 100% 1.0/1 [00:00<00:00,  6.35it/s]\n",
            "python-isal-0.11.1   | 141 KB    | : 100% 1.0/1 [00:00<00:00, 13.51it/s]\n",
            "r-withr-2.5.0        | 240 KB    | : 100% 1.0/1 [00:00<00:00, 12.80it/s]\n",
            "iniconfig-1.1.1      | 8 KB      | : 100% 1.0/1 [00:00<00:00, 23.99it/s]\n",
            "r-rcpp-1.0.8.3       | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.61it/s]\n",
            "matplotlib-3.5.1     | 6 KB      | : 100% 1.0/1 [00:00<00:00, 21.86it/s]\n",
            "pickleshare-0.7.5    | 13 KB     | : 100% 1.0/1 [00:00<00:00, 26.71it/s]\n",
            "r-testthat-3.1.3     | 1.7 MB    | : 100% 1.0/1 [00:00<00:00,  3.09it/s]\n",
            "q2templates-2022.2.0 | 189 KB    | : 100% 1.0/1 [00:00<00:00,  1.79it/s]\n",
            "psutil-5.9.0         | 346 KB    | : 100% 1.0/1 [00:00<00:00,  5.21it/s]\n",
            "argcomplete-2.0.0    | 34 KB     | : 100% 1.0/1 [00:00<00:00, 24.44it/s]\n",
            "r-mass-7.3_56        | 1.1 MB    | : 100% 1.0/1 [00:00<00:00,  3.75it/s]\n",
            "urllib3-1.26.9       | 100 KB    | : 100% 1.0/1 [00:00<00:00, 14.19it/s]\n",
            "cached-property-1.5. | 4 KB      | : 100% 1.0/1 [00:00<00:00, 28.12it/s]\n",
            "jupyter_client-7.2.2 | 90 KB     | : 100% 1.0/1 [00:00<00:00, 12.95it/s]\n",
            "q2-quality-control-2 | 286 KB    | : 100% 1.0/1 [00:00<00:00,  4.46it/s]\n",
            "backports.functools_ | 9 KB      | : 100% 1.0/1 [00:00<00:00, 19.40it/s]\n",
            "krb5-1.19.3          | 1.4 MB    | : 100% 1.0/1 [00:00<00:00,  1.98it/s]\n",
            "r-magrittr-2.0.3     | 215 KB    | : 100% 1.0/1 [00:00<00:00, 14.15it/s]\n",
            "font-ttf-dejavu-sans | 388 KB    | : 100% 1.0/1 [00:00<00:00,  8.08it/s]\n",
            "gstreamer-1.20.1     | 2.0 MB    | : 100% 1.0/1 [00:00<00:00,  1.25it/s]\n",
            "pexpect-4.8.0        | 80 KB     | : 100% 1.0/1 [00:00<00:00, 14.41it/s]\n",
            "fonttools-4.31.2     | 1.6 MB    | : 100% 1.0/1 [00:00<00:00,  1.49it/s]\n",
            "libarchive-3.5.2     | 1.6 MB    | : 100% 1.0/1 [00:00<00:00,  2.20it/s]\n",
            "r-permute-0.9_7      | 227 KB    | : 100% 1.0/1 [00:00<00:00,  7.28it/s]\n",
            "xorg-libxext-1.3.4   | 54 KB     | : 100% 1.0/1 [00:00<00:00, 14.19it/s]\n",
            "libopus-1.3.1        | 255 KB    | : 100% 1.0/1 [00:00<00:00,  6.76it/s]\n",
            "perl-types-serialise | 13 KB     | : 100% 1.0/1 [00:00<00:00, 15.39it/s]\n",
            "chardet-4.0.0        | 205 KB    | : 100% 1.0/1 [00:00<00:00,  8.21it/s]\n",
            "six-1.16.0           | 14 KB     | : 100% 1.0/1 [00:00<00:00, 23.81it/s]\n",
            "r-colorspace-2.0_3   | 2.6 MB    | : 100% 1.0/1 [00:00<00:00,  1.47it/s]               \n",
            "nbclient-0.5.13      | 64 KB     | : 100% 1.0/1 [00:00<00:00, 20.88it/s]\n",
            "r-plyr-1.8.7         | 836 KB    | : 100% 1.0/1 [00:00<00:00,  5.09it/s]\n",
            "beautifulsoup4-4.11. | 58 KB     | : 100% 1.0/1 [00:00<00:00, 17.85it/s]\n",
            "sed-4.8              | 264 KB    | : 100% 1.0/1 [00:00<00:00,  8.33it/s]\n",
            "attrs-21.4.0         | 49 KB     | : 100% 1.0/1 [00:00<00:00, 15.62it/s]\n",
            "iqtree-2.2.0_beta    | 6.9 MB    | : 100% 1.0/1 [00:01<00:00,  1.72s/it]               \n",
            "fontconfig-2.14.0    | 305 KB    | : 100% 1.0/1 [00:00<00:00,  8.12it/s]\n",
            "r-stringi-1.7.6      | 896 KB    | : 100% 1.0/1 [00:00<00:00,  5.64it/s]\n",
            "zeromq-4.3.4         | 351 KB    | : 100% 1.0/1 [00:00<00:00,  8.46it/s]\n",
            "perl-parent-0.238    | 11 KB     | : 100% 1.0/1 [00:00<00:00, 32.14it/s]\n",
            "q2-diversity-2022.2. | 259 KB    | : 100% 1.0/1 [00:00<00:00,  4.46it/s]\n",
            "requests-2.27.1      | 53 KB     | : 100% 1.0/1 [00:00<00:00, 19.16it/s]\n",
            "libblas-3.9.0        | 12 KB     | : 100% 1.0/1 [00:00<00:00, 25.04it/s]\n",
            "future-0.18.2        | 716 KB    | : 100% 1.0/1 [00:00<00:00,  3.96it/s]\n",
            "bioconductor-biobase | 2.3 MB    | : 100% 1.0/1 [00:00<00:00,  2.81it/s]\n",
            "font-ttf-ubuntu-0.83 | 1.9 MB    | : 100% 1.0/1 [00:00<00:00,  2.76it/s]\n",
            "matplotlib-inline-0. | 11 KB     | : 100% 1.0/1 [00:00<00:00, 21.65it/s]\n",
            "libssh2-1.10.0       | 233 KB    | : 100% 1.0/1 [00:00<00:00,  9.09it/s]\n",
            "importlib_metadata-4 | 4 KB      | : 100% 1.0/1 [00:00<00:00, 14.83it/s]\n",
            "llvmlite-0.36.0      | 2.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.70it/s]\n",
            "argon2-cffi-21.3.0   | 15 KB     | : 100% 1.0/1 [00:00<00:00, 32.33it/s]\n",
            "tktable-2.10         | 89 KB     | : 100% 1.0/1 [00:00<00:00, 20.49it/s]\n",
            "q2-longitudinal-2022 | 7.6 MB    | : 100% 1.0/1 [00:01<00:00,  1.36s/it]\n",
            "kernel-headers_linux | 707 KB    | : 100% 1.0/1 [00:00<00:00,  2.68it/s]\n",
            "r-labeling-0.4.2     | 67 KB     | : 100% 1.0/1 [00:00<00:00, 22.88it/s]\n",
            "soupsieve-2.3.1      | 33 KB     | : 100% 1.0/1 [00:00<00:00, 23.91it/s]\n",
            "conda-package-handli | 1018 KB   | : 100% 1.0/1 [00:00<00:00,  4.08it/s]\n",
            "r-cluster-2.1.3      | 572 KB    | : 100% 1.0/1 [00:00<00:00,  9.80it/s]\n",
            "libllvm13-13.0.1     | 33.6 MB   | : 100% 1.0/1 [00:08<00:00,  8.36s/it]               \n",
            "statsmodels-0.13.2   | 11.5 MB   | : 100% 1.0/1 [00:03<00:00,  3.54s/it]\n",
            "r-png-0.1_7          | 58 KB     | : 100% 1.0/1 [00:00<00:00,  5.97it/s]                \n",
            "nbconvert-pandoc-6.4 | 4 KB      | : 100% 1.0/1 [00:00<00:00, 34.32it/s]\n",
            "q2-taxa-2022.2.0     | 110 KB    | : 100% 1.0/1 [00:00<00:00,  5.01it/s]\n",
            "perl-compress-raw-bz | 47 KB     | : 100% 1.0/1 [00:00<00:00, 25.27it/s]\n",
            "certifi-2021.10.8    | 145 KB    | : 100% 1.0/1 [00:00<00:00, 16.90it/s]\n",
            "fasttree-2.1.10      | 477 KB    | : 100% 1.0/1 [00:00<00:00,  7.76it/s]\n",
            "qiime2-2022.2.0      | 353 KB    | : 100% 1.0/1 [00:00<00:00,  3.37it/s]\n",
            "nbformat-5.3.0       | 104 KB    | : 100% 1.0/1 [00:00<00:00, 13.11it/s]\n",
            "_r-mutex-1.0.1       | 3 KB      | : 100% 1.0/1 [00:00<00:00, 37.99it/s]\n",
            "q2-cutadapt-2022.2.0 | 6.3 MB    | : 100% 1.0/1 [00:01<00:00,  1.11s/it]\n",
            "q2-metadata-2022.2.0 | 970 KB    | : 100% 1.0/1 [00:00<00:00,  3.08it/s]\n",
            "perl-encode-3.17     | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.90it/s]\n",
            "gfortran_linux-64-10 | 24 KB     | : 100% 1.0/1 [00:00<00:00, 29.18it/s]\n",
            "pyqtchart-5.12       | 257 KB    | : 100% 1.0/1 [00:00<00:00, 10.72it/s]\n",
            "q2-vsearch-2022.2.0  | 619 KB    | : 100% 1.0/1 [00:00<00:00,  3.47it/s]\n",
            "pyqt5-sip-4.19.18    | 311 KB    | : 100% 1.0/1 [00:00<00:00,  9.60it/s]\n",
            "umap-learn-0.5.2     | 131 KB    | : 100% 1.0/1 [00:00<00:00, 15.52it/s]\n",
            "unifrac-0.20.3       | 461 KB    | : 100% 1.0/1 [00:00<00:00,  8.33it/s]\n",
            "q2-alignment-2022.2. | 20 KB     | : 100% 1.0/1 [00:00<00:00,  6.87it/s]\n",
            "gxx_linux-64-10.3.0  | 24 KB     | : 100% 1.0/1 [00:00<00:00, 17.14it/s]\n",
            "libbrotlicommon-1.0. | 65 KB     | : 100% 1.0/1 [00:00<00:00, 26.60it/s]\n",
            "nose-1.3.7           | 213 KB    | : 100% 1.0/1 [00:00<00:00,  8.93it/s]\n",
            "r-cli-3.2.0          | 1.1 MB    | : 100% 1.0/1 [00:00<00:00,  4.05it/s]\n",
            "kiwisolver-1.4.2     | 75 KB     | : 100% 1.0/1 [00:00<00:00, 20.67it/s]\n",
            "libclang-13.0.1      | 11.1 MB   | : 100% 1.0/1 [00:03<00:00,  3.23s/it]\n",
            "traitlets-5.1.1      | 82 KB     | : 100% 1.0/1 [00:00<00:00, 18.88it/s]\n",
            "ipython_genutils-0.2 | 21 KB     | : 100% 1.0/1 [00:00<00:00, 25.05it/s]\n",
            "gsl-2.7.1            | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  5.83it/s]\n",
            "r-processx-3.5.3     | 318 KB    | : 100% 1.0/1 [00:00<00:00, 13.19it/s]\n",
            "q2-types-2022.2.0    | 132 KB    | : 100% 1.0/1 [00:00<00:00,  4.21it/s]\n",
            "send2trash-1.8.0     | 17 KB     | : 100% 1.0/1 [00:00<00:00, 24.71it/s]\n",
            "unicodedata2-14.0.0  | 498 KB    | : 100% 1.0/1 [00:00<00:00,  8.82it/s]\n",
            "pcre2-10.37          | 1.1 MB    | : 100% 1.0/1 [00:00<00:00,  3.45it/s]\n",
            "q2-quality-filter-20 | 32 KB     | : 100% 1.0/1 [00:00<00:00,  7.12it/s]\n",
            "scikit-learn-0.24.1  | 7.6 MB    | : 100% 1.0/1 [00:02<00:00,  2.75s/it]               \n",
            "bioconductor-genomic | 2.2 MB    | : 100% 1.0/1 [00:00<00:00,  2.01it/s]\n",
            "make-4.3             | 507 KB    | : 100% 1.0/1 [00:00<00:00,  4.27it/s]\n",
            "gxx_impl_linux-64-10 | 11.4 MB   | : 100% 1.0/1 [00:02<00:00,  2.70s/it]\n",
            "perl-carp-1.50       | 22 KB     | : 100% 1.0/1 [00:00<00:00, 25.45it/s]\n",
            "lcms2-2.12           | 443 KB    | : 100% 1.0/1 [00:00<00:00,  8.23it/s]\n",
            "pytz-2022.1          | 242 KB    | : 100% 1.0/1 [00:00<00:00,  6.51it/s]\n",
            "bleach-5.0.0         | 123 KB    | : 100% 1.0/1 [00:00<00:00, 13.03it/s]\n",
            "q2-deblur-2022.2.0   | 271 KB    | : 100% 1.0/1 [00:00<00:00,  3.50it/s]\n",
            "flit-core-3.7.1      | 44 KB     | : 100% 1.0/1 [00:00<00:00, 17.99it/s]\n",
            "bioconductor-xvector | 728 KB    | : 100% 1.0/1 [00:00<00:00,  5.50it/s]\n",
            "cffi-1.15.0          | 226 KB    | : 100% 1.0/1 [00:00<00:00, 11.58it/s]\n",
            "dnaio-0.8.1          | 76 KB     | : 100% 1.0/1 [00:00<00:00, 11.20it/s]\n",
            "entrez-direct-16.2   | 7.5 MB    | : 100% 1.0/1 [00:01<00:00,  1.39s/it]\n",
            "cutadapt-3.7         | 201 KB    | : 100% 1.0/1 [00:00<00:00, 11.20it/s]\n",
            "pandoc-2.17.1.1      | 11.9 MB   | : 100% 1.0/1 [00:04<00:00,  4.12s/it]\n",
            "ipython-8.2.0        | 1.1 MB    | : 100% 1.0/1 [00:00<00:00,  2.57it/s]\n",
            "tk-8.6.12            | 3.3 MB    | : 100% 1.0/1 [00:00<00:00,  1.33it/s]\n",
            "libgcc-7.2.0         | 304 KB    | : 100% 1.0/1 [00:00<00:00,  8.42it/s]\n",
            "r-ggplot2-3.3.5      | 4.0 MB    | : 100% 1.0/1 [00:00<00:00,  1.11it/s]\n",
            "biom-format-2.1.10   | 10.4 MB   | : 100% 1.0/1 [00:02<00:00,  2.61s/it]\n",
            "graphite2-1.3.13     | 102 KB    | : 100% 1.0/1 [00:00<00:00, 16.80it/s]\n",
            "jsonschema-4.4.0     | 57 KB     | : 100% 1.0/1 [00:00<00:00, 19.79it/s]\n",
            "cython-0.29.28       | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.38it/s]\n",
            "r-base-4.1.3         | 24.6 MB   | : 100% 1.0/1 [00:05<00:00,  5.12s/it]\n",
            "r-bh-1.78.0_0        | 11.8 MB   | : 100% 1.0/1 [00:06<00:00,  6.98s/it]\n",
            "networkx-2.7.1       | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  2.02it/s]\n",
            "r-formatr-1.12       | 170 KB    | : 100% 1.0/1 [00:00<00:00, 16.39it/s]\n",
            "r-waldo-0.4.0        | 109 KB    | : 100% 1.0/1 [00:00<00:00, 19.96it/s]\n",
            "packaging-21.3       | 36 KB     | : 100% 1.0/1 [00:00<00:00, 19.00it/s]\n",
            "liblapack-3.9.0      | 12 KB     | : 100% 1.0/1 [00:00<00:00, 26.39it/s]\n",
            "r-pkgload-1.2.4      | 172 KB    | : 100% 1.0/1 [00:00<00:00, 13.05it/s]\n",
            "bioconductor-genomei | 7 KB      | : 100% 1.0/1 [00:00<00:00, 22.61it/s]\n",
            "r-glue-1.6.2         | 154 KB    | : 100% 1.0/1 [00:00<00:00, 17.89it/s]\n",
            "seaborn-base-0.11.2  | 223 KB    | : 100% 1.0/1 [00:00<00:00,  7.11it/s]\n",
            "prompt-toolkit-3.0.2 | 252 KB    | : 100% 1.0/1 [00:00<00:00,  5.41it/s]\n",
            "tornado-6.1          | 645 KB    | : 100% 1.0/1 [00:00<00:00,  2.80it/s]\n",
            "zipp-3.8.0           | 12 KB     | : 100% 1.0/1 [00:00<00:00, 23.69it/s]\n",
            "deblur-1.1.0         | 2.3 MB    | : 100% 1.0/1 [00:02<00:00,  2.05s/it]\n",
            "libwebp-base-1.2.2   | 824 KB    | : 100% 1.0/1 [00:00<00:00,  3.50it/s]\n",
            "libbrotlienc-1.0.9   | 287 KB    | : 100% 1.0/1 [00:00<00:00, 12.60it/s]\n",
            "pyzmq-22.3.0         | 513 KB    | : 100% 1.0/1 [00:00<00:00,  5.44it/s]\n",
            "libcblas-3.9.0       | 12 KB     | : 100% 1.0/1 [00:00<00:00, 28.90it/s]\n",
            "r-rprojroot-2.0.3    | 115 KB    | : 100% 1.0/1 [00:00<00:00, 19.52it/s]\n",
            "perl-list-moreutils- | 49 KB     | : 100% 1.0/1 [00:00<00:00, 23.03it/s]\n",
            "libffi-3.4.2         | 57 KB     | : 100% 1.0/1 [00:00<00:00, 23.15it/s]\n",
            "openssl-1.1.1n       | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  2.24it/s]\n",
            "idna-3.3             | 55 KB     | : 100% 1.0/1 [00:00<00:00, 17.41it/s]\n",
            "ncurses-6.3          | 1002 KB   | : 100% 1.0/1 [00:00<00:00,  1.65it/s]\n",
            "hdmedians-0.14.2     | 161 KB    | : 100% 1.0/1 [00:00<00:00, 14.40it/s]\n",
            "pyparsing-3.0.7      | 79 KB     | : 100% 1.0/1 [00:00<00:00, 11.73it/s]\n",
            "libgcc-devel_linux-6 | 3.3 MB    | : 100% 1.0/1 [00:00<00:00,  1.32it/s]\n",
            "blast-2.12.0         | 22.2 MB   | : 100% 1.0/1 [00:04<00:00,  4.90s/it]               \n",
            "python-3.8.13        | 25.1 MB   | : 100% 1.0/1 [00:07<00:00,  7.11s/it]\n",
            "terminado-0.13.3     | 27 KB     | : 100% 1.0/1 [00:00<00:00, 20.89it/s]\n",
            "cached_property-1.5. | 11 KB     | : 100% 1.0/1 [00:00<00:00, 20.28it/s]\n",
            "sepp-4.3.10          | 6.6 MB    | : 100% 1.0/1 [00:01<00:00,  1.39s/it]\n",
            "r-pkgconfig-2.0.3    | 25 KB     | : 100% 1.0/1 [00:00<00:00, 32.02it/s]\n",
            "bokeh-2.4.2          | 13.6 MB   | : 100% 1.0/1 [00:05<00:00,  5.16s/it]\n",
            "openjpeg-2.4.0       | 444 KB    | : 100% 1.0/1 [00:00<00:00,  7.35it/s]\n",
            "libgfortran-ng-11.2. | 23 KB     | : 100% 1.0/1 [00:00<00:00, 23.85it/s]\n",
            "r-matrixstats-0.61.0 | 468 KB    | : 100% 1.0/1 [00:00<00:00,  6.19it/s]\n",
            "r-desc-1.4.1         | 332 KB    | : 100% 1.0/1 [00:00<00:00, 10.92it/s]\n",
            "binutils_linux-64-2. | 24 KB     | : 100% 1.0/1 [00:00<00:00, 18.48it/s]\n",
            "mistune-0.8.4        | 54 KB     | : 100% 1.0/1 [00:00<00:00, 14.48it/s]\n",
            "r-diffobj-0.3.5      | 1.0 MB    | : 100% 1.0/1 [00:00<00:00,  3.82it/s]\n",
            "vsearch-2.7.0        | 713 KB    | : 100% 1.0/1 [00:00<00:00,  4.04it/s]\n",
            "libwebp-1.2.2        | 85 KB     | : 100% 1.0/1 [00:00<00:00, 17.64it/s]\n",
            "scipy-1.8.0          | 23.6 MB   | : 100% 1.0/1 [00:06<00:00,  6.17s/it]\n",
            "pyyaml-6.0           | 182 KB    | : 100% 1.0/1 [00:00<00:00, 10.81it/s]\n",
            "xorg-libxt-1.2.1     | 375 KB    | : 100% 1.0/1 [00:00<00:00,  6.21it/s]\n",
            "brotlipy-0.7.0       | 342 KB    | : 100% 1.0/1 [00:00<00:00, 10.05it/s]\n",
            "py-1.11.0            | 74 KB     | : 100% 1.0/1 [00:00<00:00, 15.96it/s]\n",
            "r-jsonlite-1.8.0     | 1.0 MB    | : 100% 1.0/1 [00:00<00:00,  3.56it/s]\n",
            "xorg-xextproto-7.3.0 | 27 KB     | : 100% 1.0/1 [00:00<00:00, 17.86it/s]\n",
            "notebook-6.4.10      | 6.3 MB    | : 100% 1.0/1 [00:01<00:00,  1.99s/it]\n",
            "r-praise-1.0.0       | 24 KB     | : 100% 1.0/1 [00:00<00:00, 22.07it/s]\n",
            "libcurl-7.82.0       | 341 KB    | : 100% 1.0/1 [00:00<00:00,  8.89it/s]\n",
            "q2-dada2-2022.2.0    | 1.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.91it/s]\n",
            "fastcluster-1.2.6    | 48 KB     | : 100% 1.0/1 [00:00<00:00,  1.15it/s]               \n",
            "r-stringr-1.4.0      | 210 KB    | : 100% 1.0/1 [00:00<00:00, 13.68it/s]\n",
            "libtiff-4.3.0        | 638 KB    | : 100% 1.0/1 [00:00<00:00,  4.37it/s]\n",
            "curl-7.82.0          | 90 KB     | : 100% 1.0/1 [00:00<00:00, 19.56it/s]\n",
            "bioconductor-biocgen | 631 KB    | : 100% 1.0/1 [00:00<00:00,  4.56it/s]\n",
            "r-rcppparallel-5.1.5 | 568 KB    | : 100% 1.0/1 [00:00<00:00,  3.27it/s]\n",
            "lerc-3.0             | 216 KB    | : 100% 1.0/1 [00:00<00:00,  8.23it/s]\n",
            "brotli-bin-1.0.9     | 19 KB     | : 100% 1.0/1 [00:00<00:00, 23.03it/s]\n",
            "r-reshape2-1.4.4     | 128 KB    | : 100% 1.0/1 [00:00<00:00, 17.42it/s]\n",
            "charset-normalizer-2 | 35 KB     | : 100% 1.0/1 [00:00<00:00, 16.99it/s]\n",
            "libpng-1.6.37        | 359 KB    | : 100% 1.0/1 [00:00<00:00,  7.99it/s]\n",
            "qt-5.12.9            | 98.6 MB   | : 100% 1.0/1 [00:25<00:00, 25.03s/it]               \n",
            "tbb-2020.2           | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  3.48it/s]\n",
            "libxml2-2.9.12       | 760 KB    | : 100% 1.0/1 [00:00<00:00,  4.62it/s]\n",
            "xorg-kbproto-1.0.7   | 26 KB     | : 100% 1.0/1 [00:00<00:00, 23.71it/s]\n",
            "bioconductor-iranges | 2.4 MB    | : 100% 1.0/1 [00:00<00:00,  1.94it/s]\n",
            "seaborn-0.11.2       | 4 KB      | : 100% 1.0/1 [00:00<00:00, 26.66it/s]\n",
            "q2-feature-table-202 | 230 KB    | : 100% 1.0/1 [00:00<00:00,  4.04it/s]\n",
            "libsodium-1.0.18     | 366 KB    | : 100% 1.0/1 [00:00<00:00,  6.63it/s]\n",
            "pandocfilters-1.5.0  | 11 KB     | : 100% 1.0/1 [00:00<00:00, 17.57it/s]\n",
            "bioconductor-dada2-1 | 3.4 MB    | : 100% 1.0/1 [00:00<00:00,  1.54it/s]\n",
            "r-rstudioapi-0.13    | 281 KB    | : 100% 1.0/1 [00:00<00:00, 12.54it/s]\n",
            "perl-json-4.05       | 54 KB     | : 100% 1.0/1 [00:00<00:00, 18.58it/s]\n",
            "r-r6-2.5.1           | 89 KB     | : 100% 1.0/1 [00:00<00:00, 18.78it/s]\n",
            "xorg-libx11-1.7.2    | 941 KB    | : 100% 1.0/1 [00:00<00:00,  2.98it/s]\n",
            "ipykernel-6.12.1     | 184 KB    | : 100% 1.0/1 [00:00<00:00,  6.67it/s]\n",
            "pandas-1.2.5         | 12.1 MB   | : 100% 1.0/1 [00:04<00:00,  4.62s/it]\n",
            "q2cli-2022.2.0       | 88 KB     | : 100% 1.0/1 [00:00<00:00,  5.11it/s]\n",
            "icu-69.1             | 13.2 MB   | : 100% 1.0/1 [00:02<00:00,  2.72s/it]\n",
            "q2-emperor-2022.2.0  | 15 KB     | : 100% 1.0/1 [00:00<00:00,  5.70it/s]\n",
            "r-backports-1.4.1    | 110 KB    | : 100% 1.0/1 [00:00<00:00, 19.97it/s]\n",
            "brotli-1.0.9         | 18 KB     | : 100% 1.0/1 [00:00<00:00, 27.93it/s]\n",
            "toml-0.10.2          | 18 KB     | : 100% 1.0/1 [00:00<00:00, 25.50it/s]\n",
            "lz4-4.0.0            | 36 KB     | : 100% 1.0/1 [00:00<00:00, 21.06it/s]\n",
            "tzlocal-2.1          | 18 KB     | : 100% 1.0/1 [00:00<00:00, 29.33it/s]\n",
            "mysql-libs-8.0.28    | 1.9 MB    | : 100% 1.0/1 [00:00<00:00,  2.52it/s]\n",
            "perl-archive-tar-2.4 | 33 KB     | : 100% 1.0/1 [00:00<00:00, 21.70it/s]\n",
            "xorg-xproto-7.0.31   | 72 KB     | : 100% 1.0/1 [00:00<00:00, 20.10it/s]\n",
            "r-digest-0.6.29      | 208 KB    | : 100% 1.0/1 [00:00<00:00, 16.81it/s]\n",
            "libxkbcommon-1.0.3   | 581 KB    | : 100% 1.0/1 [00:00<00:00,  6.53it/s]\n",
            "samtools-1.15.1      | 393 KB    | : 100% 1.0/1 [00:00<00:00,  9.31it/s]\n",
            "bioconductor-delayed | 2.5 MB    | : 100% 1.0/1 [00:00<00:00,  2.34it/s]\n",
            "decorator-4.4.2      | 11 KB     | : 100% 1.0/1 [00:00<00:00, 33.26it/s]\n",
            "defusedxml-0.7.1     | 23 KB     | : 100% 1.0/1 [00:00<00:00, 25.30it/s]\n",
            "xopen-1.5.0          | 25 KB     | : 100% 1.0/1 [00:00<00:00, 29.75it/s]\n",
            "giflib-5.2.1         | 80 KB     | : 100% 1.0/1 [00:00<00:00, 18.55it/s]\n",
            "backports-1.0        | 4 KB      | : 100% 1.0/1 [00:00<00:00, 27.60it/s]\n",
            "wcwidth-0.2.5        | 33 KB     | : 100% 1.0/1 [00:00<00:00, 23.40it/s]\n",
            "q2galaxy-2022.2.0    | 53 KB     | : 100% 1.0/1 [00:00<00:00,  6.65it/s]\n",
            "r-tibble-3.1.6       | 774 KB    | : 100% 1.0/1 [00:00<00:00,  5.49it/s]\n",
            "markupsafe-2.1.1     | 22 KB     | : 100% 1.0/1 [00:00<00:00, 23.34it/s]\n",
            "python_abi-3.8       | 4 KB      | : 100% 1.0/1 [00:00<00:00, 26.40it/s]\n",
            "executing-0.8.3      | 18 KB     | : 100% 1.0/1 [00:00<00:00, 23.78it/s]\n",
            "libdeflate-1.10      | 77 KB     | : 100% 1.0/1 [00:00<00:00, 21.23it/s]\n",
            "r-snow-0.4_4         | 113 KB    | : 100% 1.0/1 [00:00<00:00, 11.25it/s]\n",
            "widgetsnbextension-3 | 1.2 MB    | : 100% 1.0/1 [00:00<00:00,  2.61it/s]\n",
            "nest-asyncio-1.5.5   | 9 KB      | : 100% 1.0/1 [00:00<00:00, 24.32it/s]\n",
            "q2-sample-classifier | 627 KB    | : 100% 1.0/1 [00:00<00:00,  4.59it/s]\n",
            "r-fansi-1.0.3        | 323 KB    | : 100% 1.0/1 [00:00<00:00, 14.03it/s]\n",
            "openjdk-11.0.1       | 175.5 MB  | : 100% 1.0/1 [00:31<00:00, 31.60s/it]               \n",
            "r-bitops-1.0_7       | 44 KB     | : 100% 1.0/1 [00:00<00:00, 20.59it/s]\n",
            "r-vegan-2.5_7        | 3.4 MB    | : 100% 1.0/1 [00:00<00:00,  2.02it/s]\n",
            "gst-plugins-base-1.2 | 2.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.15it/s]\n",
            "jinja2-3.1.1         | 99 KB     | : 100% 1.0/1 [00:00<00:00, 14.34it/s]\n",
            "pytest-7.1.1         | 459 KB    | : 100% 1.0/1 [00:00<00:00,  5.02it/s]\n",
            "r-futile.logger-1.4. | 108 KB    | : 100% 1.0/1 [00:00<00:00, 14.28it/s]\n",
            "perl-exporter-5.74   | 19 KB     | : 100% 1.0/1 [00:00<00:00, 24.13it/s]\n",
            "r-rcolorbrewer-1.1_3 | 65 KB     | : 100% 1.0/1 [00:00<00:00, 16.98it/s]\n",
            "bioconductor-zlibbio | 114 KB    | : 100% 1.0/1 [00:00<00:00, 20.23it/s]\n",
            "libzlib-1.2.11       | 60 KB     | : 100% 1.0/1 [00:00<00:00, 17.90it/s]\n",
            "r-rlang-1.0.2        | 1.4 MB    | : 100% 1.0/1 [00:00<00:00,  4.27it/s]\n",
            "lxml-4.8.0           | 1.4 MB    | : 100% 1.0/1 [00:00<00:00,  2.68it/s]\n",
            "pygments-2.11.2      | 796 KB    | : 100% 1.0/1 [00:00<00:00,  3.27it/s]\n",
            "scikit-bio-0.5.6     | 1.3 MB    | : 100% 1.0/1 [00:00<00:00,  1.58it/s]\n",
            "python-fastjsonschem | 243 KB    | : 100% 1.0/1 [00:00<00:00,  2.20it/s]\n",
            "r-utf8-1.2.2         | 162 KB    | : 100% 1.0/1 [00:00<00:00, 12.75it/s]\n",
            "setuptools-59.8.0    | 1017 KB   | : 100% 1.0/1 [00:00<00:00,  2.35it/s]\n",
            "r-scales-1.1.1       | 557 KB    | : 100% 1.0/1 [00:00<00:00,  9.35it/s]\n",
            "pyopenssl-22.0.0     | 49 KB     | : 100% 1.0/1 [00:00<00:00, 24.45it/s]\n",
            "numba-0.53.1         | 3.7 MB    | : 100% 1.0/1 [00:01<00:00,  1.33s/it]\n",
            "importlib-metadata-4 | 33 KB     | : 100% 1.0/1 [00:00<00:00, 23.71it/s]\n",
            "pixman-0.40.0        | 627 KB    | : 100% 1.0/1 [00:00<00:00,  5.05it/s]\n",
            "stack_data-0.2.0     | 21 KB     | : 100% 1.0/1 [00:00<00:00, 21.63it/s]\n",
            "nbconvert-6.4.5      | 6 KB      | : 100% 1.0/1 [00:00<00:00, 31.75it/s]\n",
            "r-jpeg-0.1_9         | 52 KB     | : 100% 1.0/1 [00:00<00:00, 15.41it/s]\n",
            "natsort-8.1.0        | 35 KB     | : 100% 1.0/1 [00:00<00:00, 20.31it/s]\n",
            "perl-json-xs-2.34    | 64 KB     | : 100% 1.0/1 [00:00<00:00, 18.11it/s]\n",
            "libglib-2.70.2       | 3.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.20it/s]\n",
            "libllvm10-10.0.1     | 26.4 MB   | : 100% 1.0/1 [00:07<00:00,  7.11s/it]               \n",
            "threadpoolctl-3.1.0  | 18 KB     | : 100% 1.0/1 [00:00<00:00, 25.11it/s]\n",
            "libopenblas-0.3.20   | 10.1 MB   | : 100% 1.0/1 [00:02<00:00,  2.76s/it]\n",
            "r-hwriter-1.3.2.1    | 122 KB    | : 100% 1.0/1 [00:00<00:00, 10.53it/s]\n",
            "perl-common-sense-3. | 13 KB     | : 100% 1.0/1 [00:00<00:00, 24.02it/s]\n",
            "numpy-1.22.3         | 6.8 MB    | : 100% 1.0/1 [00:02<00:00,  2.35s/it]\n",
            "pcre-8.45            | 253 KB    | : 100% 1.0/1 [00:00<00:00, 10.40it/s]\n",
            "gneiss-0.4.6         | 41 KB     | : 100% 1.0/1 [00:00<00:00, 16.51it/s]\n",
            "pbzip2-1.1.13        | 114 KB    | : 100% 1.0/1 [00:00<00:00, 20.42it/s]\n",
            "xorg-libxdmcp-1.1.3  | 18 KB     | : 100% 1.0/1 [00:00<00:00, 20.04it/s]\n",
            "r-viridislite-0.4.0  | 1.3 MB    | : 100% 1.0/1 [00:00<00:00,  4.86it/s]\n",
            "pycosat-0.6.3        | 108 KB    | : 100% 1.0/1 [00:00<00:00, 17.93it/s]\n",
            "libgomp-11.2.0       | 432 KB    | : 100% 1.0/1 [00:00<00:00,  8.42it/s]\n",
            "r-lattice-0.20_45    | 1.1 MB    | : 100% 1.0/1 [00:00<00:00,  5.33it/s]\n",
            "bioconductor-biostri | 13.9 MB   | : 100% 1.0/1 [00:02<00:00,  2.24s/it]\n",
            "alsa-lib-1.2.3       | 560 KB    | : 100% 1.0/1 [00:00<00:00,  6.74it/s]\n",
            "debugpy-1.5.1        | 2.0 MB    | : 100% 1.0/1 [00:00<00:00,  1.19it/s]\n",
            "tomli-2.0.1          | 16 KB     | : 100% 1.0/1 [00:00<00:00, 23.06it/s]\n",
            "perl-io-compress-2.1 | 83 KB     | : 100% 1.0/1 [00:00<00:00, 11.01it/s]\n",
            "bioconductor-biocpar | 1.4 MB    | : 100% 1.0/1 [00:00<00:00,  4.09it/s]\n",
            "libgfortran5-11.2.0  | 1.7 MB    | : 100% 1.0/1 [00:00<00:00,  2.19it/s]\n",
            "perl-compress-raw-zl | 75 KB     | : 100% 1.0/1 [00:00<00:00, 24.49it/s]\n",
            "ipywidgets-7.7.0     | 103 KB    | : 100% 1.0/1 [00:00<00:00, 18.23it/s]\n",
            "gcc_linux-64-10.3.0  | 25 KB     | : 100% 1.0/1 [00:00<00:00, 11.70it/s]\n",
            "perl-io-zlib-1.11    | 11 KB     | : 100% 1.0/1 [00:00<00:00, 16.39it/s]\n",
            "perl-pathtools-3.75  | 42 KB     | : 100% 1.0/1 [00:00<00:00, 19.77it/s]\n",
            "prometheus_client-0. | 49 KB     | : 100% 1.0/1 [00:00<00:00, 12.04it/s]\n",
            "fribidi-1.0.10       | 113 KB    | : 100% 1.0/1 [00:00<00:00, 10.90it/s]\n",
            "jupyter_core-4.9.2   | 81 KB     | : 100% 1.0/1 [00:00<00:00, 12.77it/s]\n",
            "zlib-1.2.11          | 88 KB     | : 100% 1.0/1 [00:00<00:00, 16.44it/s]\n",
            "cycler-0.11.0        | 10 KB     | : 100% 1.0/1 [00:00<00:00, 27.64it/s]\n",
            "pure_eval-0.2.2      | 14 KB     | : 100% 1.0/1 [00:00<00:00, 23.79it/s]\n",
            "ca-certificates-2021 | 139 KB    | : 100% 1.0/1 [00:00<00:00, 17.37it/s]\n",
            "bioconductor-summari | 2.7 MB    | : 100% 1.0/1 [00:00<00:00,  1.93it/s]\n",
            "mamba-0.9.0          | 644 KB    | : 100% 1.0/1 [00:00<00:00,  4.46it/s]\n",
            "perl-extutils-makema | 145 KB    | : 100% 1.0/1 [00:00<00:00,  9.68it/s]\n",
            "dbus-1.13.6          | 604 KB    | : 100% 1.0/1 [00:00<00:00,  4.08it/s]\n",
            "libgcc-ng-11.2.0     | 911 KB    | : 100% 1.0/1 [00:00<00:00,  3.72it/s]\n",
            "pyrsistent-0.18.1    | 92 KB     | : 100% 1.0/1 [00:00<00:00,  9.12it/s]\n",
            "expat-2.4.8          | 187 KB    | : 100% 1.0/1 [00:00<00:00, 11.37it/s]\n",
            "libxslt-1.1.33       | 521 KB    | : 100% 1.0/1 [00:00<00:00,  4.62it/s]\n",
            "libbrotlidec-1.0.9   | 33 KB     | : 100% 1.0/1 [00:00<00:00, 18.99it/s]\n",
            "hmmer-3.1b2          | 5.5 MB    | : 100% 1.0/1 [00:01<00:00,  1.33s/it]\n",
            "r-latticeextra-0.6_2 | 2.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.58it/s]\n",
            "raxml-8.2.12         | 3.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.20it/s]               \n",
            "sqlite-3.37.1        | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  2.98it/s]\n",
            "bioconductor-s4vecto | 2.2 MB    | : 100% 1.0/1 [00:00<00:00,  2.56it/s]\n",
            "libpq-14.2           | 3.0 MB    | : 100% 1.0/1 [00:00<00:00,  1.16it/s]\n",
            "python-dateutil-2.8. | 240 KB    | : 100% 1.0/1 [00:00<00:00, 15.37it/s]\n",
            "gfortran_impl_linux- | 14.7 MB   | : 100% 1.0/1 [00:03<00:00,  3.95s/it]\n",
            "dendropy-4.5.2       | 308 KB    | : 100% 1.0/1 [00:00<00:00,  4.68it/s]\n",
            "r-nlme-3.1_157       | 2.3 MB    | : 100% 1.0/1 [00:00<00:00,  2.38it/s]\n",
            "pyqt-5.12.3          | 22 KB     | : 100% 1.0/1 [00:00<00:00, 24.79it/s]\n",
            "sortmerna-2.0        | 337 KB    | : 100% 1.0/1 [00:00<00:00, 10.51it/s]\n",
            "r-rematch2-2.1.2     | 52 KB     | : 100% 1.0/1 [00:00<00:00, 24.97it/s]\n",
            "jedi-0.18.1          | 999 KB    | : 100% 1.0/1 [00:00<00:00,  1.51it/s]\n",
            "pycparser-2.21       | 100 KB    | : 100% 1.0/1 [00:00<00:00, 11.07it/s]\n",
            "typing_extensions-4. | 29 KB     | : 100% 1.0/1 [00:00<00:00, 23.80it/s]\n",
            "r-rcurl-1.98_1.6     | 967 KB    | : 100% 1.0/1 [00:00<00:00,  3.27it/s]\n",
            "q2-phylogeny-2022.2. | 51 KB     | : 100% 1.0/1 [00:00<00:00,  6.03it/s]\n",
            "pango-1.50.6         | 456 KB    | : 100% 1.0/1 [00:00<00:00,  4.71it/s]\n",
            "libevent-2.1.10      | 1.1 MB    | : 100% 1.0/1 [00:00<00:00,  1.65it/s]\n",
            "jupyterlab_pygments- | 17 KB     | : 100% 1.0/1 [00:00<00:00, 16.45it/s]\n",
            "backcall-0.2.0       | 13 KB     | : 100% 1.0/1 [00:00<00:00, 20.97it/s]\n",
            "harfbuzz-4.2.0       | 2.0 MB    | : 100% 1.0/1 [00:00<00:00,  1.60it/s]\n",
            "jupyterlab_widgets-1 | 133 KB    | : 100% 1.0/1 [00:00<00:00,  6.75it/s]\n",
            "pyqt-impl-5.12.3     | 5.9 MB    | : 100% 1.0/1 [00:02<00:00,  2.19s/it]\n",
            "q2-feature-classifie | 141 KB    | : 100% 1.0/1 [00:00<00:00,  4.40it/s]\n",
            "nbconvert-core-6.4.5 | 447 KB    | : 100% 1.0/1 [00:00<00:00,  4.96it/s]\n",
            "xmltodict-0.12.0     | 11 KB     | : 100% 1.0/1 [00:00<00:00, 21.09it/s]\n",
            "libuuid-2.32.1       | 26 KB     | : 100% 1.0/1 [00:00<00:00, 20.78it/s]\n",
            "pthread-stubs-0.4    | 5 KB      | : 100% 1.0/1 [00:00<00:00, 35.62it/s]\n",
            "pyqtwebengine-5.12.1 | 175 KB    | : 100% 1.0/1 [00:00<00:00, 10.72it/s]\n",
            "bioconductor-rsamtoo | 3.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.20it/s]\n",
            "r-brio-1.1.3         | 40 KB     | : 100% 1.0/1 [00:00<00:00, 18.70it/s]\n",
            "bioconductor-rhtslib | 1.8 MB    | : 100% 1.0/1 [00:00<00:00,  1.78it/s]\n",
            "parso-0.8.3          | 69 KB     | : 100% 1.0/1 [00:00<00:00, 16.92it/s]\n",
            "matplotlib-base-3.5. | 7.4 MB    | : 100% 1.0/1 [00:02<00:00,  2.16s/it]\n",
            "r-matrix-1.4_1       | 4.8 MB    | : 100% 1.0/1 [00:01<00:00,  1.02s/it]\n",
            "hdf5-1.12.1          | 3.5 MB    | : 100% 1.0/1 [00:01<00:00,  1.01s/it]\n",
            "ptyprocess-0.7.0     | 16 KB     | : 100% 1.0/1 [00:00<00:00, 22.62it/s]\n",
            "Preparing transaction: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Verifying transaction: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- Enabling notebook extension jupyter-js-widgets/extension...\n",
            "Paths used for configuration of notebook: \n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.d/jupyterlab-plotly.json\n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.d/widgetsnbextension.json\n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.json\n",
            "Paths used for configuration of notebook: \n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.d/jupyterlab-plotly.json\n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.d/widgetsnbextension.json\n",
            "      - Validating: \u001b[32mOK\u001b[0m\n",
            "Paths used for configuration of notebook: \n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.d/jupyterlab-plotly.json\n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.d/widgetsnbextension.json\n",
            "    \t/usr/local/etc/jupyter/nbconfig/notebook.json\n",
            "\n",
            "\b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "#\n",
            "# To activate this environment, use\n",
            "#\n",
            "#     $ conda activate base\n",
            "#\n",
            "# To deactivate an active environment, use\n",
            "#\n",
            "#     $ conda deactivate\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!mamba env update -n base -f qiime2-2022.2-py38-linux-conda.yml"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MUkY2ILML2o-",
        "outputId": "5c854174-cb71-4c50-cf70-4a51621efd1c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/sample_data\n"
          ]
        }
      ],
      "source": [
        "cd /content/sample_data"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "CruNZFIOu_sY"
      },
      "source": [
        "STOP AND SKIP TO AFTER THE DENOISE STEP"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "id": "CNZyH96d-X8V"
      },
      "outputs": [],
      "source": [
        "!echo -e \"sample-id\\tabsolute-filepath\" > manifest.tsv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QZXl-pQf_YT7",
        "outputId": "a41ca59b-0f22-427c-9158-f07472c44832"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "DRR079323_1.fastq.gz  DRR079359_1.fastq.gz  DRR079395_1.fastq.gz\n",
            "DRR079327_1.fastq.gz  DRR079363_1.fastq.gz  DRR079399_1.fastq.gz\n",
            "DRR079329_1.fastq.gz  DRR079365_1.fastq.gz  DRR079401_1.fastq.gz\n",
            "DRR079333_1.fastq.gz  DRR079369_1.fastq.gz  DRR079405_1.fastq.gz\n",
            "DRR079335_1.fastq.gz  DRR079371_1.fastq.gz  DRR079407_1.fastq.gz\n",
            "DRR079339_1.fastq.gz  DRR079375_1.fastq.gz  DRR079411_1.fastq.gz\n",
            "DRR079341_1.fastq.gz  DRR079377_1.fastq.gz  DRR079413_1.fastq.gz\n",
            "DRR079345_1.fastq.gz  DRR079381_1.fastq.gz  DRR079417_1.fastq.gz\n",
            "DRR079347_1.fastq.gz  DRR079383_1.fastq.gz  DRR079419_1.fastq.gz\n",
            "DRR079351_1.fastq.gz  DRR079387_1.fastq.gz  DRR079423_1.fastq.gz\n",
            "DRR079353_1.fastq.gz  DRR079389_1.fastq.gz  DRR079425_1.fastq.gz\n",
            "DRR079357_1.fastq.gz  DRR079393_1.fastq.gz  DRR079429_1.fastq.gz\n"
          ]
        }
      ],
      "source": [
        "ls *.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {
        "id": "jWi6gzHqMEaV"
      },
      "outputs": [],
      "source": [
        "%%bash\n",
        "for f in `ls *fastq.gz | cut -d '.' -f 1 | sort -u`; do n=`basename $f`; echo -e \"${n}\\t$PWD/$f\".fastq.gz\" \"; done >> manifest.tsv "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "q2Cu1CjNMStz",
        "outputId": "6dced92e-9433-4897-9c94-29c775b6f7ce"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mImported manifest.tsv as SingleEndFastqManifestPhred33V2 to se-demux.qza\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime tools import \\\n",
        "  --input-path manifest.tsv \\\n",
        "  --type 'SampleData[SequencesWithQuality]' \\\n",
        "  --input-format SingleEndFastqManifestPhred33V2 \\\n",
        "  --output-path se-demux.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0VK0c6qWNYbx",
        "outputId": "23fa2f73-33c7-4f14-b5c8-8ed4294c14a8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33mQIIME is caching your current deployment for improved performance. This may take a few moments and should only happen once per deployment.\u001b[0m\n",
            "\u001b[32mSaved Visualization to: se-demux.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime demux summarize \\\n",
        "    --i-data  se-demux.qza \\\n",
        "    --o-visualization se-demux.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mo2viZ-2NhOn",
        "outputId": "3a74e348-92e5-4755-a220-05bf5886d27b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved FeatureTable[Frequency] to: table-dada2.qza\u001b[0m\n",
            "\u001b[32mSaved FeatureData[Sequence] to: rep-seqs-dada2.qza\u001b[0m\n",
            "\u001b[32mSaved SampleData[DADA2Stats] to: stats-dada2.qza\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime dada2 denoise-single \\\n",
        "  --i-demultiplexed-seqs se-demux.qza \\\n",
        "  --p-trunc-len 400 \\\n",
        "  --o-representative-sequences rep-seqs-dada2.qza \\\n",
        "  --o-table table-dada2.qza \\\n",
        "  --o-denoising-stats stats-dada2.qza"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "8INOnE3MvIEU"
      },
      "source": [
        "START HERE AGAIN"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 13,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9FaG-x0Jjlkh",
        "outputId": "f40d7c6b-b846-4c83-dd97-a83ca3eab0d2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: table-dada2.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime feature-table summarize \\\n",
        "    --i-table table-dada2.qza \\\n",
        "    --m-sample-metadata-file metadata1.txt \\\n",
        "    --o-visualization table-dada2.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vUVbVXuW-o-B",
        "outputId": "41124f2b-1971-4eb3-872f-ebf7a1b49e18"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: rep-seqs-dada2.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime feature-table tabulate-seqs \\\n",
        "    --i-data rep-seqs-dada2.qza \\\n",
        "    --o-visualization rep-seqs-dada2.qzv"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JsF97Gp-DJdj"
      },
      "source": [
        "#Phylogenetics\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-JgHT4KQvCoF",
        "outputId": "7e5aef92-e83e-4e13-9186-ffcdff660d1b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved FeatureData[AlignedSequence] to: aligned-rep-seqs.qza\u001b[0m\n",
            "\u001b[32mSaved FeatureData[AlignedSequence] to: masked-aligned-rep-seqs.qza\u001b[0m\n",
            "\u001b[32mSaved Phylogeny[Unrooted] to: unrooted-tree.qza\u001b[0m\n",
            "\u001b[32mSaved Phylogeny[Rooted] to: rooted-tree.qza\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime phylogeny align-to-tree-mafft-fasttree \\\n",
        "  --i-sequences rep-seqs-dada2.qza  \\\n",
        "  --o-alignment aligned-rep-seqs.qza \\\n",
        "  --o-masked-alignment masked-aligned-rep-seqs.qza \\\n",
        "  --o-tree unrooted-tree.qza \\\n",
        "  --o-rooted-tree rooted-tree.qza"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "gDYqeml5LRKY"
      },
      "source": [
        "#Taxonomy"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 16,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0JEQm-Kx-GYD",
        "outputId": "b5eb5dfd-ec7b-4bfe-8bfb-0ce55bb80348"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2022-04-11 21:11:47--  https://data.qiime2.org/2022.2/common/gg-13-8-99-nb-classifier.qza\n",
            "Resolving data.qiime2.org (data.qiime2.org)... 54.200.1.12\n",
            "Connecting to data.qiime2.org (data.qiime2.org)|54.200.1.12|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 FOUND\n",
            "Location: https://s3-us-west-2.amazonaws.com/qiime2-data/2022.2/common/gg-13-8-99-nb-classifier.qza [following]\n",
            "--2022-04-11 21:11:47--  https://s3-us-west-2.amazonaws.com/qiime2-data/2022.2/common/gg-13-8-99-nb-classifier.qza\n",
            "Resolving s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)... 52.218.183.144\n",
            "Connecting to s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)|52.218.183.144|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 104512483 (100M) [binary/octet-stream]\n",
            "Saving to: ‘gg-13-8-99-nb-classifier.qza’\n",
            "\n",
            "gg-13-8-99-nb-class 100%[===================>]  99.67M  31.1MB/s    in 3.2s    \n",
            "\n",
            "2022-04-11 21:11:51 (31.1 MB/s) - ‘gg-13-8-99-nb-classifier.qza’ saved [104512483/104512483]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!wget \\\n",
        "  -O \"gg-13-8-99-nb-classifier.qza\" \\\n",
        "  \"https://data.qiime2.org/2022.2/common/gg-13-8-99-nb-classifier.qza\""
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 17,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GUwl5soVigUY",
        "outputId": "3bc08c31-1a10-4d99-d619-595e8002608a"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved FeatureData[Taxonomy] to: taxonomy.qza\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime feature-classifier classify-sklearn \\\n",
        "  --i-reads rep-seqs-dada2.qza \\\n",
        "  --i-classifier gg-13-8-99-nb-classifier.qza \\\n",
        "  --o-classification taxonomy.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 18,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "E9_LmA66ZHsD",
        "outputId": "222f4397-8ebf-4ecc-ec18-c51826660a8e"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: taxonomy.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime metadata tabulate \\\n",
        "  --m-input-file taxonomy.qza \\\n",
        "  --o-visualization taxonomy.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 19,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tm9Z7ddWaew1",
        "outputId": "c799d148-cee2-4fc3-e985-81b7a71c1ce8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved FeatureTable[Frequency] to: table_taxa.qza\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime feature-table filter-samples \\\n",
        "  --i-table table-dada2.qza \\\n",
        "  --p-min-frequency 410 \\\n",
        "  --o-filtered-table table_taxa.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 20,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ee39bm5MPopK",
        "outputId": "a5efc539-56c6-4d47-fa3c-436fcdb72a22"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: taxa_barplot.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime taxa barplot \\\n",
        "  --i-table table_taxa.qza \\\n",
        "  --i-taxonomy taxonomy.qza \\\n",
        "  --m-metadata-file metadata1.txt \\\n",
        "  --o-visualization taxa_barplot.qzv"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Wbr-lZgYLUDS"
      },
      "source": [
        "#Diversity analysis"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 22,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "p5QNgG8ol_lB",
        "outputId": "574580b6-19f1-4a1d-d9d0-8aaa55595fd3"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: alpha-rarefaction.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime diversity alpha-rarefaction \\\n",
        "  --i-table table-dada2.qza \\\n",
        "  --i-phylogeny rooted-tree.qza \\\n",
        "  --p-max-depth 410 \\\n",
        "  --m-metadata-file metadata1.txt \\\n",
        "  --o-visualization alpha-rarefaction.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 24,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1q_hIlGRXdTn",
        "outputId": "9bc489ef-4eba-4c2f-b55f-2dce33e8f599"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved FeatureTable[Frequency] to: core-metrics-results/rarefied_table.qza\u001b[0m\n",
            "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/faith_pd_vector.qza\u001b[0m\n",
            "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/observed_features_vector.qza\u001b[0m\n",
            "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/shannon_vector.qza\u001b[0m\n",
            "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/evenness_vector.qza\u001b[0m\n",
            "\u001b[32mSaved DistanceMatrix to: core-metrics-results/unweighted_unifrac_distance_matrix.qza\u001b[0m\n",
            "\u001b[32mSaved DistanceMatrix to: core-metrics-results/weighted_unifrac_distance_matrix.qza\u001b[0m\n",
            "\u001b[32mSaved DistanceMatrix to: core-metrics-results/jaccard_distance_matrix.qza\u001b[0m\n",
            "\u001b[32mSaved DistanceMatrix to: core-metrics-results/bray_curtis_distance_matrix.qza\u001b[0m\n",
            "\u001b[32mSaved PCoAResults to: core-metrics-results/unweighted_unifrac_pcoa_results.qza\u001b[0m\n",
            "\u001b[32mSaved PCoAResults to: core-metrics-results/weighted_unifrac_pcoa_results.qza\u001b[0m\n",
            "\u001b[32mSaved PCoAResults to: core-metrics-results/jaccard_pcoa_results.qza\u001b[0m\n",
            "\u001b[32mSaved PCoAResults to: core-metrics-results/bray_curtis_pcoa_results.qza\u001b[0m\n",
            "\u001b[32mSaved Visualization to: core-metrics-results/unweighted_unifrac_emperor.qzv\u001b[0m\n",
            "\u001b[32mSaved Visualization to: core-metrics-results/weighted_unifrac_emperor.qzv\u001b[0m\n",
            "\u001b[32mSaved Visualization to: core-metrics-results/jaccard_emperor.qzv\u001b[0m\n",
            "\u001b[32mSaved Visualization to: core-metrics-results/bray_curtis_emperor.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "#Core metrics\n",
        "!qiime diversity core-metrics-phylogenetic \\\n",
        "  --i-phylogeny rooted-tree.qza \\\n",
        "  --i-table table-dada2.qza \\\n",
        "  --p-sampling-depth 410 \\\n",
        "  --m-metadata-file metadata1.txt \\\n",
        "  --output-dir core-metrics-results"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 26,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "SMvjOWkm7tdo",
        "outputId": "75d94305-79c2-4f61-f90c-bf2551c4a902"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: core-metrics-results/faith_pd-group-significance.qzv\u001b[0m\n",
            "\u001b[0m\u001b[32mSaved Visualization to: core-metrics-results/evenness-group-significance.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime diversity alpha-group-significance \\\n",
        "  --i-alpha-diversity /content/sample_data/core-metrics-results/faith_pd_vector.qza\\\n",
        "  --m-metadata-file metadata1.txt \\\n",
        "  --o-visualization core-metrics-results/faith_pd-group-significance.qzv\n",
        "\n",
        "!qiime diversity alpha-group-significance \\\n",
        "  --i-alpha-diversity core-metrics-results/evenness_vector.qza \\\n",
        "  --m-metadata-file metadata1.txt \\\n",
        "  --o-visualization core-metrics-results/evenness-group-significance.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 27,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4wmPq-ZFQUWG",
        "outputId": "7b8ab1ad-42ef-4b87-c20e-4834c95f9db8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: core-metrics-results/weighted_unifrac-significance.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime diversity beta-group-significance \\\n",
        "  --i-distance-matrix /content/sample_data/core-metrics-results/weighted_unifrac_distance_matrix.qza \\\n",
        "  --m-metadata-file metadata1.txt \\\n",
        "  --m-metadata-column Group  \\\n",
        "  --o-visualization core-metrics-results/weighted_unifrac-significance.qzv \\\n",
        "  --p-pairwise"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "euOdwdy1OwDF"
      },
      "source": [
        "#Tree viz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 28,
      "metadata": {
        "id": "tfgtmduFUU-E",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e24007f2-73a2-4c3e-aaa4-198bbdf7aeec"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting empress\n",
            "  Downloading empress-1.2.0.tar.gz (302 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m302.7/302.7 KB\u001b[0m \u001b[31m15.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.8/site-packages (from empress) (1.22.3)\n",
            "Requirement already satisfied: scipy in /usr/local/lib/python3.8/site-packages (from empress) (1.8.0)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.8/site-packages (from empress) (1.2.5)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.8/site-packages (from empress) (7.1.2)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.8/site-packages (from empress) (3.1.1)\n",
            "Requirement already satisfied: scikit-bio in /usr/local/lib/python3.8/site-packages (from empress) (0.5.6)\n",
            "Requirement already satisfied: biom-format in /usr/local/lib/python3.8/site-packages (from empress) (2.1.10)\n",
            "Collecting iow\n",
            "  Downloading iow-1.0.5.tar.gz (1.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.1/1.1 MB\u001b[0m \u001b[31m40.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: emperor>=1.0.2 in /usr/local/lib/python3.8/site-packages (from empress) (1.0.3)\n",
            "Requirement already satisfied: future in /usr/local/lib/python3.8/site-packages (from emperor>=1.0.2->empress) (0.18.2)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.8/site-packages (from jinja2->empress) (2.1.1)\n",
            "Requirement already satisfied: natsort>=4.0.3 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (8.1.0)\n",
            "Requirement already satisfied: hdmedians>=0.13 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (0.14.2)\n",
            "Requirement already satisfied: CacheControl>=0.11.5 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (0.12.10)\n",
            "Requirement already satisfied: scikit-learn>=0.19.1 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (0.24.1)\n",
            "Requirement already satisfied: decorator>=3.4.2 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (4.4.2)\n",
            "Requirement already satisfied: IPython>=3.2.0 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (8.2.0)\n",
            "Requirement already satisfied: matplotlib>=1.4.3 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (3.5.1)\n",
            "Requirement already satisfied: lockfile>=0.10.2 in /usr/local/lib/python3.8/site-packages (from scikit-bio->empress) (0.12.2)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.8/site-packages (from pandas->empress) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.8/site-packages (from pandas->empress) (2022.1)\n",
            "Requirement already satisfied: h5py in /usr/local/lib/python3.8/site-packages (from biom-format->empress) (3.6.0)\n",
            "Requirement already satisfied: cython>=0.29 in /usr/local/lib/python3.8/site-packages (from biom-format->empress) (0.29.28)\n",
            "Requirement already satisfied: six>=1.10.0 in /usr/local/lib/python3.8/site-packages (from biom-format->empress) (1.16.0)\n",
            "Requirement already satisfied: nose>=1.3.7 in /usr/local/lib/python3.8/site-packages (from iow->empress) (1.3.7)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.8/site-packages (from CacheControl>=0.11.5->scikit-bio->empress) (2.27.1)\n",
            "Requirement already satisfied: msgpack>=0.5.2 in /usr/local/lib/python3.8/site-packages (from CacheControl>=0.11.5->scikit-bio->empress) (1.0.3)\n",
            "Requirement already satisfied: pygments>=2.4.0 in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (2.11.2)\n",
            "Requirement already satisfied: jedi>=0.16 in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (0.18.1)\n",
            "Requirement already satisfied: matplotlib-inline in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (0.1.3)\n",
            "Requirement already satisfied: pexpect>4.3 in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (4.8.0)\n",
            "Requirement already satisfied: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (3.0.29)\n",
            "Requirement already satisfied: pickleshare in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (0.7.5)\n",
            "Requirement already satisfied: setuptools>=18.5 in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (59.8.0)\n",
            "Requirement already satisfied: backcall in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (0.2.0)\n",
            "Requirement already satisfied: traitlets>=5 in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (5.1.1)\n",
            "Requirement already satisfied: stack-data in /usr/local/lib/python3.8/site-packages (from IPython>=3.2.0->scikit-bio->empress) (0.2.0)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.4.3->scikit-bio->empress) (1.4.2)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.4.3->scikit-bio->empress) (0.11.0)\n",
            "Requirement already satisfied: pyparsing>=2.2.1 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.4.3->scikit-bio->empress) (3.0.7)\n",
            "Requirement already satisfied: pillow>=6.2.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.4.3->scikit-bio->empress) (9.1.0)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.4.3->scikit-bio->empress) (21.3)\n",
            "Requirement already satisfied: fonttools>=4.22.0 in /usr/local/lib/python3.8/site-packages (from matplotlib>=1.4.3->scikit-bio->empress) (4.31.2)\n",
            "Requirement already satisfied: joblib>=0.11 in /usr/local/lib/python3.8/site-packages (from scikit-learn>=0.19.1->scikit-bio->empress) (1.1.0)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.8/site-packages (from scikit-learn>=0.19.1->scikit-bio->empress) (3.1.0)\n",
            "Requirement already satisfied: parso<0.9.0,>=0.8.0 in /usr/local/lib/python3.8/site-packages (from jedi>=0.16->IPython>=3.2.0->scikit-bio->empress) (0.8.3)\n",
            "Requirement already satisfied: ptyprocess>=0.5 in /usr/local/lib/python3.8/site-packages (from pexpect>4.3->IPython>=3.2.0->scikit-bio->empress) (0.7.0)\n",
            "Requirement already satisfied: wcwidth in /usr/local/lib/python3.8/site-packages (from prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0->IPython>=3.2.0->scikit-bio->empress) (0.2.5)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.8/site-packages (from requests->CacheControl>=0.11.5->scikit-bio->empress) (3.3)\n",
            "Requirement already satisfied: charset-normalizer~=2.0.0 in /usr/local/lib/python3.8/site-packages (from requests->CacheControl>=0.11.5->scikit-bio->empress) (2.0.12)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.8/site-packages (from requests->CacheControl>=0.11.5->scikit-bio->empress) (2021.10.8)\n",
            "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.8/site-packages (from requests->CacheControl>=0.11.5->scikit-bio->empress) (1.26.9)\n",
            "Requirement already satisfied: asttokens in /usr/local/lib/python3.8/site-packages (from stack-data->IPython>=3.2.0->scikit-bio->empress) (2.0.5)\n",
            "Requirement already satisfied: executing in /usr/local/lib/python3.8/site-packages (from stack-data->IPython>=3.2.0->scikit-bio->empress) (0.8.3)\n",
            "Requirement already satisfied: pure-eval in /usr/local/lib/python3.8/site-packages (from stack-data->IPython>=3.2.0->scikit-bio->empress) (0.2.2)\n",
            "Building wheels for collected packages: empress, iow\n",
            "  Building wheel for empress (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for empress: filename=empress-1.2.0-py3-none-any.whl size=294976 sha256=db9525329b07a12ce7fe7c86991e306d7f69e9f791d725168afcd23ebecbb4d1\n",
            "  Stored in directory: /root/.cache/pip/wheels/26/3a/61/82cee33879f35e078fa8fb9fe7f941f57b1c2967885ca326f4\n",
            "  Building wheel for iow (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for iow: filename=iow-1.0.5-cp38-cp38-linux_x86_64.whl size=959560 sha256=fc3a9989152383b37c17f1a6824d199249e575d7c1d0ca3f3f541b40b852c828\n",
            "  Stored in directory: /root/.cache/pip/wheels/48/12/a6/dda1ea3b0f75b33b86da209015fa7ccc0fcd2a99d7e4d6225c\n",
            "Successfully built empress iow\n",
            "Installing collected packages: iow, empress\n",
            "Successfully installed empress-1.2.0 iow-1.0.5\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!pip install empress"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 29,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "KulFCAJpUU2q",
        "outputId": "7193f7d9-cab1-4c52-8e51-ec2b023b35a7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33mQIIME is caching your current deployment for improved performance. This may take a few moments and should only happen once per deployment.\u001b[0m\n",
            "\u001b[0mUsage: \u001b[94mqiime empress\u001b[0m [OPTIONS] COMMAND\n",
            "                     [ARGS]...\n",
            "\n",
            "  Description: This QIIME 2 plugin wraps Empress\n",
            "  and supports interactive visualization of\n",
            "  phylogenetic trees.\n",
            "\n",
            "  Plugin website:\n",
            "  http://github.com/biocore/empress\n",
            "\n",
            "  Getting user support: Please post to the QIIME\n",
            "  2 forum for help with this plugin:\n",
            "  https://forum.qiime2.org\n",
            "\n",
            "\u001b[1mOptions\u001b[0m:\n",
            "  \u001b[94m--version\u001b[0m            Show the version and exit.\n",
            "  \u001b[94m--example-data\u001b[0m PATH  Write example data and\n",
            "                       exit.\n",
            "  \u001b[94m--citations\u001b[0m          Show citations and exit.\n",
            "  \u001b[94m--help\u001b[0m               Show this message and\n",
            "                       exit.\n",
            "\n",
            "\u001b[1mCommands\u001b[0m:\n",
            "  \u001b[94mcommunity-plot\u001b[0m  Visualize phylogenies and\n",
            "                  community data with Empress\n",
            "                  (and, optionally, Emperor)\n",
            "\n",
            "  \u001b[94mtree-plot\u001b[0m       Visualize phylogenies with\n",
            "                  Empress\n"
          ]
        }
      ],
      "source": [
        "!qiime dev refresh-cache\n",
        "!qiime empress --help"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 30,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "IZO-CEs5TuM1",
        "outputId": "0f95651a-18e3-4931-d27d-2ee17bbe2cd3"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[32mSaved Visualization to: tree-viz.qzv\u001b[0m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!qiime empress tree-plot \\\n",
        "    --i-tree rooted-tree.qza \\\n",
        "    --m-feature-metadata-file taxonomy.qza \\\n",
        "    --o-visualization tree-viz.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 31,
      "metadata": {
        "id": "ueZnRELm8doA"
      },
      "outputs": [],
      "source": [
        "#make a folder in your google drive called bioc0023data\n",
        "!cp /content/sample_data/*.qz* /content/drive/MyDrive/oralmicrobiomeresults/\n",
        "!cp /content/sample_data/core-metrics-results/*.* /content/drive/MyDrive/oralmicrobiomeresults/"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "mUIeKuZ7D53b"
      },
      "source": [
        "#Alpha diversity START HERE AGAIN"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 44,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "AeYRFIJs3Fha",
        "outputId": "973d2ad2-d74b-40e5-cce2-d54d6b17d2e6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  /content/sample_data/core-metrics-results/shannon_vector.qza\n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/checksums.md5? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/checksums.md5  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/VERSION  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/metadata.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/metadata.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/citations.bib? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/citations.bib  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/VERSION  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/metadata.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/metadata.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/citations.bib? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/citations.bib  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/VERSION  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/metadata.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/metadata.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/action/action.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/action/action.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/citations.bib? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/citations.bib  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/VERSION  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/metadata.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/metadata.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/action/action.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/30c1aa00-91c0-4164-a509-216d2a819088/action/action.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/citations.bib? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/citations.bib  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/VERSION  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/metadata.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/metadata.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/action/metadata.tsv? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/action/metadata.tsv  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/action/action.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/7835cb9d-4432-42cb-b06b-3d5d43e5f437/action/action.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/citations.bib? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/citations.bib  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/VERSION  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/metadata.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/metadata.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/action/action.yaml? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/action/action.yaml  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/citations.bib? [y]es, [n]o, [A]ll, [N]one, [r]ename: y\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/citations.bib  \n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: a\n",
            "error:  invalid response [a]\n",
            "replace aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/VERSION? [y]es, [n]o, [A]ll, [N]one, [r]ename: A\n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/citations.bib  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/citations.bib  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/citations.bib  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/citations.bib  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/citations.bib  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/citations.bib  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/VERSION  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/metadata.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/action/metadata.tsv  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/provenance/action/action.yaml  \n",
            "  inflating: aaf7d92f-c65f-4162-960d-c211fbb2cada/data/alpha-diversity.tsv  \n"
          ]
        }
      ],
      "source": [
        "!unzip /content/sample_data/core-metrics-results/shannon_vector.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 50,
      "metadata": {
        "id": "SIQakA5Y39sN"
      },
      "outputs": [],
      "source": [
        "!cp /content/sample_data/aaf7d92f-c65f-4162-960d-c211fbb2cada/data/alpha-diversity.tsv ./"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 51,
      "metadata": {
        "id": "CqOUUCaV4Imf"
      },
      "outputs": [],
      "source": [
        "import pandas as pd"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 52,
      "metadata": {
        "id": "eWfbQbyM4MLk"
      },
      "outputs": [],
      "source": [
        "shannon=pd.read_table(\"/content/sample_data/alpha-diversity.tsv\", index_col=0)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 53,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "uL9Tco3j4c0m",
        "outputId": "f9e7a2db-4f54-49f7-dbdd-0994cd0ed3a7"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "             shannon_entropy\n",
              "DRR079323_1         3.269266\n",
              "DRR079327_1         2.794726\n",
              "DRR079329_1         2.881066\n",
              "DRR079333_1         3.049955\n",
              "DRR079335_1         2.951718\n",
              "DRR079339_1         3.069096\n",
              "DRR079341_1         3.327422\n",
              "DRR079345_1         3.227800\n",
              "DRR079347_1         3.241059\n",
              "DRR079351_1         3.071453\n",
              "DRR079353_1         3.325132\n",
              "DRR079357_1         2.878952\n",
              "DRR079359_1         3.009145\n",
              "DRR079363_1         3.373069\n",
              "DRR079365_1         3.156495\n",
              "DRR079369_1         3.249811\n",
              "DRR079371_1         2.959563\n",
              "DRR079375_1         3.499650\n",
              "DRR079377_1         3.289511\n",
              "DRR079381_1         3.290096\n",
              "DRR079383_1         2.916993\n",
              "DRR079387_1         3.445012\n",
              "DRR079389_1         3.240420\n",
              "DRR079393_1         3.371924\n",
              "DRR079395_1         3.674384\n",
              "DRR079399_1         3.288250\n",
              "DRR079401_1         3.944767\n",
              "DRR079405_1         3.811782\n",
              "DRR079407_1         3.331427\n",
              "DRR079411_1         3.445412\n",
              "DRR079413_1         2.909049\n",
              "DRR079417_1         2.731009\n",
              "DRR079419_1         2.978120\n",
              "DRR079423_1         3.016020\n",
              "DRR079425_1         3.123680\n",
              "DRR079429_1         2.743099"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-d464c421-7c67-4aee-92e9-241ccdfd1e71\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>shannon_entropy</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>DRR079323_1</th>\n",
              "      <td>3.269266</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079327_1</th>\n",
              "      <td>2.794726</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079329_1</th>\n",
              "      <td>2.881066</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079333_1</th>\n",
              "      <td>3.049955</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079335_1</th>\n",
              "      <td>2.951718</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079339_1</th>\n",
              "      <td>3.069096</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079341_1</th>\n",
              "      <td>3.327422</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079345_1</th>\n",
              "      <td>3.227800</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079347_1</th>\n",
              "      <td>3.241059</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079351_1</th>\n",
              "      <td>3.071453</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079353_1</th>\n",
              "      <td>3.325132</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079357_1</th>\n",
              "      <td>2.878952</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079359_1</th>\n",
              "      <td>3.009145</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079363_1</th>\n",
              "      <td>3.373069</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079365_1</th>\n",
              "      <td>3.156495</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079369_1</th>\n",
              "      <td>3.249811</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079371_1</th>\n",
              "      <td>2.959563</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079375_1</th>\n",
              "      <td>3.499650</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079377_1</th>\n",
              "      <td>3.289511</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079381_1</th>\n",
              "      <td>3.290096</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079383_1</th>\n",
              "      <td>2.916993</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079387_1</th>\n",
              "      <td>3.445012</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079389_1</th>\n",
              "      <td>3.240420</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079393_1</th>\n",
              "      <td>3.371924</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079395_1</th>\n",
              "      <td>3.674384</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079399_1</th>\n",
              "      <td>3.288250</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079401_1</th>\n",
              "      <td>3.944767</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079405_1</th>\n",
              "      <td>3.811782</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079407_1</th>\n",
              "      <td>3.331427</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079411_1</th>\n",
              "      <td>3.445412</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079413_1</th>\n",
              "      <td>2.909049</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079417_1</th>\n",
              "      <td>2.731009</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079419_1</th>\n",
              "      <td>2.978120</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079423_1</th>\n",
              "      <td>3.016020</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079425_1</th>\n",
              "      <td>3.123680</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079429_1</th>\n",
              "      <td>2.743099</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d464c421-7c67-4aee-92e9-241ccdfd1e71')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-d464c421-7c67-4aee-92e9-241ccdfd1e71 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-d464c421-7c67-4aee-92e9-241ccdfd1e71');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 53
        }
      ],
      "source": [
        "shannon"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 54,
      "metadata": {
        "id": "NanXdt-_5YWo"
      },
      "outputs": [],
      "source": [
        "shannon.index.names = ['sample-id']"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 55,
      "metadata": {
        "id": "batlB5Vg4n9h"
      },
      "outputs": [],
      "source": [
        "metadata=pd.read_table(\"/content/sample_data/metadata1.txt\", index_col=0)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 56,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "hin4LHbx41WI",
        "outputId": "cc744c21-f32c-40c5-a2ce-80034d6c7cf7"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "            Assay Type  AvgSpotLen    Bases BioProject     BioSample  \\\n",
              "sample-id                                                              \n",
              "DRR079323_1   AMPLICON         400  1202750  PRJDB5388  SAMD00069337   \n",
              "DRR079327_1   AMPLICON         401  1204646  PRJDB5388  SAMD00069341   \n",
              "DRR079329_1   AMPLICON         400  1202374  PRJDB5388  SAMD00069343   \n",
              "DRR079333_1   AMPLICON         400  1201804  PRJDB5388  SAMD00069347   \n",
              "DRR079335_1   AMPLICON         400  1201037  PRJDB5388  SAMD00069349   \n",
              "DRR079339_1   AMPLICON         401  1203454  PRJDB5388  SAMD00069353   \n",
              "DRR079341_1   AMPLICON         402  1206843  PRJDB5388  SAMD00069355   \n",
              "DRR079345_1   AMPLICON         402  1208063  PRJDB5388  SAMD00069359   \n",
              "DRR079347_1   AMPLICON         402  1206011  PRJDB5388  SAMD00069361   \n",
              "DRR079351_1   AMPLICON         402  1206119  PRJDB5388  SAMD00069365   \n",
              "DRR079353_1   AMPLICON         401  1205015  PRJDB5388  SAMD00069367   \n",
              "DRR079357_1   AMPLICON         402  1207998  PRJDB5388  SAMD00069371   \n",
              "DRR079359_1   AMPLICON         402  1208484  PRJDB5388  SAMD00069373   \n",
              "DRR079363_1   AMPLICON         403  1209574  PRJDB5388  SAMD00069377   \n",
              "DRR079365_1   AMPLICON         402  1206531  PRJDB5388  SAMD00069379   \n",
              "DRR079369_1   AMPLICON         402  1208513  PRJDB5388  SAMD00069383   \n",
              "DRR079371_1   AMPLICON         401  1203354  PRJDB5388  SAMD00069385   \n",
              "DRR079375_1   AMPLICON         401  1204724  PRJDB5388  SAMD00069389   \n",
              "DRR079377_1   AMPLICON         402  1207319  PRJDB5388  SAMD00069391   \n",
              "DRR079381_1   AMPLICON         401  1205978  PRJDB5388  SAMD00069395   \n",
              "DRR079383_1   AMPLICON         401  1204780  PRJDB5388  SAMD00069397   \n",
              "DRR079387_1   AMPLICON         401  1205197  PRJDB5388  SAMD00069401   \n",
              "DRR079389_1   AMPLICON         401  1204420  PRJDB5388  SAMD00069403   \n",
              "DRR079393_1   AMPLICON         401  1203747  PRJDB5388  SAMD00069407   \n",
              "DRR079395_1   AMPLICON         402  1206659  PRJDB5388  SAMD00069409   \n",
              "DRR079399_1   AMPLICON         402  1206266  PRJDB5388  SAMD00069413   \n",
              "DRR079401_1   AMPLICON         401  1205145  PRJDB5388  SAMD00069415   \n",
              "DRR079405_1   AMPLICON         402  1208463  PRJDB5388  SAMD00069419   \n",
              "DRR079407_1   AMPLICON         401  1204482  PRJDB5388  SAMD00069421   \n",
              "DRR079411_1   AMPLICON         401  1205234  PRJDB5388  SAMD00069425   \n",
              "DRR079413_1   AMPLICON         402  1208075  PRJDB5388  SAMD00069427   \n",
              "DRR079417_1   AMPLICON         403  1209548  PRJDB5388  SAMD00069431   \n",
              "DRR079419_1   AMPLICON         403  1211194  PRJDB5388  SAMD00069433   \n",
              "DRR079423_1   AMPLICON         402  1208636  PRJDB5388  SAMD00069437   \n",
              "DRR079425_1   AMPLICON         402  1207743  PRJDB5388  SAMD00069439   \n",
              "DRR079429_1   AMPLICON         402  1206717  PRJDB5388  SAMD00069443   \n",
              "\n",
              "                 BioSampleModel    Bytes Center xname Collection_Date Consent  \\\n",
              "sample-id                                                                       \n",
              "DRR079323_1  MIMS.me.human-oral  2904393       UT_COB         missing  public   \n",
              "DRR079327_1  MIMS.me.human-oral  2906270       UT_COB         missing  public   \n",
              "DRR079329_1  MIMS.me.human-oral  2897112       UT_COB         missing  public   \n",
              "DRR079333_1  MIMS.me.human-oral  2894179       UT_COB         missing  public   \n",
              "DRR079335_1  MIMS.me.human-oral  2892993       UT_COB         missing  public   \n",
              "DRR079339_1  MIMS.me.human-oral  2898655       UT_COB         missing  public   \n",
              "DRR079341_1  MIMS.me.human-oral  2893466       UT_COB         missing  public   \n",
              "DRR079345_1  MIMS.me.human-oral  2903400       UT_COB         missing  public   \n",
              "DRR079347_1  MIMS.me.human-oral  2879839       UT_COB         missing  public   \n",
              "DRR079351_1  MIMS.me.human-oral  2905598       UT_COB         missing  public   \n",
              "DRR079353_1  MIMS.me.human-oral  2893969       UT_COB         missing  public   \n",
              "DRR079357_1  MIMS.me.human-oral  2896737       UT_COB         missing  public   \n",
              "DRR079359_1  MIMS.me.human-oral  2949507       UT_COB         missing  public   \n",
              "DRR079363_1  MIMS.me.human-oral  2962968       UT_COB         missing  public   \n",
              "DRR079365_1  MIMS.me.human-oral  2947616       UT_COB         missing  public   \n",
              "DRR079369_1  MIMS.me.human-oral  2960925       UT_COB         missing  public   \n",
              "DRR079371_1  MIMS.me.human-oral  2943481       UT_COB         missing  public   \n",
              "DRR079375_1  MIMS.me.human-oral  2958594       UT_COB         missing  public   \n",
              "DRR079377_1  MIMS.me.human-oral  2930337       UT_COB         missing  public   \n",
              "DRR079381_1  MIMS.me.human-oral  2940426       UT_COB         missing  public   \n",
              "DRR079383_1  MIMS.me.human-oral  2920773       UT_COB         missing  public   \n",
              "DRR079387_1  MIMS.me.human-oral  2943440       UT_COB         missing  public   \n",
              "DRR079389_1  MIMS.me.human-oral  2919435       UT_COB         missing  public   \n",
              "DRR079393_1  MIMS.me.human-oral  2937432       UT_COB         missing  public   \n",
              "DRR079395_1  MIMS.me.human-oral  2928937       UT_COB         missing  public   \n",
              "DRR079399_1  MIMS.me.human-oral  2938816       UT_COB         missing  public   \n",
              "DRR079401_1  MIMS.me.human-oral  2925095       UT_COB         missing  public   \n",
              "DRR079405_1  MIMS.me.human-oral  2940193       UT_COB         missing  public   \n",
              "DRR079407_1  MIMS.me.human-oral  2921729       UT_COB         missing  public   \n",
              "DRR079411_1  MIMS.me.human-oral  2932702       UT_COB         missing  public   \n",
              "DRR079413_1  MIMS.me.human-oral  3026067       UT_COB         missing  public   \n",
              "DRR079417_1  MIMS.me.human-oral  3035504       UT_COB         missing  public   \n",
              "DRR079419_1  MIMS.me.human-oral  3038575       UT_COB         missing  public   \n",
              "DRR079423_1  MIMS.me.human-oral  3041059       UT_COB         missing  public   \n",
              "DRR079425_1  MIMS.me.human-oral  3043535       UT_COB         missing  public   \n",
              "DRR079429_1  MIMS.me.human-oral  3031928       UT_COB         missing  public   \n",
              "\n",
              "             ... LibrarySelection LibrarySource               Organism  \\\n",
              "sample-id    ...                                                         \n",
              "DRR079323_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079327_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079329_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079333_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079335_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079339_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079341_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079345_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079347_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079351_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079353_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079357_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079359_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079363_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079365_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079369_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079371_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079375_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079377_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079381_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079383_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079387_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079389_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079393_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079395_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079399_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079401_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079405_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079407_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079411_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079413_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079417_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079419_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079423_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079425_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "DRR079429_1  ...              PCR   METAGENOMIC  human oral metagenome   \n",
              "\n",
              "            Platform                                      project xname  \\\n",
              "sample-id                                                                 \n",
              "DRR079323_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079327_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079329_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079333_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079335_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079339_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079341_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079345_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079347_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079351_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079353_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079357_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079359_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079363_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079365_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079369_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079371_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079375_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079377_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079381_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079383_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079387_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079389_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079393_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079395_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079399_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079401_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079405_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079407_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079411_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079413_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079417_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079419_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079423_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079425_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079429_1    LS454  Metgenomic analysis of human saliva microbiome...   \n",
              "\n",
              "                      ReleaseDate  Sample xname        sample yname  \\\n",
              "sample-id                                                             \n",
              "DRR079323_1  2018-12-13T00:00:00Z  SAMD00069337  sub1-01-16_16s.sff   \n",
              "DRR079327_1  2018-12-13T00:00:00Z  SAMD00069341  sub1-02-08_16s.sff   \n",
              "DRR079329_1  2018-12-13T00:00:00Z  SAMD00069343  sub1-02-16_16s.sff   \n",
              "DRR079333_1  2018-12-13T00:00:00Z  SAMD00069347  sub1-03-08_16s.sff   \n",
              "DRR079335_1  2018-12-13T00:00:00Z  SAMD00069349  sub1-03-16_16s.sff   \n",
              "DRR079339_1  2018-12-13T00:00:00Z  SAMD00069353  sub1-04-08_16s.sff   \n",
              "DRR079341_1  2018-12-13T00:00:00Z  SAMD00069355  sub2-01-16_16s.sff   \n",
              "DRR079345_1  2018-12-13T00:00:00Z  SAMD00069359  sub2-02-08_16s.sff   \n",
              "DRR079347_1  2018-12-13T00:00:00Z  SAMD00069361  sub2-02-16_16s.sff   \n",
              "DRR079351_1  2018-12-13T00:00:00Z  SAMD00069365  sub2-03-08_16s.sff   \n",
              "DRR079353_1  2018-12-13T00:00:00Z  SAMD00069367  sub2-03-16_16s.sff   \n",
              "DRR079357_1  2018-12-13T00:00:00Z  SAMD00069371  sub2-04-08_16s.sff   \n",
              "DRR079359_1  2018-12-13T00:00:00Z  SAMD00069373  sub3-01-16_16s.sff   \n",
              "DRR079363_1  2018-12-13T00:00:00Z  SAMD00069377  sub3-02-08_16s.sff   \n",
              "DRR079365_1  2018-12-13T00:00:00Z  SAMD00069379  sub3-02-16_16s.sff   \n",
              "DRR079369_1  2018-12-13T00:00:00Z  SAMD00069383  sub3-03-08_16s.sff   \n",
              "DRR079371_1  2018-12-13T00:00:00Z  SAMD00069385  sub3-03-16_16s.sff   \n",
              "DRR079375_1  2018-12-13T00:00:00Z  SAMD00069389  sub3-04-08_16s.sff   \n",
              "DRR079377_1  2018-12-13T00:00:00Z  SAMD00069391  sub4-01-16_16s.sff   \n",
              "DRR079381_1  2018-12-13T00:00:00Z  SAMD00069395  sub4-02-08_16s.sff   \n",
              "DRR079383_1  2018-12-13T00:00:00Z  SAMD00069397  sub4-02-16_16s.sff   \n",
              "DRR079387_1  2018-12-13T00:00:00Z  SAMD00069401  sub4-03-08_16s.sff   \n",
              "DRR079389_1  2018-12-13T00:00:00Z  SAMD00069403  sub4-03-16_16s.sff   \n",
              "DRR079393_1  2018-12-13T00:00:00Z  SAMD00069407  sub4-04-08_16s.sff   \n",
              "DRR079395_1  2018-12-13T00:00:00Z  SAMD00069409  sub5-01-16_16s.sff   \n",
              "DRR079399_1  2018-12-13T00:00:00Z  SAMD00069413  sub5-02-08_16s.sff   \n",
              "DRR079401_1  2018-12-13T00:00:00Z  SAMD00069415  sub5-02-16_16s.sff   \n",
              "DRR079405_1  2018-12-13T00:00:00Z  SAMD00069419  sub5-03-08_16s.sff   \n",
              "DRR079407_1  2018-12-13T00:00:00Z  SAMD00069421  sub5-03-16_16s.sff   \n",
              "DRR079411_1  2018-12-13T00:00:00Z  SAMD00069425  sub5-04-08_16s.sff   \n",
              "DRR079413_1  2018-12-13T00:00:00Z  SAMD00069427  sub6-01-16_16s.sff   \n",
              "DRR079417_1  2018-12-13T00:00:00Z  SAMD00069431  sub6-02-08_16s.sff   \n",
              "DRR079419_1  2018-12-13T00:00:00Z  SAMD00069433  sub6-02-16_16s.sff   \n",
              "DRR079423_1  2018-12-13T00:00:00Z  SAMD00069437  sub6-03-08_16s.sff   \n",
              "DRR079425_1  2018-12-13T00:00:00Z  SAMD00069439  sub6-03-16_16s.sff   \n",
              "DRR079429_1  2018-12-13T00:00:00Z  SAMD00069443  sub6-04-08_16s.sff   \n",
              "\n",
              "             SRA Study Group  \n",
              "sample-id                     \n",
              "DRR079323_1  DRP004653    PM  \n",
              "DRR079327_1  DRP004653    AM  \n",
              "DRR079329_1  DRP004653    PM  \n",
              "DRR079333_1  DRP004653    AM  \n",
              "DRR079335_1  DRP004653    PM  \n",
              "DRR079339_1  DRP004653    AM  \n",
              "DRR079341_1  DRP004653    PM  \n",
              "DRR079345_1  DRP004653    AM  \n",
              "DRR079347_1  DRP004653    PM  \n",
              "DRR079351_1  DRP004653    AM  \n",
              "DRR079353_1  DRP004653    PM  \n",
              "DRR079357_1  DRP004653    AM  \n",
              "DRR079359_1  DRP004653    PM  \n",
              "DRR079363_1  DRP004653    AM  \n",
              "DRR079365_1  DRP004653    PM  \n",
              "DRR079369_1  DRP004653    AM  \n",
              "DRR079371_1  DRP004653    PM  \n",
              "DRR079375_1  DRP004653    AM  \n",
              "DRR079377_1  DRP004653    PM  \n",
              "DRR079381_1  DRP004653    AM  \n",
              "DRR079383_1  DRP004653    PM  \n",
              "DRR079387_1  DRP004653    AM  \n",
              "DRR079389_1  DRP004653    PM  \n",
              "DRR079393_1  DRP004653    AM  \n",
              "DRR079395_1  DRP004653    PM  \n",
              "DRR079399_1  DRP004653    AM  \n",
              "DRR079401_1  DRP004653    PM  \n",
              "DRR079405_1  DRP004653    AM  \n",
              "DRR079407_1  DRP004653    PM  \n",
              "DRR079411_1  DRP004653    AM  \n",
              "DRR079413_1  DRP004653    PM  \n",
              "DRR079417_1  DRP004653    AM  \n",
              "DRR079419_1  DRP004653    PM  \n",
              "DRR079423_1  DRP004653    AM  \n",
              "DRR079425_1  DRP004653    PM  \n",
              "DRR079429_1  DRP004653    AM  \n",
              "\n",
              "[36 rows x 36 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-bfacc67b-5d90-4b5c-bdcd-a584ce05ea96\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Assay Type</th>\n",
              "      <th>AvgSpotLen</th>\n",
              "      <th>Bases</th>\n",
              "      <th>BioProject</th>\n",
              "      <th>BioSample</th>\n",
              "      <th>BioSampleModel</th>\n",
              "      <th>Bytes</th>\n",
              "      <th>Center xname</th>\n",
              "      <th>Collection_Date</th>\n",
              "      <th>Consent</th>\n",
              "      <th>...</th>\n",
              "      <th>LibrarySelection</th>\n",
              "      <th>LibrarySource</th>\n",
              "      <th>Organism</th>\n",
              "      <th>Platform</th>\n",
              "      <th>project xname</th>\n",
              "      <th>ReleaseDate</th>\n",
              "      <th>Sample xname</th>\n",
              "      <th>sample yname</th>\n",
              "      <th>SRA Study</th>\n",
              "      <th>Group</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>sample-id</th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>DRR079323_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1202750</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069337</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2904393</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069337</td>\n",
              "      <td>sub1-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079327_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204646</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069341</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2906270</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069341</td>\n",
              "      <td>sub1-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079329_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1202374</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069343</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2897112</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069343</td>\n",
              "      <td>sub1-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079333_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1201804</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069347</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2894179</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069347</td>\n",
              "      <td>sub1-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079335_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1201037</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069349</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2892993</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069349</td>\n",
              "      <td>sub1-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079339_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203454</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069353</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2898655</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069353</td>\n",
              "      <td>sub1-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079341_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206843</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069355</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2893466</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069355</td>\n",
              "      <td>sub2-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079345_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208063</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069359</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2903400</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069359</td>\n",
              "      <td>sub2-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079347_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206011</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069361</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2879839</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069361</td>\n",
              "      <td>sub2-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079351_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206119</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069365</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2905598</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069365</td>\n",
              "      <td>sub2-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079353_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205015</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069367</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2893969</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069367</td>\n",
              "      <td>sub2-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079357_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207998</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069371</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2896737</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069371</td>\n",
              "      <td>sub2-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079359_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208484</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069373</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2949507</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069373</td>\n",
              "      <td>sub3-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079363_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1209574</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069377</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2962968</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069377</td>\n",
              "      <td>sub3-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079365_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206531</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069379</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2947616</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069379</td>\n",
              "      <td>sub3-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079369_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208513</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069383</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2960925</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069383</td>\n",
              "      <td>sub3-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079371_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203354</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069385</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2943481</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069385</td>\n",
              "      <td>sub3-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079375_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204724</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069389</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2958594</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069389</td>\n",
              "      <td>sub3-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079377_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207319</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069391</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2930337</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069391</td>\n",
              "      <td>sub4-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079381_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205978</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069395</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2940426</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069395</td>\n",
              "      <td>sub4-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079383_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204780</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069397</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2920773</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069397</td>\n",
              "      <td>sub4-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079387_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205197</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069401</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2943440</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069401</td>\n",
              "      <td>sub4-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079389_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204420</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069403</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2919435</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069403</td>\n",
              "      <td>sub4-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079393_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203747</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069407</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2937432</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069407</td>\n",
              "      <td>sub4-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079395_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206659</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069409</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2928937</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069409</td>\n",
              "      <td>sub5-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079399_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206266</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069413</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2938816</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069413</td>\n",
              "      <td>sub5-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079401_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205145</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069415</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2925095</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069415</td>\n",
              "      <td>sub5-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079405_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208463</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069419</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2940193</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069419</td>\n",
              "      <td>sub5-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079407_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204482</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069421</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2921729</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069421</td>\n",
              "      <td>sub5-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079411_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205234</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069425</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2932702</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069425</td>\n",
              "      <td>sub5-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079413_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208075</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069427</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3026067</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069427</td>\n",
              "      <td>sub6-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079417_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1209548</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069431</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3035504</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069431</td>\n",
              "      <td>sub6-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079419_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1211194</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069433</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3038575</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069433</td>\n",
              "      <td>sub6-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079423_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208636</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069437</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3041059</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069437</td>\n",
              "      <td>sub6-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079425_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207743</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069439</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3043535</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069439</td>\n",
              "      <td>sub6-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079429_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206717</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069443</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3031928</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069443</td>\n",
              "      <td>sub6-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>36 rows × 36 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-bfacc67b-5d90-4b5c-bdcd-a584ce05ea96')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-bfacc67b-5d90-4b5c-bdcd-a584ce05ea96 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-bfacc67b-5d90-4b5c-bdcd-a584ce05ea96');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 56
        }
      ],
      "source": [
        "metadata"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 57,
      "metadata": {
        "id": "b6YnBgWm58fV"
      },
      "outputs": [],
      "source": [
        "shannon=pd.merge(shannon, metadata, left_index=True, right_index=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 58,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "dlfLxT3c58Rm",
        "outputId": "1491d040-43d5-49a8-e6e5-312f1689b5b0"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "             shannon_entropy Assay Type  AvgSpotLen    Bases BioProject  \\\n",
              "sample-id                                                                 \n",
              "DRR079323_1         3.269266   AMPLICON         400  1202750  PRJDB5388   \n",
              "DRR079327_1         2.794726   AMPLICON         401  1204646  PRJDB5388   \n",
              "DRR079329_1         2.881066   AMPLICON         400  1202374  PRJDB5388   \n",
              "DRR079333_1         3.049955   AMPLICON         400  1201804  PRJDB5388   \n",
              "DRR079335_1         2.951718   AMPLICON         400  1201037  PRJDB5388   \n",
              "DRR079339_1         3.069096   AMPLICON         401  1203454  PRJDB5388   \n",
              "DRR079341_1         3.327422   AMPLICON         402  1206843  PRJDB5388   \n",
              "DRR079345_1         3.227800   AMPLICON         402  1208063  PRJDB5388   \n",
              "DRR079347_1         3.241059   AMPLICON         402  1206011  PRJDB5388   \n",
              "DRR079351_1         3.071453   AMPLICON         402  1206119  PRJDB5388   \n",
              "DRR079353_1         3.325132   AMPLICON         401  1205015  PRJDB5388   \n",
              "DRR079357_1         2.878952   AMPLICON         402  1207998  PRJDB5388   \n",
              "DRR079359_1         3.009145   AMPLICON         402  1208484  PRJDB5388   \n",
              "DRR079363_1         3.373069   AMPLICON         403  1209574  PRJDB5388   \n",
              "DRR079365_1         3.156495   AMPLICON         402  1206531  PRJDB5388   \n",
              "DRR079369_1         3.249811   AMPLICON         402  1208513  PRJDB5388   \n",
              "DRR079371_1         2.959563   AMPLICON         401  1203354  PRJDB5388   \n",
              "DRR079375_1         3.499650   AMPLICON         401  1204724  PRJDB5388   \n",
              "DRR079377_1         3.289511   AMPLICON         402  1207319  PRJDB5388   \n",
              "DRR079381_1         3.290096   AMPLICON         401  1205978  PRJDB5388   \n",
              "DRR079383_1         2.916993   AMPLICON         401  1204780  PRJDB5388   \n",
              "DRR079387_1         3.445012   AMPLICON         401  1205197  PRJDB5388   \n",
              "DRR079389_1         3.240420   AMPLICON         401  1204420  PRJDB5388   \n",
              "DRR079393_1         3.371924   AMPLICON         401  1203747  PRJDB5388   \n",
              "DRR079395_1         3.674384   AMPLICON         402  1206659  PRJDB5388   \n",
              "DRR079399_1         3.288250   AMPLICON         402  1206266  PRJDB5388   \n",
              "DRR079401_1         3.944767   AMPLICON         401  1205145  PRJDB5388   \n",
              "DRR079405_1         3.811782   AMPLICON         402  1208463  PRJDB5388   \n",
              "DRR079407_1         3.331427   AMPLICON         401  1204482  PRJDB5388   \n",
              "DRR079411_1         3.445412   AMPLICON         401  1205234  PRJDB5388   \n",
              "DRR079413_1         2.909049   AMPLICON         402  1208075  PRJDB5388   \n",
              "DRR079417_1         2.731009   AMPLICON         403  1209548  PRJDB5388   \n",
              "DRR079419_1         2.978120   AMPLICON         403  1211194  PRJDB5388   \n",
              "DRR079423_1         3.016020   AMPLICON         402  1208636  PRJDB5388   \n",
              "DRR079425_1         3.123680   AMPLICON         402  1207743  PRJDB5388   \n",
              "DRR079429_1         2.743099   AMPLICON         402  1206717  PRJDB5388   \n",
              "\n",
              "                BioSample      BioSampleModel    Bytes Center xname  \\\n",
              "sample-id                                                             \n",
              "DRR079323_1  SAMD00069337  MIMS.me.human-oral  2904393       UT_COB   \n",
              "DRR079327_1  SAMD00069341  MIMS.me.human-oral  2906270       UT_COB   \n",
              "DRR079329_1  SAMD00069343  MIMS.me.human-oral  2897112       UT_COB   \n",
              "DRR079333_1  SAMD00069347  MIMS.me.human-oral  2894179       UT_COB   \n",
              "DRR079335_1  SAMD00069349  MIMS.me.human-oral  2892993       UT_COB   \n",
              "DRR079339_1  SAMD00069353  MIMS.me.human-oral  2898655       UT_COB   \n",
              "DRR079341_1  SAMD00069355  MIMS.me.human-oral  2893466       UT_COB   \n",
              "DRR079345_1  SAMD00069359  MIMS.me.human-oral  2903400       UT_COB   \n",
              "DRR079347_1  SAMD00069361  MIMS.me.human-oral  2879839       UT_COB   \n",
              "DRR079351_1  SAMD00069365  MIMS.me.human-oral  2905598       UT_COB   \n",
              "DRR079353_1  SAMD00069367  MIMS.me.human-oral  2893969       UT_COB   \n",
              "DRR079357_1  SAMD00069371  MIMS.me.human-oral  2896737       UT_COB   \n",
              "DRR079359_1  SAMD00069373  MIMS.me.human-oral  2949507       UT_COB   \n",
              "DRR079363_1  SAMD00069377  MIMS.me.human-oral  2962968       UT_COB   \n",
              "DRR079365_1  SAMD00069379  MIMS.me.human-oral  2947616       UT_COB   \n",
              "DRR079369_1  SAMD00069383  MIMS.me.human-oral  2960925       UT_COB   \n",
              "DRR079371_1  SAMD00069385  MIMS.me.human-oral  2943481       UT_COB   \n",
              "DRR079375_1  SAMD00069389  MIMS.me.human-oral  2958594       UT_COB   \n",
              "DRR079377_1  SAMD00069391  MIMS.me.human-oral  2930337       UT_COB   \n",
              "DRR079381_1  SAMD00069395  MIMS.me.human-oral  2940426       UT_COB   \n",
              "DRR079383_1  SAMD00069397  MIMS.me.human-oral  2920773       UT_COB   \n",
              "DRR079387_1  SAMD00069401  MIMS.me.human-oral  2943440       UT_COB   \n",
              "DRR079389_1  SAMD00069403  MIMS.me.human-oral  2919435       UT_COB   \n",
              "DRR079393_1  SAMD00069407  MIMS.me.human-oral  2937432       UT_COB   \n",
              "DRR079395_1  SAMD00069409  MIMS.me.human-oral  2928937       UT_COB   \n",
              "DRR079399_1  SAMD00069413  MIMS.me.human-oral  2938816       UT_COB   \n",
              "DRR079401_1  SAMD00069415  MIMS.me.human-oral  2925095       UT_COB   \n",
              "DRR079405_1  SAMD00069419  MIMS.me.human-oral  2940193       UT_COB   \n",
              "DRR079407_1  SAMD00069421  MIMS.me.human-oral  2921729       UT_COB   \n",
              "DRR079411_1  SAMD00069425  MIMS.me.human-oral  2932702       UT_COB   \n",
              "DRR079413_1  SAMD00069427  MIMS.me.human-oral  3026067       UT_COB   \n",
              "DRR079417_1  SAMD00069431  MIMS.me.human-oral  3035504       UT_COB   \n",
              "DRR079419_1  SAMD00069433  MIMS.me.human-oral  3038575       UT_COB   \n",
              "DRR079423_1  SAMD00069437  MIMS.me.human-oral  3041059       UT_COB   \n",
              "DRR079425_1  SAMD00069439  MIMS.me.human-oral  3043535       UT_COB   \n",
              "DRR079429_1  SAMD00069443  MIMS.me.human-oral  3031928       UT_COB   \n",
              "\n",
              "            Collection_Date  ... LibrarySelection LibrarySource  \\\n",
              "sample-id                    ...                                  \n",
              "DRR079323_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079327_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079329_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079333_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079335_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079339_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079341_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079345_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079347_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079351_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079353_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079357_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079359_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079363_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079365_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079369_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079371_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079375_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079377_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079381_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079383_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079387_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079389_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079393_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079395_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079399_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079401_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079405_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079407_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079411_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079413_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079417_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079419_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079423_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079425_1         missing  ...              PCR   METAGENOMIC   \n",
              "DRR079429_1         missing  ...              PCR   METAGENOMIC   \n",
              "\n",
              "                          Organism Platform  \\\n",
              "sample-id                                     \n",
              "DRR079323_1  human oral metagenome    LS454   \n",
              "DRR079327_1  human oral metagenome    LS454   \n",
              "DRR079329_1  human oral metagenome    LS454   \n",
              "DRR079333_1  human oral metagenome    LS454   \n",
              "DRR079335_1  human oral metagenome    LS454   \n",
              "DRR079339_1  human oral metagenome    LS454   \n",
              "DRR079341_1  human oral metagenome    LS454   \n",
              "DRR079345_1  human oral metagenome    LS454   \n",
              "DRR079347_1  human oral metagenome    LS454   \n",
              "DRR079351_1  human oral metagenome    LS454   \n",
              "DRR079353_1  human oral metagenome    LS454   \n",
              "DRR079357_1  human oral metagenome    LS454   \n",
              "DRR079359_1  human oral metagenome    LS454   \n",
              "DRR079363_1  human oral metagenome    LS454   \n",
              "DRR079365_1  human oral metagenome    LS454   \n",
              "DRR079369_1  human oral metagenome    LS454   \n",
              "DRR079371_1  human oral metagenome    LS454   \n",
              "DRR079375_1  human oral metagenome    LS454   \n",
              "DRR079377_1  human oral metagenome    LS454   \n",
              "DRR079381_1  human oral metagenome    LS454   \n",
              "DRR079383_1  human oral metagenome    LS454   \n",
              "DRR079387_1  human oral metagenome    LS454   \n",
              "DRR079389_1  human oral metagenome    LS454   \n",
              "DRR079393_1  human oral metagenome    LS454   \n",
              "DRR079395_1  human oral metagenome    LS454   \n",
              "DRR079399_1  human oral metagenome    LS454   \n",
              "DRR079401_1  human oral metagenome    LS454   \n",
              "DRR079405_1  human oral metagenome    LS454   \n",
              "DRR079407_1  human oral metagenome    LS454   \n",
              "DRR079411_1  human oral metagenome    LS454   \n",
              "DRR079413_1  human oral metagenome    LS454   \n",
              "DRR079417_1  human oral metagenome    LS454   \n",
              "DRR079419_1  human oral metagenome    LS454   \n",
              "DRR079423_1  human oral metagenome    LS454   \n",
              "DRR079425_1  human oral metagenome    LS454   \n",
              "DRR079429_1  human oral metagenome    LS454   \n",
              "\n",
              "                                                 project xname  \\\n",
              "sample-id                                                        \n",
              "DRR079323_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079327_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079329_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079333_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079335_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079339_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079341_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079345_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079347_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079351_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079353_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079357_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079359_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079363_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079365_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079369_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079371_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079375_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079377_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079381_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079383_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079387_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079389_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079393_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079395_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079399_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079401_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079405_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079407_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079411_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079413_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079417_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079419_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079423_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079425_1  Metgenomic analysis of human saliva microbiome...   \n",
              "DRR079429_1  Metgenomic analysis of human saliva microbiome...   \n",
              "\n",
              "                      ReleaseDate  Sample xname        sample yname  \\\n",
              "sample-id                                                             \n",
              "DRR079323_1  2018-12-13T00:00:00Z  SAMD00069337  sub1-01-16_16s.sff   \n",
              "DRR079327_1  2018-12-13T00:00:00Z  SAMD00069341  sub1-02-08_16s.sff   \n",
              "DRR079329_1  2018-12-13T00:00:00Z  SAMD00069343  sub1-02-16_16s.sff   \n",
              "DRR079333_1  2018-12-13T00:00:00Z  SAMD00069347  sub1-03-08_16s.sff   \n",
              "DRR079335_1  2018-12-13T00:00:00Z  SAMD00069349  sub1-03-16_16s.sff   \n",
              "DRR079339_1  2018-12-13T00:00:00Z  SAMD00069353  sub1-04-08_16s.sff   \n",
              "DRR079341_1  2018-12-13T00:00:00Z  SAMD00069355  sub2-01-16_16s.sff   \n",
              "DRR079345_1  2018-12-13T00:00:00Z  SAMD00069359  sub2-02-08_16s.sff   \n",
              "DRR079347_1  2018-12-13T00:00:00Z  SAMD00069361  sub2-02-16_16s.sff   \n",
              "DRR079351_1  2018-12-13T00:00:00Z  SAMD00069365  sub2-03-08_16s.sff   \n",
              "DRR079353_1  2018-12-13T00:00:00Z  SAMD00069367  sub2-03-16_16s.sff   \n",
              "DRR079357_1  2018-12-13T00:00:00Z  SAMD00069371  sub2-04-08_16s.sff   \n",
              "DRR079359_1  2018-12-13T00:00:00Z  SAMD00069373  sub3-01-16_16s.sff   \n",
              "DRR079363_1  2018-12-13T00:00:00Z  SAMD00069377  sub3-02-08_16s.sff   \n",
              "DRR079365_1  2018-12-13T00:00:00Z  SAMD00069379  sub3-02-16_16s.sff   \n",
              "DRR079369_1  2018-12-13T00:00:00Z  SAMD00069383  sub3-03-08_16s.sff   \n",
              "DRR079371_1  2018-12-13T00:00:00Z  SAMD00069385  sub3-03-16_16s.sff   \n",
              "DRR079375_1  2018-12-13T00:00:00Z  SAMD00069389  sub3-04-08_16s.sff   \n",
              "DRR079377_1  2018-12-13T00:00:00Z  SAMD00069391  sub4-01-16_16s.sff   \n",
              "DRR079381_1  2018-12-13T00:00:00Z  SAMD00069395  sub4-02-08_16s.sff   \n",
              "DRR079383_1  2018-12-13T00:00:00Z  SAMD00069397  sub4-02-16_16s.sff   \n",
              "DRR079387_1  2018-12-13T00:00:00Z  SAMD00069401  sub4-03-08_16s.sff   \n",
              "DRR079389_1  2018-12-13T00:00:00Z  SAMD00069403  sub4-03-16_16s.sff   \n",
              "DRR079393_1  2018-12-13T00:00:00Z  SAMD00069407  sub4-04-08_16s.sff   \n",
              "DRR079395_1  2018-12-13T00:00:00Z  SAMD00069409  sub5-01-16_16s.sff   \n",
              "DRR079399_1  2018-12-13T00:00:00Z  SAMD00069413  sub5-02-08_16s.sff   \n",
              "DRR079401_1  2018-12-13T00:00:00Z  SAMD00069415  sub5-02-16_16s.sff   \n",
              "DRR079405_1  2018-12-13T00:00:00Z  SAMD00069419  sub5-03-08_16s.sff   \n",
              "DRR079407_1  2018-12-13T00:00:00Z  SAMD00069421  sub5-03-16_16s.sff   \n",
              "DRR079411_1  2018-12-13T00:00:00Z  SAMD00069425  sub5-04-08_16s.sff   \n",
              "DRR079413_1  2018-12-13T00:00:00Z  SAMD00069427  sub6-01-16_16s.sff   \n",
              "DRR079417_1  2018-12-13T00:00:00Z  SAMD00069431  sub6-02-08_16s.sff   \n",
              "DRR079419_1  2018-12-13T00:00:00Z  SAMD00069433  sub6-02-16_16s.sff   \n",
              "DRR079423_1  2018-12-13T00:00:00Z  SAMD00069437  sub6-03-08_16s.sff   \n",
              "DRR079425_1  2018-12-13T00:00:00Z  SAMD00069439  sub6-03-16_16s.sff   \n",
              "DRR079429_1  2018-12-13T00:00:00Z  SAMD00069443  sub6-04-08_16s.sff   \n",
              "\n",
              "             SRA Study Group  \n",
              "sample-id                     \n",
              "DRR079323_1  DRP004653    PM  \n",
              "DRR079327_1  DRP004653    AM  \n",
              "DRR079329_1  DRP004653    PM  \n",
              "DRR079333_1  DRP004653    AM  \n",
              "DRR079335_1  DRP004653    PM  \n",
              "DRR079339_1  DRP004653    AM  \n",
              "DRR079341_1  DRP004653    PM  \n",
              "DRR079345_1  DRP004653    AM  \n",
              "DRR079347_1  DRP004653    PM  \n",
              "DRR079351_1  DRP004653    AM  \n",
              "DRR079353_1  DRP004653    PM  \n",
              "DRR079357_1  DRP004653    AM  \n",
              "DRR079359_1  DRP004653    PM  \n",
              "DRR079363_1  DRP004653    AM  \n",
              "DRR079365_1  DRP004653    PM  \n",
              "DRR079369_1  DRP004653    AM  \n",
              "DRR079371_1  DRP004653    PM  \n",
              "DRR079375_1  DRP004653    AM  \n",
              "DRR079377_1  DRP004653    PM  \n",
              "DRR079381_1  DRP004653    AM  \n",
              "DRR079383_1  DRP004653    PM  \n",
              "DRR079387_1  DRP004653    AM  \n",
              "DRR079389_1  DRP004653    PM  \n",
              "DRR079393_1  DRP004653    AM  \n",
              "DRR079395_1  DRP004653    PM  \n",
              "DRR079399_1  DRP004653    AM  \n",
              "DRR079401_1  DRP004653    PM  \n",
              "DRR079405_1  DRP004653    AM  \n",
              "DRR079407_1  DRP004653    PM  \n",
              "DRR079411_1  DRP004653    AM  \n",
              "DRR079413_1  DRP004653    PM  \n",
              "DRR079417_1  DRP004653    AM  \n",
              "DRR079419_1  DRP004653    PM  \n",
              "DRR079423_1  DRP004653    AM  \n",
              "DRR079425_1  DRP004653    PM  \n",
              "DRR079429_1  DRP004653    AM  \n",
              "\n",
              "[36 rows x 37 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-782bd24d-4c82-4912-b842-a18c24bac93a\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>shannon_entropy</th>\n",
              "      <th>Assay Type</th>\n",
              "      <th>AvgSpotLen</th>\n",
              "      <th>Bases</th>\n",
              "      <th>BioProject</th>\n",
              "      <th>BioSample</th>\n",
              "      <th>BioSampleModel</th>\n",
              "      <th>Bytes</th>\n",
              "      <th>Center xname</th>\n",
              "      <th>Collection_Date</th>\n",
              "      <th>...</th>\n",
              "      <th>LibrarySelection</th>\n",
              "      <th>LibrarySource</th>\n",
              "      <th>Organism</th>\n",
              "      <th>Platform</th>\n",
              "      <th>project xname</th>\n",
              "      <th>ReleaseDate</th>\n",
              "      <th>Sample xname</th>\n",
              "      <th>sample yname</th>\n",
              "      <th>SRA Study</th>\n",
              "      <th>Group</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>sample-id</th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>DRR079323_1</th>\n",
              "      <td>3.269266</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1202750</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069337</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2904393</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069337</td>\n",
              "      <td>sub1-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079327_1</th>\n",
              "      <td>2.794726</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204646</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069341</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2906270</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069341</td>\n",
              "      <td>sub1-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079329_1</th>\n",
              "      <td>2.881066</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1202374</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069343</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2897112</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069343</td>\n",
              "      <td>sub1-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079333_1</th>\n",
              "      <td>3.049955</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1201804</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069347</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2894179</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069347</td>\n",
              "      <td>sub1-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079335_1</th>\n",
              "      <td>2.951718</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1201037</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069349</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2892993</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069349</td>\n",
              "      <td>sub1-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079339_1</th>\n",
              "      <td>3.069096</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203454</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069353</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2898655</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069353</td>\n",
              "      <td>sub1-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079341_1</th>\n",
              "      <td>3.327422</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206843</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069355</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2893466</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069355</td>\n",
              "      <td>sub2-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079345_1</th>\n",
              "      <td>3.227800</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208063</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069359</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2903400</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069359</td>\n",
              "      <td>sub2-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079347_1</th>\n",
              "      <td>3.241059</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206011</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069361</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2879839</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069361</td>\n",
              "      <td>sub2-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079351_1</th>\n",
              "      <td>3.071453</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206119</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069365</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2905598</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069365</td>\n",
              "      <td>sub2-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079353_1</th>\n",
              "      <td>3.325132</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205015</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069367</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2893969</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069367</td>\n",
              "      <td>sub2-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079357_1</th>\n",
              "      <td>2.878952</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207998</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069371</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2896737</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069371</td>\n",
              "      <td>sub2-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079359_1</th>\n",
              "      <td>3.009145</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208484</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069373</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2949507</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069373</td>\n",
              "      <td>sub3-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079363_1</th>\n",
              "      <td>3.373069</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1209574</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069377</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2962968</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069377</td>\n",
              "      <td>sub3-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079365_1</th>\n",
              "      <td>3.156495</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206531</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069379</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2947616</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069379</td>\n",
              "      <td>sub3-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079369_1</th>\n",
              "      <td>3.249811</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208513</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069383</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2960925</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069383</td>\n",
              "      <td>sub3-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079371_1</th>\n",
              "      <td>2.959563</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203354</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069385</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2943481</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069385</td>\n",
              "      <td>sub3-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079375_1</th>\n",
              "      <td>3.499650</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204724</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069389</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2958594</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069389</td>\n",
              "      <td>sub3-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079377_1</th>\n",
              "      <td>3.289511</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207319</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069391</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2930337</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069391</td>\n",
              "      <td>sub4-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079381_1</th>\n",
              "      <td>3.290096</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205978</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069395</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2940426</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069395</td>\n",
              "      <td>sub4-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079383_1</th>\n",
              "      <td>2.916993</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204780</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069397</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2920773</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069397</td>\n",
              "      <td>sub4-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079387_1</th>\n",
              "      <td>3.445012</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205197</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069401</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2943440</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069401</td>\n",
              "      <td>sub4-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079389_1</th>\n",
              "      <td>3.240420</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204420</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069403</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2919435</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069403</td>\n",
              "      <td>sub4-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079393_1</th>\n",
              "      <td>3.371924</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203747</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069407</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2937432</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069407</td>\n",
              "      <td>sub4-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079395_1</th>\n",
              "      <td>3.674384</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206659</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069409</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2928937</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069409</td>\n",
              "      <td>sub5-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079399_1</th>\n",
              "      <td>3.288250</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206266</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069413</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2938816</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069413</td>\n",
              "      <td>sub5-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079401_1</th>\n",
              "      <td>3.944767</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205145</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069415</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2925095</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069415</td>\n",
              "      <td>sub5-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079405_1</th>\n",
              "      <td>3.811782</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208463</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069419</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2940193</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069419</td>\n",
              "      <td>sub5-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079407_1</th>\n",
              "      <td>3.331427</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204482</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069421</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2921729</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069421</td>\n",
              "      <td>sub5-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079411_1</th>\n",
              "      <td>3.445412</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205234</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069425</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2932702</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069425</td>\n",
              "      <td>sub5-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079413_1</th>\n",
              "      <td>2.909049</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208075</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069427</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3026067</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069427</td>\n",
              "      <td>sub6-01-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079417_1</th>\n",
              "      <td>2.731009</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1209548</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069431</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3035504</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069431</td>\n",
              "      <td>sub6-02-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079419_1</th>\n",
              "      <td>2.978120</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1211194</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069433</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3038575</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069433</td>\n",
              "      <td>sub6-02-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079423_1</th>\n",
              "      <td>3.016020</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208636</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069437</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3041059</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069437</td>\n",
              "      <td>sub6-03-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079425_1</th>\n",
              "      <td>3.123680</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207743</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069439</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3043535</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069439</td>\n",
              "      <td>sub6-03-16_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>PM</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079429_1</th>\n",
              "      <td>2.743099</td>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206717</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069443</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3031928</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>...</td>\n",
              "      <td>PCR</td>\n",
              "      <td>METAGENOMIC</td>\n",
              "      <td>human oral metagenome</td>\n",
              "      <td>LS454</td>\n",
              "      <td>Metgenomic analysis of human saliva microbiome...</td>\n",
              "      <td>2018-12-13T00:00:00Z</td>\n",
              "      <td>SAMD00069443</td>\n",
              "      <td>sub6-04-08_16s.sff</td>\n",
              "      <td>DRP004653</td>\n",
              "      <td>AM</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>36 rows × 37 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-782bd24d-4c82-4912-b842-a18c24bac93a')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-782bd24d-4c82-4912-b842-a18c24bac93a button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-782bd24d-4c82-4912-b842-a18c24bac93a');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 58
        }
      ],
      "source": [
        "shannon"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 59,
      "metadata": {
        "id": "0L4E505W6FMw"
      },
      "outputs": [],
      "source": [
        "import plotly.express as px"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 61,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 542
        },
        "id": "q6qCUl8p6Mli",
        "outputId": "d1172490-6308-4e90-97df-a731de4a62b4"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script src=\"https://cdn.plot.ly/plotly-2.8.3.min.js\"></script>                <div id=\"4a01cfd9-ddbc-4bb6-9713-5357f09afe10\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"4a01cfd9-ddbc-4bb6-9713-5357f09afe10\")) {                    Plotly.newPlot(                        \"4a01cfd9-ddbc-4bb6-9713-5357f09afe10\",                        [{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Group=%{x}<br>shannon_entropy=%{y}<extra></extra>\",\"legendgroup\":\"\",\"marker\":{\"color\":\"#636efa\"},\"name\":\"\",\"notched\":false,\"offsetgroup\":\"\",\"orientation\":\"v\",\"showlegend\":false,\"x\":[\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\",\"PM\",\"AM\"],\"x0\":\" \",\"xaxis\":\"x\",\"y\":[3.269265586984025,2.794725717426189,2.881065859800299,3.0499545948913256,2.951718032563308,3.069096356092575,3.3274217506745027,3.227800002780591,3.241058981674689,3.0714530323150044,3.3251322730667785,2.878952063237984,3.009144841446952,3.373068530736889,3.156494758961114,3.2498112404959247,2.95956322826193,3.4996502591361045,3.2895105346457973,3.2900956119934115,2.916992778099673,3.445011698729496,3.240420009434675,3.371923847976559,3.674383948461577,3.288250278562436,3.944766553095388,3.811782457118309,3.331426862279861,3.445411856935212,2.909048526828443,2.731008574155161,2.9781196839890582,3.0160199272975965,3.1236798333293234,2.743098929200242],\"y0\":\" \",\"yaxis\":\"y\",\"type\":\"box\"}],                        {\"template\":{\"data\":{\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"choropleth\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"choropleth\"}],\"contour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"contour\"}],\"contourcarpet\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"contourcarpet\"}],\"heatmap\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmap\"}],\"heatmapgl\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmapgl\"}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"histogram2d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2d\"}],\"histogram2dcontour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2dcontour\"}],\"mesh3d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"mesh3d\"}],\"parcoords\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"parcoords\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}],\"scatter\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter\"}],\"scatter3d\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter3d\"}],\"scattercarpet\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattercarpet\"}],\"scattergeo\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergeo\"}],\"scattergl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergl\"}],\"scattermapbox\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattermapbox\"}],\"scatterpolar\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolar\"}],\"scatterpolargl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolargl\"}],\"scatterternary\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterternary\"}],\"surface\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"surface\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}]},\"layout\":{\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"autotypenumbers\":\"strict\",\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]],\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]},\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"geo\":{\"bgcolor\":\"white\",\"lakecolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"showlakes\":true,\"showland\":true,\"subunitcolor\":\"white\"},\"hoverlabel\":{\"align\":\"left\"},\"hovermode\":\"closest\",\"mapbox\":{\"style\":\"light\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"ternary\":{\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"title\":{\"x\":0.05},\"xaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2},\"yaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"Group\"},\"showline\":true,\"linewidth\":2,\"linecolor\":\"black\"},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"shannon_entropy\"},\"showline\":true,\"linewidth\":2,\"linecolor\":\"black\"},\"legend\":{\"tracegroupgap\":0},\"margin\":{\"t\":60},\"boxmode\":\"group\",\"plot_bgcolor\":\"white\"},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('4a01cfd9-ddbc-4bb6-9713-5357f09afe10');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "fig = px.box(shannon, x=\"Group\", y=\"shannon_entropy\")\n",
        "fig.update_layout(plot_bgcolor = \"white\")\n",
        "fig.update_xaxes(showline=True, linewidth=2, linecolor='black')\n",
        "fig.update_yaxes(showline=True, linewidth=2, linecolor='black')"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MAgxEzn2ikfJ"
      },
      "source": [
        "beta diversity example"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 63,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "kWmpALVy9N9P",
        "outputId": "a7f8528e-272c-4b09-c12e-3d11eb00476f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  /content/sample_data/core-metrics-results/bray_curtis_pcoa_results.qza\n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/checksums.md5  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/332cfd22-bb27-478b-9fb7-59cfddc5e1ff/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/332cfd22-bb27-478b-9fb7-59cfddc5e1ff/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/332cfd22-bb27-478b-9fb7-59cfddc5e1ff/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/332cfd22-bb27-478b-9fb7-59cfddc5e1ff/action/metadata.tsv  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/332cfd22-bb27-478b-9fb7-59cfddc5e1ff/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/61816e33-5cec-4092-9721-af64f38c907c/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/61816e33-5cec-4092-9721-af64f38c907c/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/61816e33-5cec-4092-9721-af64f38c907c/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/61816e33-5cec-4092-9721-af64f38c907c/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/10a8eb31-7da8-468f-ba16-c53a0663ffe6/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/10a8eb31-7da8-468f-ba16-c53a0663ffe6/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/10a8eb31-7da8-468f-ba16-c53a0663ffe6/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/10a8eb31-7da8-468f-ba16-c53a0663ffe6/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/citations.bib  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/VERSION  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/metadata.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/action/metadata.tsv  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/provenance/action/action.yaml  \n",
            "  inflating: b6dc6908-d0c8-4857-8f6b-70055706934b/data/ordination.txt  \n"
          ]
        }
      ],
      "source": [
        "!unzip /content/sample_data/core-metrics-results/bray_curtis_pcoa_results.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 64,
      "metadata": {
        "id": "OJAzwjAv9Wtf"
      },
      "outputs": [],
      "source": [
        "!cp /content/sample_data/b6dc6908-d0c8-4857-8f6b-70055706934b/data/ordination.txt /content/sample_data/"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 65,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9Y1rXGnZ9eBb",
        "outputId": "18140d1f-ba33-40cb-c933-d4f2612d59e2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/pandas/util/_decorators.py:311: ParserWarning:\n",
            "\n",
            "Falling back to the 'python' engine because the 'c' engine does not support skipfooter; you can avoid this warning by specifying engine='python'.\n",
            "\n"
          ]
        }
      ],
      "source": [
        "pcoa=pd.read_table(\"ordination.txt\", skiprows=9,skipfooter=4, header=None, index_col=0)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 66,
      "metadata": {
        "id": "cOYsIJq1-nkX"
      },
      "outputs": [],
      "source": [
        "pcoa.index.names = ['sample-id']"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 67,
      "metadata": {
        "id": "cILDp9jtA8tw"
      },
      "outputs": [],
      "source": [
        "pcoa=pd.merge(pcoa, metadata, left_index=True, right_index=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 70,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 542
        },
        "id": "VfUQfKa0BXq1",
        "outputId": "7e881d8f-6ef1-4d12-d2e6-66874df43837"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script src=\"https://cdn.plot.ly/plotly-2.8.3.min.js\"></script>                <div id=\"a0854360-0f23-4c66-9440-59935e01b3df\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"a0854360-0f23-4c66-9440-59935e01b3df\")) {                    Plotly.newPlot(                        \"a0854360-0f23-4c66-9440-59935e01b3df\",                        [{\"hovertemplate\":\"Group=PM<br>PC1=%{x}<br>PC2=%{y}<extra></extra>\",\"legendgroup\":\"PM\",\"marker\":{\"color\":\"#636efa\",\"symbol\":\"circle\"},\"mode\":\"markers\",\"name\":\"PM\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[-0.0289084476033472,-0.0289084476033536,-0.0289084476033474,0.4914436092569088,-0.0289084476033472,-0.0289084476033474,-0.0289084476033472,-0.0289084476033474,-0.0289084476033473,-0.0289084476033475,-0.0289084476033474,-0.0289084476033476,-0.0289084476033475,-0.0289084476033473,-0.0289084476033472,-0.0289084476033473,-0.0289084476033472,-0.0289084476033474],\"xaxis\":\"x\",\"y\":[0.0,0.0630650298144413,0.0575998032416573,1.5229900567841357e-17,-0.0191342465781668,0.0527375579529945,-0.0655405227352289,0.0243255918691434,-0.0110129934874568,-0.1867243651175898,0.0464919371013933,-0.0009270781371171,0.2007715792874383,-0.2330461133045359,-0.0034742138117708,-0.2795234615966704,-0.2158643717961404,0.0169839580725455],\"yaxis\":\"y\",\"type\":\"scatter\"},{\"hovertemplate\":\"Group=AM<br>PC1=%{x}<br>PC2=%{y}<extra></extra>\",\"legendgroup\":\"AM\",\"marker\":{\"color\":\"#EF553B\",\"symbol\":\"circle\"},\"mode\":\"markers\",\"name\":\"AM\",\"orientation\":\"v\",\"showlegend\":true,\"x\":[-0.0289084476033449,-0.0289084476033474,-0.0289084476033474,-0.0289084476033472,-0.0289084476033474,-0.0289084476033472,-0.0289084476033472,-0.0289084476033474,-0.0289084476033474,-0.0289084476033475,0.4914436092569096,-0.0289084476033477,-0.0289084476033475,-0.0289084476033472,-0.0289084476033474,-0.0289084476033473,-0.0289084476033471,-0.0289084476033472],\"xaxis\":\"x\",\"y\":[-0.0029673467877403,0.0678092066279121,0.0105246381169719,0.0303435971772791,0.0053507731863831,0.0521781698346102,-0.0328128628023486,-0.0184209119572357,-0.0271354897102913,-0.0613047600584647,7.692269382218912e-16,-0.01880978477079,0.1459426132858416,0.4263182720831664,-0.0068506175934995,-0.0438516409386831,0.0760726831417036,-0.0491146296097519],\"yaxis\":\"y\",\"type\":\"scatter\"}],                        {\"template\":{\"data\":{\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"choropleth\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"choropleth\"}],\"contour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"contour\"}],\"contourcarpet\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"contourcarpet\"}],\"heatmap\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmap\"}],\"heatmapgl\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmapgl\"}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"histogram2d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2d\"}],\"histogram2dcontour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2dcontour\"}],\"mesh3d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"mesh3d\"}],\"parcoords\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"parcoords\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}],\"scatter\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter\"}],\"scatter3d\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter3d\"}],\"scattercarpet\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattercarpet\"}],\"scattergeo\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergeo\"}],\"scattergl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergl\"}],\"scattermapbox\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattermapbox\"}],\"scatterpolar\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolar\"}],\"scatterpolargl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolargl\"}],\"scatterternary\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterternary\"}],\"surface\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"surface\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}]},\"layout\":{\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"autotypenumbers\":\"strict\",\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]],\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]},\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"geo\":{\"bgcolor\":\"white\",\"lakecolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"showlakes\":true,\"showland\":true,\"subunitcolor\":\"white\"},\"hoverlabel\":{\"align\":\"left\"},\"hovermode\":\"closest\",\"mapbox\":{\"style\":\"light\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"ternary\":{\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"title\":{\"x\":0.05},\"xaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2},\"yaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"PC1\"},\"showline\":true,\"linewidth\":2,\"linecolor\":\"black\"},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"PC2\"},\"showline\":true,\"linewidth\":2,\"linecolor\":\"black\"},\"legend\":{\"title\":{\"text\":\"Group\"},\"tracegroupgap\":0},\"margin\":{\"t\":60},\"plot_bgcolor\":\"white\"},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('a0854360-0f23-4c66-9440-59935e01b3df');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "fig = px.scatter(pcoa, x=1,y=2, color=\"Group\",  labels={\"1\": \"PC1\", \"2\":\"PC2\"})\n",
        "fig.update_layout(plot_bgcolor = \"white\")\n",
        "fig.update_xaxes(showline=True, linewidth=2, linecolor='black')\n",
        "fig.update_yaxes(showline=True, linewidth=2, linecolor='black')\n",
        "fig.show()\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Fw2QTBNMiz73"
      },
      "source": [
        "taxa barplot, much better to use taxa_bar.qzv"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 71,
      "metadata": {
        "id": "fB7MwH33XDc5",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3d312c23-9614-43e9-8bef-7ca2c042cfff"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: biom-format in /usr/local/lib/python3.8/site-packages (2.1.10)\n",
            "Requirement already satisfied: future>=0.16.0 in /usr/local/lib/python3.8/site-packages (from biom-format) (0.18.2)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.8/site-packages (from biom-format) (7.1.2)\n",
            "Requirement already satisfied: h5py in /usr/local/lib/python3.8/site-packages (from biom-format) (3.6.0)\n",
            "Requirement already satisfied: pandas>=0.20.0 in /usr/local/lib/python3.8/site-packages (from biom-format) (1.2.5)\n",
            "Requirement already satisfied: six>=1.10.0 in /usr/local/lib/python3.8/site-packages (from biom-format) (1.16.0)\n",
            "Requirement already satisfied: scipy>=1.3.1 in /usr/local/lib/python3.8/site-packages (from biom-format) (1.8.0)\n",
            "Requirement already satisfied: numpy>=1.9.2 in /usr/local/lib/python3.8/site-packages (from biom-format) (1.22.3)\n",
            "Requirement already satisfied: cython>=0.29 in /usr/local/lib/python3.8/site-packages (from biom-format) (0.29.28)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.8/site-packages (from pandas>=0.20.0->biom-format) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.8/site-packages (from pandas>=0.20.0->biom-format) (2022.1)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!pip install biom-format"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 73,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "zoj0QPr5SNrO",
        "outputId": "f903b2d4-fd43-4b7d-dac8-0e98d6f8a973"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  /content/sample_data/core-metrics-results/rarefied_table.qza\n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/checksums.md5  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/c5ed7d99-1620-4a06-bfd6-732ae6471424/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/37c7a189-9356-427e-a0de-24c9574b8d04/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/81b0850e-c9f6-4f31-865e-de6856753ab6/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/d39863ec-df0d-44c8-b515-c1c65cf7680e/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/842310a6-b0bf-44b5-ac81-0922e4af23ef/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/842310a6-b0bf-44b5-ac81-0922e4af23ef/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/842310a6-b0bf-44b5-ac81-0922e4af23ef/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/842310a6-b0bf-44b5-ac81-0922e4af23ef/action/metadata.tsv  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/842310a6-b0bf-44b5-ac81-0922e4af23ef/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/0e1aa7f0-eed9-4d37-aa75-fc1796fa3ec9/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/fe97929a-b346-48a4-b0b0-c11817464f62/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/citations.bib  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/VERSION  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/metadata.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/artifacts/b8f51e8e-0841-48c6-be8f-38bef9f9cf38/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/action/metadata.tsv  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/provenance/action/action.yaml  \n",
            "  inflating: 3cc13aa0-347a-477c-bc37-e1695817be27/data/feature-table.biom  \n"
          ]
        }
      ],
      "source": [
        "!unzip /content/sample_data/core-metrics-results/rarefied_table.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 74,
      "metadata": {
        "id": "Fdqu3VxjZ5jh"
      },
      "outputs": [],
      "source": [
        "!biom convert -i /content/sample_data/3cc13aa0-347a-477c-bc37-e1695817be27//data/feature-table.biom -o table.from_biom.txt --to-tsv "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 75,
      "metadata": {
        "id": "IGyyKe98TI3e"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "table=pd.read_table(\"table.from_biom.txt\", skiprows=[0],index_col=[0])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 76,
      "metadata": {
        "id": "EW-v2pl8wdrS"
      },
      "outputs": [],
      "source": [
        "rel_table = table/table[table.columns].sum()*100"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 77,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VApojVSbwrMs",
        "outputId": "ee1c8420-bc10-4bd9-b4f4-6dc72d958b80"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "DRR079323_1    100.0\n",
              "DRR079327_1    100.0\n",
              "DRR079329_1    100.0\n",
              "DRR079333_1    100.0\n",
              "DRR079335_1    100.0\n",
              "DRR079339_1    100.0\n",
              "DRR079341_1    100.0\n",
              "DRR079345_1    100.0\n",
              "DRR079347_1    100.0\n",
              "DRR079351_1    100.0\n",
              "DRR079353_1    100.0\n",
              "DRR079357_1    100.0\n",
              "DRR079359_1    100.0\n",
              "DRR079363_1    100.0\n",
              "DRR079365_1    100.0\n",
              "DRR079369_1    100.0\n",
              "DRR079371_1    100.0\n",
              "DRR079375_1    100.0\n",
              "DRR079377_1    100.0\n",
              "DRR079381_1    100.0\n",
              "DRR079383_1    100.0\n",
              "DRR079387_1    100.0\n",
              "DRR079389_1    100.0\n",
              "DRR079393_1    100.0\n",
              "DRR079395_1    100.0\n",
              "DRR079399_1    100.0\n",
              "DRR079401_1    100.0\n",
              "DRR079405_1    100.0\n",
              "DRR079407_1    100.0\n",
              "DRR079411_1    100.0\n",
              "DRR079413_1    100.0\n",
              "DRR079417_1    100.0\n",
              "DRR079419_1    100.0\n",
              "DRR079423_1    100.0\n",
              "DRR079425_1    100.0\n",
              "DRR079429_1    100.0\n",
              "dtype: float64"
            ]
          },
          "metadata": {},
          "execution_count": 77
        }
      ],
      "source": [
        "rel_table.sum()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 79,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bilc6YMNUuvx",
        "outputId": "3d8f1308-de25-480b-cc92-90dfd706ba17"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  /content/sample_data/taxonomy.qza\n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/checksums.md5  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/e9bf12ee-681c-4eae-8f44-7d45b5030199/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/e9bf12ee-681c-4eae-8f44-7d45b5030199/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/e9bf12ee-681c-4eae-8f44-7d45b5030199/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/e9bf12ee-681c-4eae-8f44-7d45b5030199/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/6d0c1726-2a4c-4bdc-a23d-9334f813bbfd/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/6d0c1726-2a4c-4bdc-a23d-9334f813bbfd/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/6d0c1726-2a4c-4bdc-a23d-9334f813bbfd/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/6d0c1726-2a4c-4bdc-a23d-9334f813bbfd/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/390789c9-6f44-4d21-a30d-5281c9cecce5/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/aacdcb16-5bad-48f7-ac7d-3f30c35b0d67/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/aacdcb16-5bad-48f7-ac7d-3f30c35b0d67/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/aacdcb16-5bad-48f7-ac7d-3f30c35b0d67/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/aacdcb16-5bad-48f7-ac7d-3f30c35b0d67/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/82a5cdd1-0d21-417f-b98c-b656f97a6d12/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/5fddc947-1a8a-410d-b4ed-123cc5990533/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/5fddc947-1a8a-410d-b4ed-123cc5990533/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/5fddc947-1a8a-410d-b4ed-123cc5990533/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/5fddc947-1a8a-410d-b4ed-123cc5990533/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/47fc85c0-7cbe-46f4-a4ea-6eb3ce3e4d47/citations.bib  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/47fc85c0-7cbe-46f4-a4ea-6eb3ce3e4d47/VERSION  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/47fc85c0-7cbe-46f4-a4ea-6eb3ce3e4d47/metadata.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/artifacts/47fc85c0-7cbe-46f4-a4ea-6eb3ce3e4d47/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/provenance/action/action.yaml  \n",
            "  inflating: 670ddb1f-8a32-434c-b6b0-142375394531/data/taxonomy.tsv  \n"
          ]
        }
      ],
      "source": [
        "!unzip /content/sample_data/taxonomy.qza"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 80,
      "metadata": {
        "id": "hsRBQYA9GfkA"
      },
      "outputs": [],
      "source": [
        "taxon=pd.read_table(\"/content/sample_data/670ddb1f-8a32-434c-b6b0-142375394531/data/taxonomy.tsv\", index_col=0)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 81,
      "metadata": {
        "id": "fksfdZ-HG7r4"
      },
      "outputs": [],
      "source": [
        "taxon[['kingdom', 'phylum', 'class', 'order', 'family', 'genus', 'species']]=taxon['Taxon'].str.split(';', expand=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 82,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "d5aeZJxLn5ZC",
        "outputId": "c41bd94f-6ffc-42a1-b02e-6ae7766456da"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Index(['Taxon', 'Confidence', 'kingdom', 'phylum', 'class', 'order', 'family',\n",
              "       'genus', 'species'],\n",
              "      dtype='object')"
            ]
          },
          "metadata": {},
          "execution_count": 82
        }
      ],
      "source": [
        "taxon.columns\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 83,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 901
        },
        "id": "7V40jVzB7h7v",
        "outputId": "8d266d3a-c1b5-4d11-aaab-5e8656490f40"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                              Taxon  \\\n",
              "Feature ID                                                                            \n",
              "4a03d0dd563b718371ad1cc7654e853d  k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...   \n",
              "0d144ccee08439248107ed843c526622  k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...   \n",
              "b2833106fa09c1c09255eec4c5861d72  k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...   \n",
              "26296c0695bce607aaeab9a7e181dc8b  k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...   \n",
              "dc6c62da559b74f4f140607729b7d891  k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...   \n",
              "...                                                                             ...   \n",
              "ff7a93c29382effb4ef222f2f85fb0f1  k__Bacteria; p__Actinobacteria; c__Actinobacte...   \n",
              "399de2e5861bdf021e5ed1b8689dc462  k__Bacteria; p__Firmicutes; c__Erysipelotrichi...   \n",
              "8ed733cbdfc288387161a38749f60d26  k__Bacteria; p__Actinobacteria; c__Actinobacte...   \n",
              "343407c4e2672edc029c8fb77e64ee89  k__Bacteria; p__Firmicutes; c__Clostridia; o__...   \n",
              "75c9b98212f0547fa8ecebf2d845af6d  k__Bacteria; p__Firmicutes; c__Clostridia; o__...   \n",
              "\n",
              "                                  Confidence      kingdom              phylum  \\\n",
              "Feature ID                                                                      \n",
              "4a03d0dd563b718371ad1cc7654e853d    0.999930  k__Bacteria    p__Bacteroidetes   \n",
              "0d144ccee08439248107ed843c526622    0.999701  k__Bacteria    p__Bacteroidetes   \n",
              "b2833106fa09c1c09255eec4c5861d72    0.999896  k__Bacteria    p__Bacteroidetes   \n",
              "26296c0695bce607aaeab9a7e181dc8b    0.999947  k__Bacteria    p__Bacteroidetes   \n",
              "dc6c62da559b74f4f140607729b7d891    0.999762  k__Bacteria    p__Bacteroidetes   \n",
              "...                                      ...          ...                 ...   \n",
              "ff7a93c29382effb4ef222f2f85fb0f1    0.999979  k__Bacteria   p__Actinobacteria   \n",
              "399de2e5861bdf021e5ed1b8689dc462    0.969562  k__Bacteria       p__Firmicutes   \n",
              "8ed733cbdfc288387161a38749f60d26    0.996359  k__Bacteria   p__Actinobacteria   \n",
              "343407c4e2672edc029c8fb77e64ee89    0.998670  k__Bacteria       p__Firmicutes   \n",
              "75c9b98212f0547fa8ecebf2d845af6d    0.999987  k__Bacteria       p__Firmicutes   \n",
              "\n",
              "                                                class                   order  \\\n",
              "Feature ID                                                                      \n",
              "4a03d0dd563b718371ad1cc7654e853d       c__Bacteroidia        o__Bacteroidales   \n",
              "0d144ccee08439248107ed843c526622       c__Bacteroidia        o__Bacteroidales   \n",
              "b2833106fa09c1c09255eec4c5861d72       c__Bacteroidia        o__Bacteroidales   \n",
              "26296c0695bce607aaeab9a7e181dc8b       c__Bacteroidia        o__Bacteroidales   \n",
              "dc6c62da559b74f4f140607729b7d891       c__Bacteroidia        o__Bacteroidales   \n",
              "...                                               ...                     ...   \n",
              "ff7a93c29382effb4ef222f2f85fb0f1    c__Actinobacteria      o__Actinomycetales   \n",
              "399de2e5861bdf021e5ed1b8689dc462   c__Erysipelotrichi   o__Erysipelotrichales   \n",
              "8ed733cbdfc288387161a38749f60d26    c__Actinobacteria      o__Actinomycetales   \n",
              "343407c4e2672edc029c8fb77e64ee89        c__Clostridia        o__Clostridiales   \n",
              "75c9b98212f0547fa8ecebf2d845af6d        c__Clostridia        o__Clostridiales   \n",
              "\n",
              "                                                   family  \\\n",
              "Feature ID                                                  \n",
              "4a03d0dd563b718371ad1cc7654e853d        f__Prevotellaceae   \n",
              "0d144ccee08439248107ed843c526622        f__Prevotellaceae   \n",
              "b2833106fa09c1c09255eec4c5861d72        f__Prevotellaceae   \n",
              "26296c0695bce607aaeab9a7e181dc8b        f__Prevotellaceae   \n",
              "dc6c62da559b74f4f140607729b7d891        f__Prevotellaceae   \n",
              "...                                                   ...   \n",
              "ff7a93c29382effb4ef222f2f85fb0f1      f__Actinomycetaceae   \n",
              "399de2e5861bdf021e5ed1b8689dc462   f__Erysipelotrichaceae   \n",
              "8ed733cbdfc288387161a38749f60d26    f__Corynebacteriaceae   \n",
              "343407c4e2672edc029c8fb77e64ee89       f__Lachnospiraceae   \n",
              "75c9b98212f0547fa8ecebf2d845af6d       f__Lachnospiraceae   \n",
              "\n",
              "                                                genus             species  \n",
              "Feature ID                                                                 \n",
              "4a03d0dd563b718371ad1cc7654e853d        g__Prevotella   s__melaninogenica  \n",
              "0d144ccee08439248107ed843c526622        g__Prevotella   s__melaninogenica  \n",
              "b2833106fa09c1c09255eec4c5861d72        g__Prevotella   s__melaninogenica  \n",
              "26296c0695bce607aaeab9a7e181dc8b        g__Prevotella   s__melaninogenica  \n",
              "dc6c62da559b74f4f140607729b7d891        g__Prevotella   s__melaninogenica  \n",
              "...                                               ...                 ...  \n",
              "ff7a93c29382effb4ef222f2f85fb0f1       g__Actinomyces                 s__  \n",
              "399de2e5861bdf021e5ed1b8689dc462                 None                None  \n",
              "8ed733cbdfc288387161a38749f60d26   g__Corynebacterium                 s__  \n",
              "343407c4e2672edc029c8fb77e64ee89      g__Oribacterium                 s__  \n",
              "75c9b98212f0547fa8ecebf2d845af6d         g__Catonella                 s__  \n",
              "\n",
              "[484 rows x 9 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-6b286e10-50bc-4399-8ab4-a7d24c2aa51f\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Taxon</th>\n",
              "      <th>Confidence</th>\n",
              "      <th>kingdom</th>\n",
              "      <th>phylum</th>\n",
              "      <th>class</th>\n",
              "      <th>order</th>\n",
              "      <th>family</th>\n",
              "      <th>genus</th>\n",
              "      <th>species</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>Feature ID</th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>4a03d0dd563b718371ad1cc7654e853d</th>\n",
              "      <td>k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...</td>\n",
              "      <td>0.999930</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Bacteroidetes</td>\n",
              "      <td>c__Bacteroidia</td>\n",
              "      <td>o__Bacteroidales</td>\n",
              "      <td>f__Prevotellaceae</td>\n",
              "      <td>g__Prevotella</td>\n",
              "      <td>s__melaninogenica</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>0d144ccee08439248107ed843c526622</th>\n",
              "      <td>k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...</td>\n",
              "      <td>0.999701</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Bacteroidetes</td>\n",
              "      <td>c__Bacteroidia</td>\n",
              "      <td>o__Bacteroidales</td>\n",
              "      <td>f__Prevotellaceae</td>\n",
              "      <td>g__Prevotella</td>\n",
              "      <td>s__melaninogenica</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>b2833106fa09c1c09255eec4c5861d72</th>\n",
              "      <td>k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...</td>\n",
              "      <td>0.999896</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Bacteroidetes</td>\n",
              "      <td>c__Bacteroidia</td>\n",
              "      <td>o__Bacteroidales</td>\n",
              "      <td>f__Prevotellaceae</td>\n",
              "      <td>g__Prevotella</td>\n",
              "      <td>s__melaninogenica</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>26296c0695bce607aaeab9a7e181dc8b</th>\n",
              "      <td>k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...</td>\n",
              "      <td>0.999947</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Bacteroidetes</td>\n",
              "      <td>c__Bacteroidia</td>\n",
              "      <td>o__Bacteroidales</td>\n",
              "      <td>f__Prevotellaceae</td>\n",
              "      <td>g__Prevotella</td>\n",
              "      <td>s__melaninogenica</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>dc6c62da559b74f4f140607729b7d891</th>\n",
              "      <td>k__Bacteria; p__Bacteroidetes; c__Bacteroidia;...</td>\n",
              "      <td>0.999762</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Bacteroidetes</td>\n",
              "      <td>c__Bacteroidia</td>\n",
              "      <td>o__Bacteroidales</td>\n",
              "      <td>f__Prevotellaceae</td>\n",
              "      <td>g__Prevotella</td>\n",
              "      <td>s__melaninogenica</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>ff7a93c29382effb4ef222f2f85fb0f1</th>\n",
              "      <td>k__Bacteria; p__Actinobacteria; c__Actinobacte...</td>\n",
              "      <td>0.999979</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Actinobacteria</td>\n",
              "      <td>c__Actinobacteria</td>\n",
              "      <td>o__Actinomycetales</td>\n",
              "      <td>f__Actinomycetaceae</td>\n",
              "      <td>g__Actinomyces</td>\n",
              "      <td>s__</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>399de2e5861bdf021e5ed1b8689dc462</th>\n",
              "      <td>k__Bacteria; p__Firmicutes; c__Erysipelotrichi...</td>\n",
              "      <td>0.969562</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Firmicutes</td>\n",
              "      <td>c__Erysipelotrichi</td>\n",
              "      <td>o__Erysipelotrichales</td>\n",
              "      <td>f__Erysipelotrichaceae</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8ed733cbdfc288387161a38749f60d26</th>\n",
              "      <td>k__Bacteria; p__Actinobacteria; c__Actinobacte...</td>\n",
              "      <td>0.996359</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Actinobacteria</td>\n",
              "      <td>c__Actinobacteria</td>\n",
              "      <td>o__Actinomycetales</td>\n",
              "      <td>f__Corynebacteriaceae</td>\n",
              "      <td>g__Corynebacterium</td>\n",
              "      <td>s__</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>343407c4e2672edc029c8fb77e64ee89</th>\n",
              "      <td>k__Bacteria; p__Firmicutes; c__Clostridia; o__...</td>\n",
              "      <td>0.998670</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Firmicutes</td>\n",
              "      <td>c__Clostridia</td>\n",
              "      <td>o__Clostridiales</td>\n",
              "      <td>f__Lachnospiraceae</td>\n",
              "      <td>g__Oribacterium</td>\n",
              "      <td>s__</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>75c9b98212f0547fa8ecebf2d845af6d</th>\n",
              "      <td>k__Bacteria; p__Firmicutes; c__Clostridia; o__...</td>\n",
              "      <td>0.999987</td>\n",
              "      <td>k__Bacteria</td>\n",
              "      <td>p__Firmicutes</td>\n",
              "      <td>c__Clostridia</td>\n",
              "      <td>o__Clostridiales</td>\n",
              "      <td>f__Lachnospiraceae</td>\n",
              "      <td>g__Catonella</td>\n",
              "      <td>s__</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>484 rows × 9 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-6b286e10-50bc-4399-8ab4-a7d24c2aa51f')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-6b286e10-50bc-4399-8ab4-a7d24c2aa51f button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-6b286e10-50bc-4399-8ab4-a7d24c2aa51f');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 83
        }
      ],
      "source": [
        "taxon"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 84,
      "metadata": {
        "id": "4UExYd62Y7K5"
      },
      "outputs": [],
      "source": [
        "taxa_results=pd.merge(taxon, rel_table, left_index=True, right_index=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 85,
      "metadata": {
        "id": "BZNkItzIdPmF"
      },
      "outputs": [],
      "source": [
        "sumtax=taxa_results.groupby('genus').sum()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 86,
      "metadata": {
        "id": "AYH6XxI2fjV3"
      },
      "outputs": [],
      "source": [
        "sumtax=sumtax.drop(['Confidence'], axis=1)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 87,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "DSojKPOXd2nu",
        "outputId": "292079ec-06ec-41c4-ac92-c7f65835cc95"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "DRR079323_1     92.682927\n",
              "DRR079327_1    100.000000\n",
              "DRR079329_1    100.000000\n",
              "DRR079333_1     99.024390\n",
              "DRR079335_1     80.000000\n",
              "DRR079339_1    100.000000\n",
              "DRR079341_1     92.926829\n",
              "DRR079345_1     96.097561\n",
              "DRR079347_1     93.414634\n",
              "DRR079351_1    100.000000\n",
              "DRR079353_1     96.097561\n",
              "DRR079357_1     96.097561\n",
              "DRR079359_1     93.902439\n",
              "DRR079363_1    100.000000\n",
              "DRR079365_1     96.585366\n",
              "DRR079369_1     97.073171\n",
              "DRR079371_1     84.390244\n",
              "DRR079375_1     87.073171\n",
              "DRR079377_1    100.000000\n",
              "DRR079381_1     84.878049\n",
              "DRR079383_1     90.243902\n",
              "DRR079387_1     89.512195\n",
              "DRR079389_1     89.268293\n",
              "DRR079393_1     90.000000\n",
              "DRR079395_1     85.853659\n",
              "DRR079399_1    100.000000\n",
              "DRR079401_1     97.804878\n",
              "DRR079405_1     87.317073\n",
              "DRR079407_1    100.000000\n",
              "DRR079411_1     84.146341\n",
              "DRR079413_1     91.463415\n",
              "DRR079417_1     92.682927\n",
              "DRR079419_1     96.829268\n",
              "DRR079423_1     92.926829\n",
              "DRR079425_1    100.000000\n",
              "DRR079429_1    100.000000\n",
              "dtype: float64"
            ]
          },
          "metadata": {},
          "execution_count": 87
        }
      ],
      "source": [
        "sumtax.sum()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 88,
      "metadata": {
        "id": "lp7tRzuGj3Ic"
      },
      "outputs": [],
      "source": [
        "sumtax=sumtax.T"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 89,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "UZyA9pNHjlwO",
        "outputId": "09321270-60b6-497a-e554-1ca60edf59c4"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "genus              g__   g__Abiotrophia   g__Actinomyces   g__Catonella  \\\n",
              "DRR079323_1   2.926829         0.000000         0.000000       0.000000   \n",
              "DRR079327_1  12.439024         0.000000         0.000000       0.000000   \n",
              "DRR079329_1  22.926829         0.975610         0.000000       0.000000   \n",
              "DRR079333_1  15.853659         0.000000         0.000000       0.000000   \n",
              "DRR079335_1   3.902439         0.000000         0.000000       0.000000   \n",
              "DRR079339_1  24.634146         0.000000         0.000000       0.000000   \n",
              "DRR079341_1   0.000000         0.000000         0.000000       0.000000   \n",
              "DRR079345_1   0.487805         0.000000         0.000000       0.000000   \n",
              "DRR079347_1   0.000000         0.000000         0.000000       0.000000   \n",
              "DRR079351_1   5.853659         0.000000         0.000000       0.000000   \n",
              "DRR079353_1   0.000000         0.000000         0.000000       0.000000   \n",
              "DRR079357_1   1.951220         0.000000         0.000000       0.000000   \n",
              "DRR079359_1   0.731707         0.000000         0.000000       0.000000   \n",
              "DRR079363_1   1.707317         0.000000         0.000000       0.000000   \n",
              "DRR079365_1   3.414634         0.000000         0.000000       0.000000   \n",
              "DRR079369_1   1.707317         0.000000         0.000000       0.000000   \n",
              "DRR079371_1   0.000000         0.000000         0.000000       0.000000   \n",
              "DRR079375_1   2.682927         0.000000         0.000000       1.951220   \n",
              "DRR079377_1  11.707317         0.000000         0.000000       0.000000   \n",
              "DRR079381_1   1.463415         0.000000         0.000000       0.000000   \n",
              "DRR079383_1  26.341463         0.000000         0.487805       0.000000   \n",
              "DRR079387_1   4.634146         0.000000         0.000000       0.000000   \n",
              "DRR079389_1  27.804878         0.000000         0.000000       0.487805   \n",
              "DRR079393_1   0.731707         0.000000         0.000000       0.000000   \n",
              "DRR079395_1   5.609756         2.682927         4.146341       0.000000   \n",
              "DRR079399_1  16.829268         0.000000         4.390244       0.000000   \n",
              "DRR079401_1  16.097561         0.000000         2.439024       0.000000   \n",
              "DRR079405_1   4.146341         0.000000         0.000000       2.195122   \n",
              "DRR079407_1  35.121951         1.463415         0.000000       0.000000   \n",
              "DRR079411_1   6.829268         0.000000         2.926829       0.000000   \n",
              "DRR079413_1  15.609756         0.000000         0.000000       0.000000   \n",
              "DRR079417_1  10.487805         0.000000         0.000000       0.000000   \n",
              "DRR079419_1  17.804878         0.000000         0.000000       0.000000   \n",
              "DRR079423_1  11.219512         0.000000         0.000000       0.000000   \n",
              "DRR079425_1  17.560976         0.000000         0.000000       0.243902   \n",
              "DRR079429_1  19.512195         0.000000         0.487805       0.000000   \n",
              "\n",
              "genus         g__Corynebacterium   g__Dialister   g__Filifactor   g__Gemella  \\\n",
              "DRR079323_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079327_1             0.000000       1.463415        0.000000     0.000000   \n",
              "DRR079329_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079333_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079335_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079339_1             0.000000       0.487805        0.000000     0.000000   \n",
              "DRR079341_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079345_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079347_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079351_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079353_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079357_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079359_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079363_1             0.000000       0.000000        0.000000     5.365854   \n",
              "DRR079365_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079369_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079371_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079375_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079377_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079381_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079383_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079387_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079389_1             0.000000       0.487805        0.000000     0.000000   \n",
              "DRR079393_1             0.000000       0.487805        0.000000     0.000000   \n",
              "DRR079395_1             0.243902       0.000000        2.682927     0.000000   \n",
              "DRR079399_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079401_1             0.000000       0.000000        0.731707     0.000000   \n",
              "DRR079405_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079407_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079411_1             0.000000       0.000000        2.439024     0.000000   \n",
              "DRR079413_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079417_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079419_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079423_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079425_1             0.000000       0.000000        0.000000     0.000000   \n",
              "DRR079429_1             0.000000       0.000000        0.000000     0.000000   \n",
              "\n",
              "genus         g__Granulicatella   g__Lachnoanaerobaculum   g__Megasphaera  \\\n",
              "DRR079323_1            5.609756                 2.926829         1.463415   \n",
              "DRR079327_1           13.902439                 0.000000         0.000000   \n",
              "DRR079329_1           14.390244                 0.000000         0.000000   \n",
              "DRR079333_1           20.975610                 0.000000         0.000000   \n",
              "DRR079335_1           20.000000                 0.000000         0.000000   \n",
              "DRR079339_1           13.902439                 0.000000         0.000000   \n",
              "DRR079341_1           18.780488                 0.000000         0.000000   \n",
              "DRR079345_1           12.926829                 0.000000         0.000000   \n",
              "DRR079347_1           13.658537                 0.000000         0.000000   \n",
              "DRR079351_1           16.341463                 0.000000         0.000000   \n",
              "DRR079353_1           13.902439                 0.000000         0.000000   \n",
              "DRR079357_1           10.487805                 0.975610         0.000000   \n",
              "DRR079359_1            9.268293                 0.000000         0.000000   \n",
              "DRR079363_1            6.341463                 1.463415         0.000000   \n",
              "DRR079365_1            5.609756                 0.000000         0.000000   \n",
              "DRR079369_1            6.341463                 1.219512         1.707317   \n",
              "DRR079371_1           15.365854                 0.000000         0.000000   \n",
              "DRR079375_1           18.048780                 0.000000         0.000000   \n",
              "DRR079377_1            8.048780                 2.682927         0.731707   \n",
              "DRR079381_1           13.658537                 0.000000         0.000000   \n",
              "DRR079383_1           14.146341                 0.000000         0.000000   \n",
              "DRR079387_1           10.975610                 0.000000         0.000000   \n",
              "DRR079389_1            7.560976                 0.000000         0.000000   \n",
              "DRR079393_1           11.219512                 0.975610         0.975610   \n",
              "DRR079395_1           14.390244                 1.219512         0.000000   \n",
              "DRR079399_1           14.878049                 0.000000         0.000000   \n",
              "DRR079401_1           10.731707                 0.731707         0.000000   \n",
              "DRR079405_1            8.048780                 0.000000         0.000000   \n",
              "DRR079407_1           13.414634                 0.000000         0.000000   \n",
              "DRR079411_1           12.682927                 0.000000         0.000000   \n",
              "DRR079413_1           11.463415                 0.000000         0.000000   \n",
              "DRR079417_1            8.780488                 0.000000         0.000000   \n",
              "DRR079419_1            8.536585                 0.000000         2.439024   \n",
              "DRR079423_1            9.756098                 0.000000         0.000000   \n",
              "DRR079425_1           10.731707                 0.000000         0.000000   \n",
              "DRR079429_1            9.024390                 0.000000         0.000000   \n",
              "\n",
              "genus         g__Oribacterium   g__Parvimonas   g__Peptococcus  \\\n",
              "DRR079323_1          0.000000        0.000000         0.000000   \n",
              "DRR079327_1          0.000000        0.000000         0.000000   \n",
              "DRR079329_1          0.000000        0.487805         0.000000   \n",
              "DRR079333_1          0.000000        2.195122         0.000000   \n",
              "DRR079335_1          0.000000        1.219512         0.000000   \n",
              "DRR079339_1          0.000000        0.000000         0.000000   \n",
              "DRR079341_1          0.731707        0.000000         0.000000   \n",
              "DRR079345_1          0.000000        0.000000         0.000000   \n",
              "DRR079347_1          0.000000        0.000000         0.000000   \n",
              "DRR079351_1          0.000000        0.000000         0.000000   \n",
              "DRR079353_1          0.000000        0.000000         0.000000   \n",
              "DRR079357_1          0.000000        0.000000         0.000000   \n",
              "DRR079359_1          0.000000        0.000000         0.000000   \n",
              "DRR079363_1          0.000000        0.000000         0.000000   \n",
              "DRR079365_1          0.000000        0.000000         0.000000   \n",
              "DRR079369_1          0.000000        0.000000         0.000000   \n",
              "DRR079371_1          0.000000        0.000000         0.000000   \n",
              "DRR079375_1          1.951220        0.731707         0.000000   \n",
              "DRR079377_1          0.000000        0.000000         0.243902   \n",
              "DRR079381_1          0.000000        0.000000         0.000000   \n",
              "DRR079383_1          0.000000        4.878049         0.000000   \n",
              "DRR079387_1          0.000000        0.000000         0.000000   \n",
              "DRR079389_1          0.000000        7.317073         0.000000   \n",
              "DRR079393_1          0.000000        0.000000         1.463415   \n",
              "DRR079395_1          0.000000        5.121951         0.000000   \n",
              "DRR079399_1          0.243902        1.707317         0.731707   \n",
              "DRR079401_1          0.000000        3.170732         3.414634   \n",
              "DRR079405_1          0.000000        1.707317         0.000000   \n",
              "DRR079407_1          0.000000        6.829268         0.000000   \n",
              "DRR079411_1          0.000000        0.000000         1.463415   \n",
              "DRR079413_1          0.000000        0.000000         0.000000   \n",
              "DRR079417_1          0.000000        0.000000         0.000000   \n",
              "DRR079419_1          0.000000        0.000000         0.000000   \n",
              "DRR079423_1          0.000000        0.000000         0.000000   \n",
              "DRR079425_1          0.000000        0.000000         0.000000   \n",
              "DRR079429_1          0.000000        0.000000         0.731707   \n",
              "\n",
              "genus         g__Peptostreptococcus   g__Porphyromonas   g__Prevotella  \\\n",
              "DRR079323_1                0.000000           4.146341       43.170732   \n",
              "DRR079327_1                0.000000           0.000000       45.121951   \n",
              "DRR079329_1                0.000000           0.000000       34.390244   \n",
              "DRR079333_1                0.975610           0.000000       38.292683   \n",
              "DRR079335_1                0.975610           9.512195       19.512195   \n",
              "DRR079339_1                0.000000           0.000000       36.585366   \n",
              "DRR079341_1                0.000000           0.000000       21.951220   \n",
              "DRR079345_1                0.731707           0.000000       51.219512   \n",
              "DRR079347_1                0.000000           0.000000       27.073171   \n",
              "DRR079351_1                0.000000           0.000000       45.609756   \n",
              "DRR079353_1                0.000000           0.000000       17.804878   \n",
              "DRR079357_1                0.000000           0.000000       43.170732   \n",
              "DRR079359_1                0.000000           0.000000       42.195122   \n",
              "DRR079363_1                2.195122           0.000000       52.682927   \n",
              "DRR079365_1                0.000000           0.000000       45.609756   \n",
              "DRR079369_1                1.219512           2.195122       53.414634   \n",
              "DRR079371_1                0.000000           2.195122       28.536585   \n",
              "DRR079375_1                1.951220           3.414634       36.097561   \n",
              "DRR079377_1                0.000000           1.707317       15.853659   \n",
              "DRR079381_1                0.000000           6.341463       34.634146   \n",
              "DRR079383_1                0.000000           0.000000        8.048780   \n",
              "DRR079387_1                2.439024           6.341463       30.243902   \n",
              "DRR079389_1                0.000000           0.000000       12.682927   \n",
              "DRR079393_1                0.000000           9.756098       35.609756   \n",
              "DRR079395_1                0.000000           0.000000        3.658537   \n",
              "DRR079399_1                0.000000           0.000000       30.731707   \n",
              "DRR079401_1                0.975610           4.390244       29.512195   \n",
              "DRR079405_1                1.463415           0.000000       23.902439   \n",
              "DRR079407_1                0.000000           0.000000        4.390244   \n",
              "DRR079411_1                0.000000           0.000000       13.658537   \n",
              "DRR079413_1                1.707317           0.000000       24.634146   \n",
              "DRR079417_1                0.000000           0.000000       37.073171   \n",
              "DRR079419_1                1.463415           0.000000       31.219512   \n",
              "DRR079423_1                2.195122           0.000000       35.609756   \n",
              "DRR079425_1                0.000000           0.000000       38.536585   \n",
              "DRR079429_1                0.000000           0.000000       35.853659   \n",
              "\n",
              "genus         g__Streptococcus   g__Veillonella   g__[Prevotella]  \n",
              "DRR079323_1          18.048780         9.268293          5.121951  \n",
              "DRR079327_1          13.658537         8.048780          5.365854  \n",
              "DRR079329_1          16.097561         6.829268          3.902439  \n",
              "DRR079333_1           5.121951         4.634146         10.975610  \n",
              "DRR079335_1          18.780488         6.097561          0.000000  \n",
              "DRR079339_1           8.536585         5.121951         10.731707  \n",
              "DRR079341_1          32.682927        17.317073          1.463415  \n",
              "DRR079345_1          16.097561         8.780488          5.853659  \n",
              "DRR079347_1          36.585366        15.121951          0.975610  \n",
              "DRR079351_1          19.268293         8.536585          4.390244  \n",
              "DRR079353_1          49.024390        14.878049          0.487805  \n",
              "DRR079357_1          24.878049        10.975610          3.658537  \n",
              "DRR079359_1          19.512195        22.195122          0.000000  \n",
              "DRR079363_1          14.146341        16.097561          0.000000  \n",
              "DRR079365_1          21.951220        20.000000          0.000000  \n",
              "DRR079369_1          15.121951        13.170732          0.975610  \n",
              "DRR079371_1          22.682927        15.609756          0.000000  \n",
              "DRR079375_1           7.804878         6.585366          5.853659  \n",
              "DRR079377_1          29.512195        29.512195          0.000000  \n",
              "DRR079381_1          10.731707         7.804878         10.243902  \n",
              "DRR079383_1          24.634146        11.707317          0.000000  \n",
              "DRR079387_1           6.097561        11.219512         17.560976  \n",
              "DRR079389_1          25.121951         7.804878          0.000000  \n",
              "DRR079393_1           5.853659        13.414634          9.512195  \n",
              "DRR079395_1          31.219512         7.317073          7.560976  \n",
              "DRR079399_1          14.146341         8.292683          8.048780  \n",
              "DRR079401_1          15.121951         3.658537          6.829268  \n",
              "DRR079405_1          12.195122        23.902439          9.756098  \n",
              "DRR079407_1          17.804878         7.073171         13.902439  \n",
              "DRR079411_1          10.000000        12.926829         21.219512  \n",
              "DRR079413_1           9.512195        26.585366          1.951220  \n",
              "DRR079417_1          17.073171        19.268293          0.000000  \n",
              "DRR079419_1           4.390244        30.975610          0.000000  \n",
              "DRR079423_1          13.170732        20.975610          0.000000  \n",
              "DRR079425_1           9.268293        23.658537          0.000000  \n",
              "DRR079429_1          16.585366        17.804878          0.000000  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-9bdf8b38-2615-44bf-93a6-d876e9f7e311\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th>genus</th>\n",
              "      <th>g__</th>\n",
              "      <th>g__Abiotrophia</th>\n",
              "      <th>g__Actinomyces</th>\n",
              "      <th>g__Catonella</th>\n",
              "      <th>g__Corynebacterium</th>\n",
              "      <th>g__Dialister</th>\n",
              "      <th>g__Filifactor</th>\n",
              "      <th>g__Gemella</th>\n",
              "      <th>g__Granulicatella</th>\n",
              "      <th>g__Lachnoanaerobaculum</th>\n",
              "      <th>g__Megasphaera</th>\n",
              "      <th>g__Oribacterium</th>\n",
              "      <th>g__Parvimonas</th>\n",
              "      <th>g__Peptococcus</th>\n",
              "      <th>g__Peptostreptococcus</th>\n",
              "      <th>g__Porphyromonas</th>\n",
              "      <th>g__Prevotella</th>\n",
              "      <th>g__Streptococcus</th>\n",
              "      <th>g__Veillonella</th>\n",
              "      <th>g__[Prevotella]</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>DRR079323_1</th>\n",
              "      <td>2.926829</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>5.609756</td>\n",
              "      <td>2.926829</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.146341</td>\n",
              "      <td>43.170732</td>\n",
              "      <td>18.048780</td>\n",
              "      <td>9.268293</td>\n",
              "      <td>5.121951</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079327_1</th>\n",
              "      <td>12.439024</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.902439</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>45.121951</td>\n",
              "      <td>13.658537</td>\n",
              "      <td>8.048780</td>\n",
              "      <td>5.365854</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079329_1</th>\n",
              "      <td>22.926829</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>14.390244</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>34.390244</td>\n",
              "      <td>16.097561</td>\n",
              "      <td>6.829268</td>\n",
              "      <td>3.902439</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079333_1</th>\n",
              "      <td>15.853659</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>20.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>38.292683</td>\n",
              "      <td>5.121951</td>\n",
              "      <td>4.634146</td>\n",
              "      <td>10.975610</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079335_1</th>\n",
              "      <td>3.902439</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>20.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.219512</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>9.512195</td>\n",
              "      <td>19.512195</td>\n",
              "      <td>18.780488</td>\n",
              "      <td>6.097561</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079339_1</th>\n",
              "      <td>24.634146</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.902439</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>36.585366</td>\n",
              "      <td>8.536585</td>\n",
              "      <td>5.121951</td>\n",
              "      <td>10.731707</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079341_1</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>18.780488</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>21.951220</td>\n",
              "      <td>32.682927</td>\n",
              "      <td>17.317073</td>\n",
              "      <td>1.463415</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079345_1</th>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>12.926829</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>51.219512</td>\n",
              "      <td>16.097561</td>\n",
              "      <td>8.780488</td>\n",
              "      <td>5.853659</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079347_1</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.658537</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>27.073171</td>\n",
              "      <td>36.585366</td>\n",
              "      <td>15.121951</td>\n",
              "      <td>0.975610</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079351_1</th>\n",
              "      <td>5.853659</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>16.341463</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>45.609756</td>\n",
              "      <td>19.268293</td>\n",
              "      <td>8.536585</td>\n",
              "      <td>4.390244</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079353_1</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.902439</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>17.804878</td>\n",
              "      <td>49.024390</td>\n",
              "      <td>14.878049</td>\n",
              "      <td>0.487805</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079357_1</th>\n",
              "      <td>1.951220</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>10.487805</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>43.170732</td>\n",
              "      <td>24.878049</td>\n",
              "      <td>10.975610</td>\n",
              "      <td>3.658537</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079359_1</th>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>9.268293</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>42.195122</td>\n",
              "      <td>19.512195</td>\n",
              "      <td>22.195122</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079363_1</th>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>5.365854</td>\n",
              "      <td>6.341463</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>52.682927</td>\n",
              "      <td>14.146341</td>\n",
              "      <td>16.097561</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079365_1</th>\n",
              "      <td>3.414634</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>5.609756</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>45.609756</td>\n",
              "      <td>21.951220</td>\n",
              "      <td>20.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079369_1</th>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>6.341463</td>\n",
              "      <td>1.219512</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.219512</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>53.414634</td>\n",
              "      <td>15.121951</td>\n",
              "      <td>13.170732</td>\n",
              "      <td>0.975610</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079371_1</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>15.365854</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>28.536585</td>\n",
              "      <td>22.682927</td>\n",
              "      <td>15.609756</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079375_1</th>\n",
              "      <td>2.682927</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.951220</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>18.048780</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.951220</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.951220</td>\n",
              "      <td>3.414634</td>\n",
              "      <td>36.097561</td>\n",
              "      <td>7.804878</td>\n",
              "      <td>6.585366</td>\n",
              "      <td>5.853659</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079377_1</th>\n",
              "      <td>11.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>8.048780</td>\n",
              "      <td>2.682927</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.243902</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>15.853659</td>\n",
              "      <td>29.512195</td>\n",
              "      <td>29.512195</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079381_1</th>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.658537</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>6.341463</td>\n",
              "      <td>34.634146</td>\n",
              "      <td>10.731707</td>\n",
              "      <td>7.804878</td>\n",
              "      <td>10.243902</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079383_1</th>\n",
              "      <td>26.341463</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>14.146341</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.878049</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>8.048780</td>\n",
              "      <td>24.634146</td>\n",
              "      <td>11.707317</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079387_1</th>\n",
              "      <td>4.634146</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>10.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.439024</td>\n",
              "      <td>6.341463</td>\n",
              "      <td>30.243902</td>\n",
              "      <td>6.097561</td>\n",
              "      <td>11.219512</td>\n",
              "      <td>17.560976</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079389_1</th>\n",
              "      <td>27.804878</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>7.560976</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>7.317073</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>12.682927</td>\n",
              "      <td>25.121951</td>\n",
              "      <td>7.804878</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079393_1</th>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>11.219512</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>9.756098</td>\n",
              "      <td>35.609756</td>\n",
              "      <td>5.853659</td>\n",
              "      <td>13.414634</td>\n",
              "      <td>9.512195</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079395_1</th>\n",
              "      <td>5.609756</td>\n",
              "      <td>2.682927</td>\n",
              "      <td>4.146341</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.243902</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.682927</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>14.390244</td>\n",
              "      <td>1.219512</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>5.121951</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>3.658537</td>\n",
              "      <td>31.219512</td>\n",
              "      <td>7.317073</td>\n",
              "      <td>7.560976</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079399_1</th>\n",
              "      <td>16.829268</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>14.878049</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.243902</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>30.731707</td>\n",
              "      <td>14.146341</td>\n",
              "      <td>8.292683</td>\n",
              "      <td>8.048780</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079401_1</th>\n",
              "      <td>16.097561</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.439024</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>10.731707</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>3.170732</td>\n",
              "      <td>3.414634</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>29.512195</td>\n",
              "      <td>15.121951</td>\n",
              "      <td>3.658537</td>\n",
              "      <td>6.829268</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079405_1</th>\n",
              "      <td>4.146341</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>8.048780</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>23.902439</td>\n",
              "      <td>12.195122</td>\n",
              "      <td>23.902439</td>\n",
              "      <td>9.756098</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079407_1</th>\n",
              "      <td>35.121951</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.414634</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>6.829268</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>17.804878</td>\n",
              "      <td>7.073171</td>\n",
              "      <td>13.902439</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079411_1</th>\n",
              "      <td>6.829268</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.926829</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.439024</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>12.682927</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.658537</td>\n",
              "      <td>10.000000</td>\n",
              "      <td>12.926829</td>\n",
              "      <td>21.219512</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079413_1</th>\n",
              "      <td>15.609756</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>11.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>24.634146</td>\n",
              "      <td>9.512195</td>\n",
              "      <td>26.585366</td>\n",
              "      <td>1.951220</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079417_1</th>\n",
              "      <td>10.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>8.780488</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>37.073171</td>\n",
              "      <td>17.073171</td>\n",
              "      <td>19.268293</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079419_1</th>\n",
              "      <td>17.804878</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>8.536585</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.439024</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>31.219512</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>30.975610</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079423_1</th>\n",
              "      <td>11.219512</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>9.756098</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>35.609756</td>\n",
              "      <td>13.170732</td>\n",
              "      <td>20.975610</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079425_1</th>\n",
              "      <td>17.560976</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.243902</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>10.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>38.536585</td>\n",
              "      <td>9.268293</td>\n",
              "      <td>23.658537</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079429_1</th>\n",
              "      <td>19.512195</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>9.024390</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>35.853659</td>\n",
              "      <td>16.585366</td>\n",
              "      <td>17.804878</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-9bdf8b38-2615-44bf-93a6-d876e9f7e311')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-9bdf8b38-2615-44bf-93a6-d876e9f7e311 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-9bdf8b38-2615-44bf-93a6-d876e9f7e311');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 89
        }
      ],
      "source": [
        "sumtax"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 90,
      "metadata": {
        "id": "14QR2E6bjVjp"
      },
      "outputs": [],
      "source": [
        "sumtax=pd.merge(metadata,sumtax, left_index=True, right_index=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 91,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "nB0v3S-RjW4c",
        "outputId": "99a7a080-21a1-4a8d-a1f7-13138c80213a"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "            Assay Type  AvgSpotLen    Bases BioProject     BioSample  \\\n",
              "sample-id                                                              \n",
              "DRR079323_1   AMPLICON         400  1202750  PRJDB5388  SAMD00069337   \n",
              "DRR079327_1   AMPLICON         401  1204646  PRJDB5388  SAMD00069341   \n",
              "DRR079329_1   AMPLICON         400  1202374  PRJDB5388  SAMD00069343   \n",
              "DRR079333_1   AMPLICON         400  1201804  PRJDB5388  SAMD00069347   \n",
              "DRR079335_1   AMPLICON         400  1201037  PRJDB5388  SAMD00069349   \n",
              "DRR079339_1   AMPLICON         401  1203454  PRJDB5388  SAMD00069353   \n",
              "DRR079341_1   AMPLICON         402  1206843  PRJDB5388  SAMD00069355   \n",
              "DRR079345_1   AMPLICON         402  1208063  PRJDB5388  SAMD00069359   \n",
              "DRR079347_1   AMPLICON         402  1206011  PRJDB5388  SAMD00069361   \n",
              "DRR079351_1   AMPLICON         402  1206119  PRJDB5388  SAMD00069365   \n",
              "DRR079353_1   AMPLICON         401  1205015  PRJDB5388  SAMD00069367   \n",
              "DRR079357_1   AMPLICON         402  1207998  PRJDB5388  SAMD00069371   \n",
              "DRR079359_1   AMPLICON         402  1208484  PRJDB5388  SAMD00069373   \n",
              "DRR079363_1   AMPLICON         403  1209574  PRJDB5388  SAMD00069377   \n",
              "DRR079365_1   AMPLICON         402  1206531  PRJDB5388  SAMD00069379   \n",
              "DRR079369_1   AMPLICON         402  1208513  PRJDB5388  SAMD00069383   \n",
              "DRR079371_1   AMPLICON         401  1203354  PRJDB5388  SAMD00069385   \n",
              "DRR079375_1   AMPLICON         401  1204724  PRJDB5388  SAMD00069389   \n",
              "DRR079377_1   AMPLICON         402  1207319  PRJDB5388  SAMD00069391   \n",
              "DRR079381_1   AMPLICON         401  1205978  PRJDB5388  SAMD00069395   \n",
              "DRR079383_1   AMPLICON         401  1204780  PRJDB5388  SAMD00069397   \n",
              "DRR079387_1   AMPLICON         401  1205197  PRJDB5388  SAMD00069401   \n",
              "DRR079389_1   AMPLICON         401  1204420  PRJDB5388  SAMD00069403   \n",
              "DRR079393_1   AMPLICON         401  1203747  PRJDB5388  SAMD00069407   \n",
              "DRR079395_1   AMPLICON         402  1206659  PRJDB5388  SAMD00069409   \n",
              "DRR079399_1   AMPLICON         402  1206266  PRJDB5388  SAMD00069413   \n",
              "DRR079401_1   AMPLICON         401  1205145  PRJDB5388  SAMD00069415   \n",
              "DRR079405_1   AMPLICON         402  1208463  PRJDB5388  SAMD00069419   \n",
              "DRR079407_1   AMPLICON         401  1204482  PRJDB5388  SAMD00069421   \n",
              "DRR079411_1   AMPLICON         401  1205234  PRJDB5388  SAMD00069425   \n",
              "DRR079413_1   AMPLICON         402  1208075  PRJDB5388  SAMD00069427   \n",
              "DRR079417_1   AMPLICON         403  1209548  PRJDB5388  SAMD00069431   \n",
              "DRR079419_1   AMPLICON         403  1211194  PRJDB5388  SAMD00069433   \n",
              "DRR079423_1   AMPLICON         402  1208636  PRJDB5388  SAMD00069437   \n",
              "DRR079425_1   AMPLICON         402  1207743  PRJDB5388  SAMD00069439   \n",
              "DRR079429_1   AMPLICON         402  1206717  PRJDB5388  SAMD00069443   \n",
              "\n",
              "                 BioSampleModel    Bytes Center xname Collection_Date Consent  \\\n",
              "sample-id                                                                       \n",
              "DRR079323_1  MIMS.me.human-oral  2904393       UT_COB         missing  public   \n",
              "DRR079327_1  MIMS.me.human-oral  2906270       UT_COB         missing  public   \n",
              "DRR079329_1  MIMS.me.human-oral  2897112       UT_COB         missing  public   \n",
              "DRR079333_1  MIMS.me.human-oral  2894179       UT_COB         missing  public   \n",
              "DRR079335_1  MIMS.me.human-oral  2892993       UT_COB         missing  public   \n",
              "DRR079339_1  MIMS.me.human-oral  2898655       UT_COB         missing  public   \n",
              "DRR079341_1  MIMS.me.human-oral  2893466       UT_COB         missing  public   \n",
              "DRR079345_1  MIMS.me.human-oral  2903400       UT_COB         missing  public   \n",
              "DRR079347_1  MIMS.me.human-oral  2879839       UT_COB         missing  public   \n",
              "DRR079351_1  MIMS.me.human-oral  2905598       UT_COB         missing  public   \n",
              "DRR079353_1  MIMS.me.human-oral  2893969       UT_COB         missing  public   \n",
              "DRR079357_1  MIMS.me.human-oral  2896737       UT_COB         missing  public   \n",
              "DRR079359_1  MIMS.me.human-oral  2949507       UT_COB         missing  public   \n",
              "DRR079363_1  MIMS.me.human-oral  2962968       UT_COB         missing  public   \n",
              "DRR079365_1  MIMS.me.human-oral  2947616       UT_COB         missing  public   \n",
              "DRR079369_1  MIMS.me.human-oral  2960925       UT_COB         missing  public   \n",
              "DRR079371_1  MIMS.me.human-oral  2943481       UT_COB         missing  public   \n",
              "DRR079375_1  MIMS.me.human-oral  2958594       UT_COB         missing  public   \n",
              "DRR079377_1  MIMS.me.human-oral  2930337       UT_COB         missing  public   \n",
              "DRR079381_1  MIMS.me.human-oral  2940426       UT_COB         missing  public   \n",
              "DRR079383_1  MIMS.me.human-oral  2920773       UT_COB         missing  public   \n",
              "DRR079387_1  MIMS.me.human-oral  2943440       UT_COB         missing  public   \n",
              "DRR079389_1  MIMS.me.human-oral  2919435       UT_COB         missing  public   \n",
              "DRR079393_1  MIMS.me.human-oral  2937432       UT_COB         missing  public   \n",
              "DRR079395_1  MIMS.me.human-oral  2928937       UT_COB         missing  public   \n",
              "DRR079399_1  MIMS.me.human-oral  2938816       UT_COB         missing  public   \n",
              "DRR079401_1  MIMS.me.human-oral  2925095       UT_COB         missing  public   \n",
              "DRR079405_1  MIMS.me.human-oral  2940193       UT_COB         missing  public   \n",
              "DRR079407_1  MIMS.me.human-oral  2921729       UT_COB         missing  public   \n",
              "DRR079411_1  MIMS.me.human-oral  2932702       UT_COB         missing  public   \n",
              "DRR079413_1  MIMS.me.human-oral  3026067       UT_COB         missing  public   \n",
              "DRR079417_1  MIMS.me.human-oral  3035504       UT_COB         missing  public   \n",
              "DRR079419_1  MIMS.me.human-oral  3038575       UT_COB         missing  public   \n",
              "DRR079423_1  MIMS.me.human-oral  3041059       UT_COB         missing  public   \n",
              "DRR079425_1  MIMS.me.human-oral  3043535       UT_COB         missing  public   \n",
              "DRR079429_1  MIMS.me.human-oral  3031928       UT_COB         missing  public   \n",
              "\n",
              "             ...  g__Megasphaera  g__Oribacterium  g__Parvimonas  \\\n",
              "sample-id    ...                                                   \n",
              "DRR079323_1  ...        1.463415         0.000000       0.000000   \n",
              "DRR079327_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079329_1  ...        0.000000         0.000000       0.487805   \n",
              "DRR079333_1  ...        0.000000         0.000000       2.195122   \n",
              "DRR079335_1  ...        0.000000         0.000000       1.219512   \n",
              "DRR079339_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079341_1  ...        0.000000         0.731707       0.000000   \n",
              "DRR079345_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079347_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079351_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079353_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079357_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079359_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079363_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079365_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079369_1  ...        1.707317         0.000000       0.000000   \n",
              "DRR079371_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079375_1  ...        0.000000         1.951220       0.731707   \n",
              "DRR079377_1  ...        0.731707         0.000000       0.000000   \n",
              "DRR079381_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079383_1  ...        0.000000         0.000000       4.878049   \n",
              "DRR079387_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079389_1  ...        0.000000         0.000000       7.317073   \n",
              "DRR079393_1  ...        0.975610         0.000000       0.000000   \n",
              "DRR079395_1  ...        0.000000         0.000000       5.121951   \n",
              "DRR079399_1  ...        0.000000         0.243902       1.707317   \n",
              "DRR079401_1  ...        0.000000         0.000000       3.170732   \n",
              "DRR079405_1  ...        0.000000         0.000000       1.707317   \n",
              "DRR079407_1  ...        0.000000         0.000000       6.829268   \n",
              "DRR079411_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079413_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079417_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079419_1  ...        2.439024         0.000000       0.000000   \n",
              "DRR079423_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079425_1  ...        0.000000         0.000000       0.000000   \n",
              "DRR079429_1  ...        0.000000         0.000000       0.000000   \n",
              "\n",
              "             g__Peptococcus  g__Peptostreptococcus  g__Porphyromonas  \\\n",
              "sample-id                                                              \n",
              "DRR079323_1        0.000000               0.000000          4.146341   \n",
              "DRR079327_1        0.000000               0.000000          0.000000   \n",
              "DRR079329_1        0.000000               0.000000          0.000000   \n",
              "DRR079333_1        0.000000               0.975610          0.000000   \n",
              "DRR079335_1        0.000000               0.975610          9.512195   \n",
              "DRR079339_1        0.000000               0.000000          0.000000   \n",
              "DRR079341_1        0.000000               0.000000          0.000000   \n",
              "DRR079345_1        0.000000               0.731707          0.000000   \n",
              "DRR079347_1        0.000000               0.000000          0.000000   \n",
              "DRR079351_1        0.000000               0.000000          0.000000   \n",
              "DRR079353_1        0.000000               0.000000          0.000000   \n",
              "DRR079357_1        0.000000               0.000000          0.000000   \n",
              "DRR079359_1        0.000000               0.000000          0.000000   \n",
              "DRR079363_1        0.000000               2.195122          0.000000   \n",
              "DRR079365_1        0.000000               0.000000          0.000000   \n",
              "DRR079369_1        0.000000               1.219512          2.195122   \n",
              "DRR079371_1        0.000000               0.000000          2.195122   \n",
              "DRR079375_1        0.000000               1.951220          3.414634   \n",
              "DRR079377_1        0.243902               0.000000          1.707317   \n",
              "DRR079381_1        0.000000               0.000000          6.341463   \n",
              "DRR079383_1        0.000000               0.000000          0.000000   \n",
              "DRR079387_1        0.000000               2.439024          6.341463   \n",
              "DRR079389_1        0.000000               0.000000          0.000000   \n",
              "DRR079393_1        1.463415               0.000000          9.756098   \n",
              "DRR079395_1        0.000000               0.000000          0.000000   \n",
              "DRR079399_1        0.731707               0.000000          0.000000   \n",
              "DRR079401_1        3.414634               0.975610          4.390244   \n",
              "DRR079405_1        0.000000               1.463415          0.000000   \n",
              "DRR079407_1        0.000000               0.000000          0.000000   \n",
              "DRR079411_1        1.463415               0.000000          0.000000   \n",
              "DRR079413_1        0.000000               1.707317          0.000000   \n",
              "DRR079417_1        0.000000               0.000000          0.000000   \n",
              "DRR079419_1        0.000000               1.463415          0.000000   \n",
              "DRR079423_1        0.000000               2.195122          0.000000   \n",
              "DRR079425_1        0.000000               0.000000          0.000000   \n",
              "DRR079429_1        0.731707               0.000000          0.000000   \n",
              "\n",
              "             g__Prevotella  g__Streptococcus  g__Veillonella  g__[Prevotella]  \n",
              "sample-id                                                                      \n",
              "DRR079323_1      43.170732         18.048780        9.268293         5.121951  \n",
              "DRR079327_1      45.121951         13.658537        8.048780         5.365854  \n",
              "DRR079329_1      34.390244         16.097561        6.829268         3.902439  \n",
              "DRR079333_1      38.292683          5.121951        4.634146        10.975610  \n",
              "DRR079335_1      19.512195         18.780488        6.097561         0.000000  \n",
              "DRR079339_1      36.585366          8.536585        5.121951        10.731707  \n",
              "DRR079341_1      21.951220         32.682927       17.317073         1.463415  \n",
              "DRR079345_1      51.219512         16.097561        8.780488         5.853659  \n",
              "DRR079347_1      27.073171         36.585366       15.121951         0.975610  \n",
              "DRR079351_1      45.609756         19.268293        8.536585         4.390244  \n",
              "DRR079353_1      17.804878         49.024390       14.878049         0.487805  \n",
              "DRR079357_1      43.170732         24.878049       10.975610         3.658537  \n",
              "DRR079359_1      42.195122         19.512195       22.195122         0.000000  \n",
              "DRR079363_1      52.682927         14.146341       16.097561         0.000000  \n",
              "DRR079365_1      45.609756         21.951220       20.000000         0.000000  \n",
              "DRR079369_1      53.414634         15.121951       13.170732         0.975610  \n",
              "DRR079371_1      28.536585         22.682927       15.609756         0.000000  \n",
              "DRR079375_1      36.097561          7.804878        6.585366         5.853659  \n",
              "DRR079377_1      15.853659         29.512195       29.512195         0.000000  \n",
              "DRR079381_1      34.634146         10.731707        7.804878        10.243902  \n",
              "DRR079383_1       8.048780         24.634146       11.707317         0.000000  \n",
              "DRR079387_1      30.243902          6.097561       11.219512        17.560976  \n",
              "DRR079389_1      12.682927         25.121951        7.804878         0.000000  \n",
              "DRR079393_1      35.609756          5.853659       13.414634         9.512195  \n",
              "DRR079395_1       3.658537         31.219512        7.317073         7.560976  \n",
              "DRR079399_1      30.731707         14.146341        8.292683         8.048780  \n",
              "DRR079401_1      29.512195         15.121951        3.658537         6.829268  \n",
              "DRR079405_1      23.902439         12.195122       23.902439         9.756098  \n",
              "DRR079407_1       4.390244         17.804878        7.073171        13.902439  \n",
              "DRR079411_1      13.658537         10.000000       12.926829        21.219512  \n",
              "DRR079413_1      24.634146          9.512195       26.585366         1.951220  \n",
              "DRR079417_1      37.073171         17.073171       19.268293         0.000000  \n",
              "DRR079419_1      31.219512          4.390244       30.975610         0.000000  \n",
              "DRR079423_1      35.609756         13.170732       20.975610         0.000000  \n",
              "DRR079425_1      38.536585          9.268293       23.658537         0.000000  \n",
              "DRR079429_1      35.853659         16.585366       17.804878         0.000000  \n",
              "\n",
              "[36 rows x 56 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-93f0ff32-7e9c-44ac-9935-7ecdfabf397b\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Assay Type</th>\n",
              "      <th>AvgSpotLen</th>\n",
              "      <th>Bases</th>\n",
              "      <th>BioProject</th>\n",
              "      <th>BioSample</th>\n",
              "      <th>BioSampleModel</th>\n",
              "      <th>Bytes</th>\n",
              "      <th>Center xname</th>\n",
              "      <th>Collection_Date</th>\n",
              "      <th>Consent</th>\n",
              "      <th>...</th>\n",
              "      <th>g__Megasphaera</th>\n",
              "      <th>g__Oribacterium</th>\n",
              "      <th>g__Parvimonas</th>\n",
              "      <th>g__Peptococcus</th>\n",
              "      <th>g__Peptostreptococcus</th>\n",
              "      <th>g__Porphyromonas</th>\n",
              "      <th>g__Prevotella</th>\n",
              "      <th>g__Streptococcus</th>\n",
              "      <th>g__Veillonella</th>\n",
              "      <th>g__[Prevotella]</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>sample-id</th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>DRR079323_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1202750</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069337</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2904393</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.146341</td>\n",
              "      <td>43.170732</td>\n",
              "      <td>18.048780</td>\n",
              "      <td>9.268293</td>\n",
              "      <td>5.121951</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079327_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204646</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069341</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2906270</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>45.121951</td>\n",
              "      <td>13.658537</td>\n",
              "      <td>8.048780</td>\n",
              "      <td>5.365854</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079329_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1202374</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069343</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2897112</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.487805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>34.390244</td>\n",
              "      <td>16.097561</td>\n",
              "      <td>6.829268</td>\n",
              "      <td>3.902439</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079333_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1201804</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069347</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2894179</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>38.292683</td>\n",
              "      <td>5.121951</td>\n",
              "      <td>4.634146</td>\n",
              "      <td>10.975610</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079335_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>400</td>\n",
              "      <td>1201037</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069349</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2892993</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.219512</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>9.512195</td>\n",
              "      <td>19.512195</td>\n",
              "      <td>18.780488</td>\n",
              "      <td>6.097561</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079339_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203454</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069353</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2898655</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>36.585366</td>\n",
              "      <td>8.536585</td>\n",
              "      <td>5.121951</td>\n",
              "      <td>10.731707</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079341_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206843</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069355</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2893466</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>21.951220</td>\n",
              "      <td>32.682927</td>\n",
              "      <td>17.317073</td>\n",
              "      <td>1.463415</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079345_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208063</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069359</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2903400</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>51.219512</td>\n",
              "      <td>16.097561</td>\n",
              "      <td>8.780488</td>\n",
              "      <td>5.853659</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079347_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206011</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069361</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2879839</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>27.073171</td>\n",
              "      <td>36.585366</td>\n",
              "      <td>15.121951</td>\n",
              "      <td>0.975610</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079351_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206119</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069365</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2905598</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>45.609756</td>\n",
              "      <td>19.268293</td>\n",
              "      <td>8.536585</td>\n",
              "      <td>4.390244</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079353_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205015</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069367</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2893969</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>17.804878</td>\n",
              "      <td>49.024390</td>\n",
              "      <td>14.878049</td>\n",
              "      <td>0.487805</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079357_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207998</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069371</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2896737</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>43.170732</td>\n",
              "      <td>24.878049</td>\n",
              "      <td>10.975610</td>\n",
              "      <td>3.658537</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079359_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208484</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069373</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2949507</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>42.195122</td>\n",
              "      <td>19.512195</td>\n",
              "      <td>22.195122</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079363_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1209574</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069377</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2962968</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>52.682927</td>\n",
              "      <td>14.146341</td>\n",
              "      <td>16.097561</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079365_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206531</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069379</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2947616</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>45.609756</td>\n",
              "      <td>21.951220</td>\n",
              "      <td>20.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079369_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208513</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069383</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2960925</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.219512</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>53.414634</td>\n",
              "      <td>15.121951</td>\n",
              "      <td>13.170732</td>\n",
              "      <td>0.975610</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079371_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203354</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069385</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2943481</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>28.536585</td>\n",
              "      <td>22.682927</td>\n",
              "      <td>15.609756</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079375_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204724</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069389</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2958594</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.951220</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.951220</td>\n",
              "      <td>3.414634</td>\n",
              "      <td>36.097561</td>\n",
              "      <td>7.804878</td>\n",
              "      <td>6.585366</td>\n",
              "      <td>5.853659</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079377_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207319</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069391</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2930337</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.243902</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>15.853659</td>\n",
              "      <td>29.512195</td>\n",
              "      <td>29.512195</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079381_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205978</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069395</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2940426</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>6.341463</td>\n",
              "      <td>34.634146</td>\n",
              "      <td>10.731707</td>\n",
              "      <td>7.804878</td>\n",
              "      <td>10.243902</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079383_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204780</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069397</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2920773</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.878049</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>8.048780</td>\n",
              "      <td>24.634146</td>\n",
              "      <td>11.707317</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079387_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205197</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069401</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2943440</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.439024</td>\n",
              "      <td>6.341463</td>\n",
              "      <td>30.243902</td>\n",
              "      <td>6.097561</td>\n",
              "      <td>11.219512</td>\n",
              "      <td>17.560976</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079389_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204420</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069403</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2919435</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>7.317073</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>12.682927</td>\n",
              "      <td>25.121951</td>\n",
              "      <td>7.804878</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079393_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1203747</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069407</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2937432</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>9.756098</td>\n",
              "      <td>35.609756</td>\n",
              "      <td>5.853659</td>\n",
              "      <td>13.414634</td>\n",
              "      <td>9.512195</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079395_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206659</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069409</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2928937</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>5.121951</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>3.658537</td>\n",
              "      <td>31.219512</td>\n",
              "      <td>7.317073</td>\n",
              "      <td>7.560976</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079399_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206266</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069413</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2938816</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.243902</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>30.731707</td>\n",
              "      <td>14.146341</td>\n",
              "      <td>8.292683</td>\n",
              "      <td>8.048780</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079401_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205145</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069415</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2925095</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>3.170732</td>\n",
              "      <td>3.414634</td>\n",
              "      <td>0.975610</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>29.512195</td>\n",
              "      <td>15.121951</td>\n",
              "      <td>3.658537</td>\n",
              "      <td>6.829268</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079405_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208463</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069419</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2940193</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>23.902439</td>\n",
              "      <td>12.195122</td>\n",
              "      <td>23.902439</td>\n",
              "      <td>9.756098</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079407_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1204482</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069421</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2921729</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>6.829268</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>17.804878</td>\n",
              "      <td>7.073171</td>\n",
              "      <td>13.902439</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079411_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>401</td>\n",
              "      <td>1205234</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069425</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>2932702</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>13.658537</td>\n",
              "      <td>10.000000</td>\n",
              "      <td>12.926829</td>\n",
              "      <td>21.219512</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079413_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208075</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069427</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3026067</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.707317</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>24.634146</td>\n",
              "      <td>9.512195</td>\n",
              "      <td>26.585366</td>\n",
              "      <td>1.951220</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079417_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1209548</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069431</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3035504</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>37.073171</td>\n",
              "      <td>17.073171</td>\n",
              "      <td>19.268293</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079419_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>403</td>\n",
              "      <td>1211194</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069433</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3038575</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>2.439024</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.463415</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>31.219512</td>\n",
              "      <td>4.390244</td>\n",
              "      <td>30.975610</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079423_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1208636</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069437</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3041059</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.195122</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>35.609756</td>\n",
              "      <td>13.170732</td>\n",
              "      <td>20.975610</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079425_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1207743</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069439</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3043535</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>38.536585</td>\n",
              "      <td>9.268293</td>\n",
              "      <td>23.658537</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>DRR079429_1</th>\n",
              "      <td>AMPLICON</td>\n",
              "      <td>402</td>\n",
              "      <td>1206717</td>\n",
              "      <td>PRJDB5388</td>\n",
              "      <td>SAMD00069443</td>\n",
              "      <td>MIMS.me.human-oral</td>\n",
              "      <td>3031928</td>\n",
              "      <td>UT_COB</td>\n",
              "      <td>missing</td>\n",
              "      <td>public</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.731707</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>35.853659</td>\n",
              "      <td>16.585366</td>\n",
              "      <td>17.804878</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>36 rows × 56 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-93f0ff32-7e9c-44ac-9935-7ecdfabf397b')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-93f0ff32-7e9c-44ac-9935-7ecdfabf397b button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-93f0ff32-7e9c-44ac-9935-7ecdfabf397b');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 91
        }
      ],
      "source": [
        "sumtax"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 92,
      "metadata": {
        "id": "v4p07dv6lppQ"
      },
      "outputs": [],
      "source": [
        "sumtax['sample-id'] = sumtax.index"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 93,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "f35p79YK6mW6",
        "outputId": "ffeb638e-3ae9-4c2d-abe0-013c76683818"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Index(['Assay Type', 'AvgSpotLen', 'Bases', 'BioProject', 'BioSample',\n",
              "       'BioSampleModel', 'Bytes', 'Center xname', 'Collection_Date', 'Consent',\n",
              "       'DATASTORE filetype', 'DATASTORE provider', 'DATASTORE region',\n",
              "       'env_biome', 'env_feature', 'env_material', 'Experiment',\n",
              "       'geo_loc_name_country', 'geo_loc_name_country_continent',\n",
              "       'geo_loc_name', 'Host', 'host_subject_id', 'Instrument', 'Lat_Lon',\n",
              "       'Library xname', 'LibraryLayout', 'LibrarySelection', 'LibrarySource',\n",
              "       'Organism', 'Platform', 'project xname', 'ReleaseDate', 'Sample xname',\n",
              "       'sample yname', 'SRA Study', 'Group', ' g__', ' g__Abiotrophia',\n",
              "       ' g__Actinomyces', ' g__Catonella', ' g__Corynebacterium',\n",
              "       ' g__Dialister', ' g__Filifactor', ' g__Gemella', ' g__Granulicatella',\n",
              "       ' g__Lachnoanaerobaculum', ' g__Megasphaera', ' g__Oribacterium',\n",
              "       ' g__Parvimonas', ' g__Peptococcus', ' g__Peptostreptococcus',\n",
              "       ' g__Porphyromonas', ' g__Prevotella', ' g__Streptococcus',\n",
              "       ' g__Veillonella', ' g__[Prevotella]', 'sample-id'],\n",
              "      dtype='object')"
            ]
          },
          "metadata": {},
          "execution_count": 93
        }
      ],
      "source": [
        "sumtax.columns"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 94,
      "metadata": {
        "id": "49_qJlsl5EWY"
      },
      "outputs": [],
      "source": [
        "df2 = sumtax.filter(regex=' g')\n",
        "lst=df2.columns.values.tolist()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 96,
      "metadata": {
        "id": "RinfsvIaAeFr"
      },
      "outputs": [],
      "source": [
        "sumtax=sumtax.groupby(['Group']).mean()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 98,
      "metadata": {
        "id": "Os_7-zSxAt7e"
      },
      "outputs": [],
      "source": [
        "sumtax['Group']=sumtax.index"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 99,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "6clkDVQD7JHZ",
        "outputId": "dbdec241-0ec5-48b5-c305-ab7ed9cc91c6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "   Group                     Taxa        Abs\n",
            "0     AM                      g__   7.953930\n",
            "1     PM                      g__  11.531165\n",
            "2     AM           g__Abiotrophia   0.000000\n",
            "3     PM           g__Abiotrophia   0.284553\n",
            "4     AM           g__Actinomyces   0.433604\n",
            "5     PM           g__Actinomyces   0.392954\n",
            "6     AM             g__Catonella   0.230352\n",
            "7     PM             g__Catonella   0.040650\n",
            "8     AM       g__Corynebacterium   0.000000\n",
            "9     PM       g__Corynebacterium   0.013550\n",
            "10    AM             g__Dialister   0.135501\n",
            "11    PM             g__Dialister   0.027100\n",
            "12    AM            g__Filifactor   0.135501\n",
            "13    PM            g__Filifactor   0.189702\n",
            "14    AM               g__Gemella   0.298103\n",
            "15    PM               g__Gemella   0.000000\n",
            "16    AM        g__Granulicatella  12.127371\n",
            "17    PM        g__Granulicatella  11.978320\n",
            "18    AM   g__Lachnoanaerobaculum   0.257453\n",
            "19    PM   g__Lachnoanaerobaculum   0.420054\n",
            "20    AM           g__Megasphaera   0.149051\n",
            "21    PM           g__Megasphaera   0.257453\n",
            "22    AM          g__Oribacterium   0.121951\n",
            "23    PM          g__Oribacterium   0.040650\n",
            "24    AM            g__Parvimonas   0.352304\n",
            "25    PM            g__Parvimonas   1.612466\n",
            "26    AM           g__Peptococcus   0.243902\n",
            "27    PM           g__Peptococcus   0.203252\n",
            "28    AM    g__Peptostreptococcus   0.731707\n",
            "29    PM    g__Peptostreptococcus   0.284553\n",
            "30    AM         g__Porphyromonas   1.558266\n",
            "31    PM         g__Porphyromonas   1.219512\n",
            "32    AM            g__Prevotella  37.750678\n",
            "33    PM            g__Prevotella  24.932249\n",
            "34    AM         g__Streptococcus  12.804878\n",
            "35    PM         g__Streptococcus  22.330623\n",
            "36    AM           g__Veillonella  12.086721\n",
            "37    PM           g__Veillonella  15.311653\n",
            "38    AM          g__[Prevotella]   6.897019\n",
            "39    PM          g__[Prevotella]   2.344173\n"
          ]
        }
      ],
      "source": [
        "taxabar= pd.melt(sumtax,\n",
        "              id_vars=['Group'],\n",
        "              value_vars=sumtax.filter(regex=' g'),\n",
        "              var_name= 'Taxa', \n",
        "              value_name='Abs')\n",
        "print (taxabar)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 101,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 542
        },
        "id": "ETaH68LO-3kR",
        "outputId": "c9b9b588-262d-42ce-d420-67242e0fe920"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script src=\"https://cdn.plot.ly/plotly-2.8.3.min.js\"></script>                <div id=\"fb289a98-92ba-4b50-bcfe-abe997085b5a\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"fb289a98-92ba-4b50-bcfe-abe997085b5a\")) {                    Plotly.newPlot(                        \"fb289a98-92ba-4b50-bcfe-abe997085b5a\",                        [{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__\",\"marker\":{\"color\":\"#636efa\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__\",\"offsetgroup\":\" g__\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[7.953929539295394,11.531165311653117],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Abiotrophia<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Abiotrophia\",\"marker\":{\"color\":\"#EF553B\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Abiotrophia\",\"offsetgroup\":\" g__Abiotrophia\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.0,0.2845528455284553],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Actinomyces<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Actinomyces\",\"marker\":{\"color\":\"#00cc96\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Actinomyces\",\"offsetgroup\":\" g__Actinomyces\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.4336043360433604,0.3929539295392954],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Catonella<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Catonella\",\"marker\":{\"color\":\"#ab63fa\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Catonella\",\"offsetgroup\":\" g__Catonella\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.23035230352303523,0.04065040650406504],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Corynebacterium<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Corynebacterium\",\"marker\":{\"color\":\"#FFA15A\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Corynebacterium\",\"offsetgroup\":\" g__Corynebacterium\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.0,0.013550135501355013],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Dialister<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Dialister\",\"marker\":{\"color\":\"#19d3f3\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Dialister\",\"offsetgroup\":\" g__Dialister\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.13550135501355015,0.027100271002710025],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Filifactor<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Filifactor\",\"marker\":{\"color\":\"#FF6692\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Filifactor\",\"offsetgroup\":\" g__Filifactor\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.13550135501355012,0.18970189701897022],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Gemella<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Gemella\",\"marker\":{\"color\":\"#B6E880\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Gemella\",\"offsetgroup\":\" g__Gemella\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.2981029810298103,0.0],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Granulicatella<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Granulicatella\",\"marker\":{\"color\":\"#FF97FF\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Granulicatella\",\"offsetgroup\":\" g__Granulicatella\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[12.127371273712738,11.978319783197833],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Lachnoanaerobaculum<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Lachnoanaerobaculum\",\"marker\":{\"color\":\"#FECB52\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Lachnoanaerobaculum\",\"offsetgroup\":\" g__Lachnoanaerobaculum\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.25745257452574527,0.42005420054200543],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Megasphaera<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Megasphaera\",\"marker\":{\"color\":\"#636efa\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Megasphaera\",\"offsetgroup\":\" g__Megasphaera\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.14905149051490515,0.25745257452574527],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Oribacterium<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Oribacterium\",\"marker\":{\"color\":\"#EF553B\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Oribacterium\",\"offsetgroup\":\" g__Oribacterium\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.12195121951219513,0.04065040650406505],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Parvimonas<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Parvimonas\",\"marker\":{\"color\":\"#00cc96\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Parvimonas\",\"offsetgroup\":\" g__Parvimonas\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.3523035230352304,1.6124661246612466],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Peptococcus<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Peptococcus\",\"marker\":{\"color\":\"#ab63fa\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Peptococcus\",\"offsetgroup\":\" g__Peptococcus\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.24390243902439027,0.20325203252032523],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Peptostreptococcus<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Peptostreptococcus\",\"marker\":{\"color\":\"#FFA15A\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Peptostreptococcus\",\"offsetgroup\":\" g__Peptostreptococcus\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[0.7317073170731707,0.2845528455284553],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Porphyromonas<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Porphyromonas\",\"marker\":{\"color\":\"#19d3f3\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Porphyromonas\",\"offsetgroup\":\" g__Porphyromonas\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[1.5582655826558265,1.2195121951219512],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Prevotella<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Prevotella\",\"marker\":{\"color\":\"#FF6692\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Prevotella\",\"offsetgroup\":\" g__Prevotella\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[37.75067750677507,24.932249322493227],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Streptococcus<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Streptococcus\",\"marker\":{\"color\":\"#B6E880\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Streptococcus\",\"offsetgroup\":\" g__Streptococcus\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[12.804878048780488,22.33062330623306],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__Veillonella<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__Veillonella\",\"marker\":{\"color\":\"#FF97FF\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__Veillonella\",\"offsetgroup\":\" g__Veillonella\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[12.086720867208673,15.311653116531167],\"yaxis\":\"y\",\"type\":\"bar\"},{\"alignmentgroup\":\"True\",\"hovertemplate\":\"Taxa= g__[Prevotella]<br>Group=%{x}<br>Abs=%{y}<extra></extra>\",\"legendgroup\":\" g__[Prevotella]\",\"marker\":{\"color\":\"#FECB52\",\"pattern\":{\"shape\":\"\"}},\"name\":\" g__[Prevotella]\",\"offsetgroup\":\" g__[Prevotella]\",\"orientation\":\"v\",\"showlegend\":true,\"textposition\":\"auto\",\"x\":[\"AM\",\"PM\"],\"xaxis\":\"x\",\"y\":[6.897018970189702,2.3441734417344176],\"yaxis\":\"y\",\"type\":\"bar\"}],                        {\"template\":{\"data\":{\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"choropleth\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"choropleth\"}],\"contour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"contour\"}],\"contourcarpet\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"contourcarpet\"}],\"heatmap\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmap\"}],\"heatmapgl\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"heatmapgl\"}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"histogram2d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2d\"}],\"histogram2dcontour\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"histogram2dcontour\"}],\"mesh3d\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"type\":\"mesh3d\"}],\"parcoords\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"parcoords\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}],\"scatter\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter\"}],\"scatter3d\":[{\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatter3d\"}],\"scattercarpet\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattercarpet\"}],\"scattergeo\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergeo\"}],\"scattergl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattergl\"}],\"scattermapbox\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scattermapbox\"}],\"scatterpolar\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolar\"}],\"scatterpolargl\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterpolargl\"}],\"scatterternary\":[{\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"type\":\"scatterternary\"}],\"surface\":[{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"type\":\"surface\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}]},\"layout\":{\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"autotypenumbers\":\"strict\",\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]],\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]},\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"geo\":{\"bgcolor\":\"white\",\"lakecolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"showlakes\":true,\"showland\":true,\"subunitcolor\":\"white\"},\"hoverlabel\":{\"align\":\"left\"},\"hovermode\":\"closest\",\"mapbox\":{\"style\":\"light\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"gridwidth\":2,\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\"}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"ternary\":{\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"bgcolor\":\"#E5ECF6\",\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"title\":{\"x\":0.05},\"xaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2},\"yaxis\":{\"automargin\":true,\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"zerolinewidth\":2}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"Group\"}},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"Abs\"}},\"legend\":{\"title\":{\"text\":\"Taxa\"},\"tracegroupgap\":0},\"margin\":{\"t\":60},\"barmode\":\"relative\"},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('fb289a98-92ba-4b50-bcfe-abe997085b5a');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "fig = px.bar(taxabar, x=\"Group\", y=\"Abs\", color=\"Taxa\")\n",
        "fig.show()"
      ]
    }
  ],
  "metadata": {
    "colab": {
      "collapsed_sections": [],
      "name": "Copy of DiversityNasal_Novogene2022_data/oral(3).ipynb",
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}